diff --git a/mibs/cisco/cat3750/v2/CISCO-VLAN-MEMBERSHIP-MIB.my b/mibs/cisco/cat3750/v2/CISCO-VLAN-MEMBERSHIP-MIB.my
index 3f09fe3..f0945f6 100644
--- a/mibs/cisco/cat3750/v2/CISCO-VLAN-MEMBERSHIP-MIB.my
+++ b/mibs/cisco/cat3750/v2/CISCO-VLAN-MEMBERSHIP-MIB.my
@@ -1,8 +1,8 @@
 -- *****************************************************************
 -- CISCO-VLAN-MEMBERSHIP-MIB
---   
+--
 -- November 1998, Chin-Chai Low
---   
+--
 -- Copyright (c) 1998-2004, 2007 by Cisco Systems Inc.
 -- All rights reserved.
 -- *****************************************************************
@@ -22,7 +22,8 @@ IMPORTS
     NOTIFICATION-GROUP
         FROM SNMPv2-CONF
     RowStatus,
-    TruthValue
+    TruthValue,
+    DisplayString
         FROM SNMPv2-TC
     ifIndex
         FROM IF-MIB
@@ -73,7 +74,7 @@ ciscoVlanMembershipMIB MODULE-IDENTITY
     REVISION        "200306050000Z"
     DESCRIPTION
         "Deprecate vmMembershipSummaryMemberPorts and define
-        vmMembershipSummaryMember2kPorts to support up to 2k 
+        vmMembershipSummaryMember2kPorts to support up to 2k
         bridge ports."
     REVISION        "200203280000Z"
     DESCRIPTION
@@ -132,7 +133,7 @@ vmVmpsVQPVersion OBJECT-TYPE
         membership assignments of dynamic VLAN ports.
         A VMPS provides VLAN membership policy
         assignments based on the content of the packets
-        received on a port." 
+        received on a port."
     ::= { vmVmps 1 }
 
 vmVmpsRetries OBJECT-TYPE
@@ -141,7 +142,7 @@ vmVmpsRetries OBJECT-TYPE
     STATUS          current
     DESCRIPTION
         "The number of retries for VQP requests to a VMPS before
-        using the next available VMPS." 
+        using the next available VMPS."
     ::= { vmVmps 2 }
 
 vmVmpsReconfirmInterval OBJECT-TYPE
@@ -155,7 +156,7 @@ vmVmpsReconfirmInterval OBJECT-TYPE
         the interval to perform reconfirmation. If the value is
         set to 0, the switch does not reconfirm membership with
         VMPS."
-    DEFVAL          { 60 } 
+    DEFVAL          { 60 }
     ::= { vmVmps 3 }
 
 vmVmpsReconfirm OBJECT-TYPE
@@ -168,7 +169,7 @@ vmVmpsReconfirm OBJECT-TYPE
     DESCRIPTION
         "Setting this object to execute(2) causes the switch
         to reconfirm membership of every dynamic port.
-        Reading this object always return ready(1)." 
+        Reading this object always return ready(1)."
     ::= { vmVmps 4 }
 
 vmVmpsReconfirmResult OBJECT-TYPE
@@ -195,7 +196,7 @@ vmVmpsReconfirmResult OBJECT-TYPE
                                   VMPS responded
              noVmps(5)          - No VMPS configured
              noDynamicPort(6)   - No dynamic ports configured
-             noHostConnected(7) - No hosts on dynamic ports" 
+             noHostConnected(7) - No hosts on dynamic ports"
     ::= { vmVmps 5 }
 
 vmVmpsCurrent OBJECT-TYPE
@@ -203,11 +204,11 @@ vmVmpsCurrent OBJECT-TYPE
     MAX-ACCESS      read-only
     STATUS          current
     DESCRIPTION
-        "This is the IpAddress of the current VMPS used." 
+        "This is the IpAddress of the current VMPS used."
     ::= { vmVmps 6 }
 
 vmVmpsTable OBJECT-TYPE
-    SYNTAX          SEQUENCE OF VmVmpsEntry 
+    SYNTAX          SEQUENCE OF VmVmpsEntry
     MAX-ACCESS      not-accessible
     STATUS          current
     DESCRIPTION
@@ -228,7 +229,7 @@ vmVmpsEntry OBJECT-TYPE
     STATUS          current
     DESCRIPTION
         "An entry (conceptual row) in the vmVmpsTable."
-    INDEX           { vmVmpsIpAddress } 
+    INDEX           { vmVmpsIpAddress }
     ::= { vmVmpsTable 1 }
 
 VmVmpsEntry ::= SEQUENCE {
@@ -242,7 +243,7 @@ vmVmpsIpAddress OBJECT-TYPE
     MAX-ACCESS      read-only
     STATUS          current
     DESCRIPTION
-        "The Ip Address of the VMPS." 
+        "The Ip Address of the VMPS."
     ::= { vmVmpsEntry 1 }
 
 vmVmpsPrimary OBJECT-TYPE
@@ -258,7 +259,7 @@ vmVmpsPrimary OBJECT-TYPE
         a value of false after creation will result in
         a return of bad value. Deleting an entry whose
         value is true will result in the first entry
-        in the table being set to true." 
+        in the table being set to true."
     ::= { vmVmpsEntry 2 }
 
 vmVmpsRowStatus OBJECT-TYPE
@@ -266,14 +267,14 @@ vmVmpsRowStatus OBJECT-TYPE
     MAX-ACCESS      read-create
     STATUS          current
     DESCRIPTION
-        "The status of this conceptual row." 
+        "The status of this conceptual row."
     ::= { vmVmpsEntry 3 }
- 
+
 
 -- VLAN Membership Info
 
 vmMembershipSummaryTable OBJECT-TYPE
-    SYNTAX          SEQUENCE OF VmMembershipSummaryEntry 
+    SYNTAX          SEQUENCE OF VmMembershipSummaryEntry
     MAX-ACCESS      not-accessible
     STATUS          current
     DESCRIPTION
@@ -296,12 +297,12 @@ vmMembershipSummaryEntry OBJECT-TYPE
     DESCRIPTION
         "An entry (conceptual row) in the
         vmMembershipSummaryTable."
-    INDEX           { vmMembershipSummaryVlanIndex } 
+    INDEX           { vmMembershipSummaryVlanIndex }
     ::= { vmMembershipSummaryTable 1 }
 
 VmMembershipSummaryEntry ::= SEQUENCE {
         vmMembershipSummaryVlanIndex     VlanIndex,
-        vmMembershipSummaryMemberPorts   OCTET STRING,
+        vmMembershipSummaryMemberPorts   DisplayString,
         vmMembershipSummaryMember2kPorts CiscoPortList
 }
 
@@ -310,11 +311,11 @@ vmMembershipSummaryVlanIndex OBJECT-TYPE
     MAX-ACCESS      not-accessible
     STATUS          current
     DESCRIPTION
-        "The VLAN id of the VLAN." 
+        "The VLAN id of the VLAN."
     ::= { vmMembershipSummaryEntry 1 }
 
 vmMembershipSummaryMemberPorts OBJECT-TYPE
-    SYNTAX          OCTET STRING (SIZE  (0..128))
+    SYNTAX          DisplayString (SIZE  (0..128))
     MAX-ACCESS      read-only
     STATUS          deprecated
     DESCRIPTION
@@ -335,7 +336,7 @@ vmMembershipSummaryMemberPorts OBJECT-TYPE
         value of '0'.
 
         A port number is the value of dot1dBasePort for
-        the port in the BRIDGE-MIB (RFC 1493)." 
+        the port in the BRIDGE-MIB (RFC 1493)."
     ::= { vmMembershipSummaryEntry 2 }
 
 vmMembershipSummaryMember2kPorts OBJECT-TYPE
@@ -345,11 +346,11 @@ vmMembershipSummaryMember2kPorts OBJECT-TYPE
     DESCRIPTION
         "The set of the device's member ports that belong
         to the VLAN. It has the VLAN membership information
-        of up to 2048 ports with the port number from 1 to 
+        of up to 2048 ports with the port number from 1 to
         2048.
 
         Each octet within the value of this object specifies a
-        set of eight ports, with the first octet specifying 
+        set of eight ports, with the first octet specifying
         ports 1 through 8, the second octet specifying ports 9
         through 16, etc.   Within each octet, the most
         significant bit represents the lowest numbered
@@ -362,13 +363,13 @@ vmMembershipSummaryMember2kPorts OBJECT-TYPE
         value of '0'.
 
         A port number is the value of dot1dBasePort for
-        the port in the BRIDGE-MIB (RFC 1493)." 
+        the port in the BRIDGE-MIB (RFC 1493)."
     ::= { vmMembershipSummaryEntry 3 }
- 
+
 
 
 vmMembershipTable OBJECT-TYPE
-    SYNTAX          SEQUENCE OF VmMembershipEntry 
+    SYNTAX          SEQUENCE OF VmMembershipEntry
     MAX-ACCESS      not-accessible
     STATUS          current
     DESCRIPTION
@@ -377,7 +378,7 @@ vmMembershipTable OBJECT-TYPE
         assigned to a static or dynamic access port. Trunk
         ports are not  represented in this table.  An entry
         may be created and deleted when ports are created or
-        deleted via SNMP or the management console on a 
+        deleted via SNMP or the management console on a
         device."
     ::= { vmMembership 2 }
 
@@ -387,17 +388,17 @@ vmMembershipEntry OBJECT-TYPE
     STATUS          current
     DESCRIPTION
         "An entry (conceptual row) in the vmMembershipTable."
-    INDEX           { ifIndex } 
+    INDEX           { ifIndex }
     ::= { vmMembershipTable 1 }
 
 VmMembershipEntry ::= SEQUENCE {
         vmVlanType   INTEGER ,
         vmVlan       INTEGER,
         vmPortStatus INTEGER ,
-        vmVlans      OCTET STRING,
-        vmVlans2k    OCTET STRING,
-        vmVlans3k    OCTET STRING,
-        vmVlans4k    OCTET STRING
+        vmVlans      DisplayString,
+        vmVlans2k    DisplayString,
+        vmVlans3k    DisplayString,
+        vmVlans4k    DisplayString
 }
 
 vmVlanType OBJECT-TYPE
@@ -419,7 +420,7 @@ vmVlanType OBJECT-TYPE
 
         A static or dynamic port membership is specified
         by the value of vmVlan. A multiVlan port membership is
-        specified by the value of vmVlans." 
+        specified by the value of vmVlans."
     ::= { vmMembershipEntry 1 }
 
 vmVlan OBJECT-TYPE
@@ -440,7 +441,7 @@ vmVlan OBJECT-TYPE
 
         If vmVlanType is dynamic the object's value is
         0 if the port is currently not assigned to a VLAN.
-        In addition, the object may be set to 0 only." 
+        In addition, the object may be set to 0 only."
     ::= { vmMembershipEntry 2 }
 
 vmPortStatus OBJECT-TYPE
@@ -454,16 +455,16 @@ vmPortStatus OBJECT-TYPE
     DESCRIPTION
         "An indication of the current VLAN status of the port.
         A status of inactive(1) indicates that a dynamic port
-        does not yet have a VLAN assigned, or a port is 
-        assigned to a VLAN that is currently not active. A 
-        status of active(2) indicates that the currently 
-        assigned VLAN is active. A status of shutdown(3) 
+        does not yet have a VLAN assigned, or a port is
+        assigned to a VLAN that is currently not active. A
+        status of active(2) indicates that the currently
+        assigned VLAN is active. A status of shutdown(3)
         indicates that the port has been disabled as a result
-        of VQP shutdown response." 
+        of VQP shutdown response."
     ::= { vmMembershipEntry 3 }
 
 vmVlans OBJECT-TYPE
-    SYNTAX          OCTET STRING (SIZE  (0..128))
+    SYNTAX          DisplayString (SIZE  (0..128))
     MAX-ACCESS      read-write
     STATUS          current
     DESCRIPTION
@@ -486,11 +487,11 @@ vmVlans OBJECT-TYPE
         value of this object.  If that bit has a value of
         '1' then that VLAN is included in the set of
         VLANs; the VLAN is not included if its bit has a
-        value of '0'." 
+        value of '0'."
     ::= { vmMembershipEntry 4 }
 
 vmVlans2k OBJECT-TYPE
-    SYNTAX          OCTET STRING (SIZE  (0..128))
+    SYNTAX          DisplayString (SIZE  (0..128))
     MAX-ACCESS      read-write
     STATUS          current
     DESCRIPTION
@@ -504,20 +505,20 @@ vmVlans2k OBJECT-TYPE
 
         Each octet within the value of this object specifies a
         set of eight VLANs, with the first octet specifying
-        VLAN id 1024 through 1031, the second octet specifying 
-        VLAN ids 1032 through 1039, etc.  Within each octet, 
-        the most significant bit represents the lowest 
-        numbered VLAN id, and the least significant bit 
-        represents the highest numbered VLAN id.  Thus, each 
+        VLAN id 1024 through 1031, the second octet specifying
+        VLAN ids 1032 through 1039, etc.  Within each octet,
+        the most significant bit represents the lowest
+        numbered VLAN id, and the least significant bit
+        represents the highest numbered VLAN id.  Thus, each
         VLAN of the port is represented by a single bit within
         the value of this object.  If that bit has a value of
         '1' then that VLAN is included in the set of
         VLANs; the VLAN is not included if its bit has a
-        value of '0'." 
+        value of '0'."
     ::= { vmMembershipEntry 5 }
 
 vmVlans3k OBJECT-TYPE
-    SYNTAX          OCTET STRING (SIZE  (0..128))
+    SYNTAX          DisplayString (SIZE  (0..128))
     MAX-ACCESS      read-write
     STATUS          current
     DESCRIPTION
@@ -531,20 +532,20 @@ vmVlans3k OBJECT-TYPE
 
         Each octet within the value of this object specifies a
         set of eight VLANs, with the first octet specifying
-        VLAN id 2048 through 2055, the second octet specifying 
-        VLAN ids 2056 through 2063, etc.   Within each octet, 
-        the most significant bit represents the lowest 
-        numbered VLAN id, and the least significant bit 
+        VLAN id 2048 through 2055, the second octet specifying
+        VLAN ids 2056 through 2063, etc.   Within each octet,
+        the most significant bit represents the lowest
+        numbered VLAN id, and the least significant bit
         represents the highest numbered VLAN id.  Thus, each
         VLAN of the port is represented by a single bit within
         the value of this object.  If that bit has a value of
         '1' then that VLAN is included in the set of
         VLANs; the VLAN is not included if its bit has a
-        value of '0'." 
+        value of '0'."
     ::= { vmMembershipEntry 6 }
 
 vmVlans4k OBJECT-TYPE
-    SYNTAX          OCTET STRING (SIZE  (0..128))
+    SYNTAX          DisplayString (SIZE  (0..128))
     MAX-ACCESS      read-write
     STATUS          current
     DESCRIPTION
@@ -558,33 +559,33 @@ vmVlans4k OBJECT-TYPE
 
         Each octet within the value of this object specifies a
         set of eight VLANs, with the first octet specifying
-        VLAN id 3072 through 3079, the second octet specifying 
-        VLAN ids 3040 through 3047, etc.   Within each octet, 
-        the most significant bit represents the lowest 
-        numbered VLAN id, and the least significant bit 
+        VLAN id 3072 through 3079, the second octet specifying
+        VLAN ids 3040 through 3047, etc.   Within each octet,
+        the most significant bit represents the lowest
+        numbered VLAN id, and the least significant bit
         represents the highest numbered VLAN id.  Thus, each
         VLAN of the port is represented by a single bit within
         the value of this object.  If that bit has a value of
         '1' then that VLAN is included in the set of
         VLANs; the VLAN is not included if its bit has a
-        value of '0'." 
+        value of '0'."
     ::= { vmMembershipEntry 7 }
- 
+
 
 
 vmMembershipSummaryExtTable OBJECT-TYPE
-    SYNTAX          SEQUENCE OF VmMembershipSummaryExtEntry 
+    SYNTAX          SEQUENCE OF VmMembershipSummaryExtEntry
     MAX-ACCESS      not-accessible
     STATUS          current
     DESCRIPTION
         "A summary of VLAN membership of non-trunk
-        bridge ports. This table is used for 
+        bridge ports. This table is used for
         retrieving VLAN membership information
-        for the device which supports dot1dBasePort 
+        for the device which supports dot1dBasePort
         with value greater than 2048.
 
         A row is created for a VLAN and a particular
-        bridge port range, where at least one port 
+        bridge port range, where at least one port
         in the range is assigned to this VLAN.
 
         VLAN membership can only be modified via the
@@ -601,7 +602,7 @@ vmMembershipSummaryExtEntry OBJECT-TYPE
     INDEX           {
                         vmMembershipSummaryVlanIndex,
                         vmMembershipPortRangeIndex
-                    } 
+                    }
     ::= { vmMembershipSummaryExtTable 1 }
 
 VmMembershipSummaryExtEntry ::= SEQUENCE {
@@ -614,7 +615,7 @@ vmMembershipPortRangeIndex OBJECT-TYPE
     MAX-ACCESS      not-accessible
     STATUS          current
     DESCRIPTION
-        "The bridge port range index of this row." 
+        "The bridge port range index of this row."
     ::= { vmMembershipSummaryExtEntry 1 }
 
 vmMembershipSummaryExtPorts OBJECT-TYPE
@@ -629,12 +630,12 @@ vmMembershipSummaryExtPorts OBJECT-TYPE
         object of the same row. For example, if the value
         of vmMembershipPortRangeIndex is 'twoKto4K', the
         port number indicated in this object starting from
-        2049 and ending to 4096. 
+        2049 and ending to 4096.
 
         A port number is the value of dot1dBasePort for
-        the port in the BRIDGE-MIB (RFC 1493)." 
+        the port in the BRIDGE-MIB (RFC 1493)."
     ::= { vmMembershipSummaryExtEntry 2 }
- 
+
 
 
 vmVlanCreationMode OBJECT-TYPE
@@ -656,7 +657,7 @@ vmVlanCreationMode OBJECT-TYPE
         manual(2):     a non-existing VLAN will not be created
                        automatically by the system and need to be
                        manually created by the users after assigned
-                       to a port." 
+                       to a port."
     ::= { vmMembership 4 }
 
 -- VQP Statistics
@@ -667,7 +668,7 @@ vmVQPQueries OBJECT-TYPE
     STATUS          current
     DESCRIPTION
         "The total number of VQP requests sent by this device
-        to all VMPS since last system re-initialization." 
+        to all VMPS since last system re-initialization."
     ::= { vmStatistics 1 }
 
 vmVQPResponses OBJECT-TYPE
@@ -676,7 +677,7 @@ vmVQPResponses OBJECT-TYPE
     STATUS          current
     DESCRIPTION
         "The number of VQP responses received by this device
-        from all VMPS since last system re-initialization." 
+        from all VMPS since last system re-initialization."
     ::= { vmStatistics 2 }
 
 vmVmpsChanges OBJECT-TYPE
@@ -686,8 +687,8 @@ vmVmpsChanges OBJECT-TYPE
     DESCRIPTION
         "The number of times, since last system
         re-initialization, the current VMPS was changed. The
-        current VMPS is changed whenever the VMPS fails to 
-        response after vmVmpsRetries of a VQP request." 
+        current VMPS is changed whenever the VMPS fails to
+        response after vmVmpsRetries of a VQP request."
     ::= { vmStatistics 3 }
 
 vmVQPShutdown OBJECT-TYPE
@@ -696,10 +697,10 @@ vmVQPShutdown OBJECT-TYPE
     STATUS          current
     DESCRIPTION
         "The number of times, since last system
-        re-initialization, a VQP response indicates 
-        'shutdown'. A 'shutdown' response is a result of 
+        re-initialization, a VQP response indicates
+        'shutdown'. A 'shutdown' response is a result of
         the membership policy configured at a VMPS
-        by the administrator." 
+        by the administrator."
     ::= { vmStatistics 4 }
 
 vmVQPDenied OBJECT-TYPE
@@ -708,10 +709,10 @@ vmVQPDenied OBJECT-TYPE
     STATUS          current
     DESCRIPTION
         "The number of times, since last system
-        re-initialization, a VQP response indicates 
-        'denied'. A 'denied' response is a result of 
+        re-initialization, a VQP response indicates
+        'denied'. A 'denied' response is a result of
         the membership policy configured at a VMPS
-        by the administrator." 
+        by the administrator."
     ::= { vmStatistics 5 }
 
 vmVQPWrongDomain OBJECT-TYPE
@@ -720,11 +721,11 @@ vmVQPWrongDomain OBJECT-TYPE
     STATUS          current
     DESCRIPTION
         "The number of times, since last system
-        re-initialization, a VQP response indicates wrong 
-        management domain. A wrong management domain 
-        response indicates that the VMPS used serves a 
+        re-initialization, a VQP response indicates wrong
+        management domain. A wrong management domain
+        response indicates that the VMPS used serves a
         management domain that is different
-        from the device's management domain." 
+        from the device's management domain."
     ::= { vmStatistics 6 }
 
 vmVQPWrongVersion OBJECT-TYPE
@@ -733,11 +734,11 @@ vmVQPWrongVersion OBJECT-TYPE
     STATUS          current
     DESCRIPTION
         "The number of times, since last system
-        re-initialization, a VQP response indicates wrong 
-        VQP version. A wrong VQP version response 
-        indicates that the VMPS used supports a VQP 
-        version that is different from the device's 
-        VQP version." 
+        re-initialization, a VQP response indicates wrong
+        VQP version. A wrong VQP version response
+        indicates that the VMPS used supports a VQP
+        version that is different from the device's
+        VQP version."
     ::= { vmStatistics 7 }
 
 vmInsufficientResources OBJECT-TYPE
@@ -746,11 +747,11 @@ vmInsufficientResources OBJECT-TYPE
     STATUS          current
     DESCRIPTION
         "The number of times, since last system
-        re-initialization, a VQP response indicates 
-        insufficient resources. An insufficient resources 
-        response indicates that the VMPS used does not 
+        re-initialization, a VQP response indicates
+        insufficient resources. An insufficient resources
+        response indicates that the VMPS used does not
         have the required resources to verify the
-        membership assignment requested." 
+        membership assignment requested."
     ::= { vmStatistics 8 }
 
 -- Status
@@ -761,12 +762,12 @@ vmNotificationsEnabled OBJECT-TYPE
     STATUS          current
     DESCRIPTION
         "An indication of whether the notifications/traps
-        defined in this MIB are enabled." 
+        defined in this MIB are enabled."
     ::= { vmStatus 1 }
 -- Voice Vlan Configuration
 
 vmVoiceVlanTable OBJECT-TYPE
-    SYNTAX          SEQUENCE OF VmVoiceVlanEntry 
+    SYNTAX          SEQUENCE OF VmVoiceVlanEntry
     MAX-ACCESS      not-accessible
     STATUS          current
     DESCRIPTION
@@ -783,7 +784,7 @@ vmVoiceVlanEntry OBJECT-TYPE
         "An entry (conceptual row) in the vmVoiceVlanTable.
         Only interfaces which support Voice Vlan feature
         are shown."
-    INDEX           { ifIndex } 
+    INDEX           { ifIndex }
     ::= { vmVoiceVlanTable 1 }
 
 VmVoiceVlanEntry ::= SEQUENCE {
@@ -799,13 +800,13 @@ vmVoiceVlanId OBJECT-TYPE
         "The Voice Vlan ID (VVID) to which this
         port belongs to.
 
-        0    -    The CDP packets transmitting 
+        0    -    The CDP packets transmitting
                   through this port would contain
-                  Appliance VLAN-ID TLV with value 
-                  of 0. VoIP and related packets 
-                  are expected to be sent and 
-                  received with VLAN-id=0 and an 
-                  802.1p priority. 
+                  Appliance VLAN-ID TLV with value
+                  of 0. VoIP and related packets
+                  are expected to be sent and
+                  received with VLAN-id=0 and an
+                  802.1p priority.
 
         1..4094 - The CDP packets transmitting
                   through this port would contain
@@ -819,17 +820,17 @@ vmVoiceVlanId OBJECT-TYPE
                   through this port would contain
                   Appliance VLAN-ID TLV with value
                   of 4095. VoIP and related packets
-                  are expected to be sent and 
-                  received untagged without an 
+                  are expected to be sent and
+                  received untagged without an
                   802.1p priority.
 
-        4096  -   The CDP packets transmitting 
-                  through this port would not 
-                  include Appliance VLAN-ID TLV; 
-                  or, if the VVID is not supported 
+        4096  -   The CDP packets transmitting
+                  through this port would not
+                  include Appliance VLAN-ID TLV;
+                  or, if the VVID is not supported
                   on the port, this MIB object will
-                  not be configurable and will 
-                  return 4096." 
+                  not be configurable and will
+                  return 4096."
     ::= { vmVoiceVlanEntry 1 }
 
 vmVoiceVlanCdpVerifyEnable OBJECT-TYPE
@@ -840,16 +841,16 @@ vmVoiceVlanCdpVerifyEnable OBJECT-TYPE
         "Enable or Disable the feature of CDP message
         verification of voice VLANs.
 
-        true   - The voice VLAN vmVoiceVlan is enabled 
-                 only after CDP messages are received 
+        true   - The voice VLAN vmVoiceVlan is enabled
+                 only after CDP messages are received
                  from the IP phone.
 
         false -  The voice VLAN vmVoiceVlan is enabled
                  as soon as the IP phone interface is
-                 up. There is no verification needed 
-                 from CDP messages from the IP phone." 
+                 up. There is no verification needed
+                 from CDP messages from the IP phone."
     ::= { vmVoiceVlanEntry 2 }
- 
+
 
 -- Notifications
 
@@ -969,7 +970,7 @@ vmMIBCompliance4 MODULE-COMPLIANCE
     GROUP           vmStatusGroup
     DESCRIPTION
         "This group must be implemented by the
-        entities which support the 
+        entities which support the
         traps/notifications defined in this
         MIB."
 
@@ -1098,8 +1099,8 @@ vmMIBCompliance6 MODULE-COMPLIANCE
     GROUP           vmMembershipExtGroup
     DESCRIPTION
         "This group must be implemented by the
-        entities which support the VLAN 
-        membership feature with bridge port 
+        entities which support the VLAN
+        membership feature with bridge port
         ID greater than 2048."
 
     OBJECT          vmVlanType
@@ -1263,7 +1264,7 @@ vm4kVlanGroup OBJECT-GROUP
     STATUS          current
     DESCRIPTION
         "A collection of objects providing multiVlan
-        information for VLANS with VlanIndex from 
+        information for VLANS with VlanIndex from
         1024 to 4095 which are assigned to one port."
     ::= { vmMIBGroups 6 }
 
@@ -1280,7 +1281,7 @@ vm1kVlanGroup OBJECT-GROUP
     STATUS          current
     DESCRIPTION
         "A collection of objects providing multiVlan
-        information for VLANS with VlanIndex from 
+        information for VLANS with VlanIndex from
         0 to 1023 which are assigned to one port."
     ::= { vmMIBGroups 8 }
 
@@ -1302,7 +1303,7 @@ vmVoiceVlanExtGroup OBJECT-GROUP
     STATUS          current
     DESCRIPTION
         "A collection of objects providing the
-        information of CDP verification 
+        information of CDP verification
         of Voice Vlan feature."
     ::= { vmMIBGroups 10 }
 
@@ -1311,7 +1312,7 @@ vmMembershipExtGroup OBJECT-GROUP
     STATUS          current
     DESCRIPTION
         "A collection of objects providing
-        information for VLAN membership with 
+        information for VLAN membership with
         bridge port ID greater than 2048."
     ::= { vmMIBGroups 11 }
 
