diff --git a/mibs/cisco/wlc/v2/CISCO-LWAPP-DOT11-MIB.my b/mibs/cisco/wlc/v2/CISCO-LWAPP-DOT11-MIB.my
index 0cb6ffb..ee57cc8 100644
--- a/mibs/cisco/wlc/v2/CISCO-LWAPP-DOT11-MIB.my
+++ b/mibs/cisco/wlc/v2/CISCO-LWAPP-DOT11-MIB.my
@@ -1,7 +1,7 @@
 -- *******************************************************************
 -- CISCO-LWAPP-DOT11-MIB.my
 -- January 2007, Devesh Pujari, Srinath Candadai
---   
+--
 -- Copyright (c) 2007, 2009-2010-2014, 2017 by Cisco Systems Inc.
 -- All rights reserved.
 -- *******************************************************************
@@ -19,7 +19,8 @@ IMPORTS
     OBJECT-GROUP,
     NOTIFICATION-GROUP
         FROM SNMPv2-CONF
-    TruthValue
+    TruthValue,
+    DisplayString
         FROM SNMPv2-TC
     SnmpAdminString
         FROM SNMP-FRAMEWORK-MIB
@@ -109,7 +110,7 @@ ciscoLwappDot11MIB MODULE-IDENTITY
         An entity that contains an 802.11 medium access
         control ( MAC ) and physical layer ( PHY ) interface
         and provides access to the distribution services via
-        the wireless medium for associated clients.  
+        the wireless medium for associated clients.
 
         LWAPP APs encapsulate all the 802.11 frames in
         LWAPP frames and sends them to the controller to which
@@ -121,22 +122,22 @@ ciscoLwappDot11MIB MODULE-IDENTITY
         battery power of WLAN-enabled cell phones by providing
         a high rate packet on the order of tens of milliseconds
         such that these kind of phones can wake up and wait at
-        predefined intervals, to reduce battery power.  The 
+        predefined intervals, to reduce battery power.  The
         GPR packet is transmitted from the AP at a predefined
         time interval.
 
 
         Light Weight Access Point Protocol ( LWAPP )
 
-        This is a generic protocol that defines the 
+        This is a generic protocol that defines the
         communication between the Access Points and the
-        Central Controller. 
+        Central Controller.
 
         Mobile Node ( MN )
 
         A roaming 802.11 wireless device in a wireless
-        network associated with an access point. Mobile Node 
-        and client are used interchangeably. 
+        network associated with an access point. Mobile Node
+        and client are used interchangeably.
 
         TU
 
@@ -145,7 +146,7 @@ ciscoLwappDot11MIB MODULE-IDENTITY
         802.11n
 
         802.11n builds upon previous 802.11 standards by
-        adding MIMO (multiple-input multiple-output). 
+        adding MIMO (multiple-input multiple-output).
         MIMO uses multiple transmitter and receiver antennas
         to allow for increased data throughput through spatial
         multiplexing and increased range .
@@ -172,49 +173,49 @@ ciscoLwappDot11MIB MODULE-IDENTITY
 
         This is a value that determines the modulation, coding
         and number of spatial channels.  Each scheme specifies
-        the modulation technique, coding rate , number of 
+        the modulation technique, coding rate , number of
         spatial streams  etc and the corresponding data rate.
 
         Guard Interval
 
-        Guard intervals are used to ensure that distinct 
-        transmissions do not interfere with one another. 
-        The purpose of the guard interval is to introduce 
-        immunity to propagation delays, echoes and 
-        reflections, to which digital data is normally 
+        Guard intervals are used to ensure that distinct
+        transmissions do not interfere with one another.
+        The purpose of the guard interval is to introduce
+        immunity to propagation delays, echoes and
+        reflections, to which digital data is normally
         very sensitive.
 
         Media Access Control ( MAC )
 
-        The Media Access Control Layer is one of two sublayers 
-        that make up the Data Link Layer.  The MAC layer is 
-        responsible for moving data packets to and from one 
-        Network Interface Card (NIC) to another across a shared 
+        The Media Access Control Layer is one of two sublayers
+        that make up the Data Link Layer.  The MAC layer is
+        responsible for moving data packets to and from one
+        Network Interface Card (NIC) to another across a shared
         channel.
 
         Suppression Table
 
-        When the Band Select feature is ON, AP suppresses 
-        the probe response to mobile stations on 2.4 GHz. AP 
-        suppresses probe response to new mobile stattions for 
-        all SSIDs that are being Band Select enabled. 
-        Suppressed mobile station and corresponding suprression 
-        counts are recorded in a table known as Suppression 
+        When the Band Select feature is ON, AP suppresses
+        the probe response to mobile stations on 2.4 GHz. AP
+        suppresses probe response to new mobile stattions for
+        all SSIDs that are being Band Select enabled.
+        Suppressed mobile station and corresponding suprression
+        counts are recorded in a table known as Suppression
         Table which is stored in internal database of controller.
-        Entries of this table aged-out to make place for new 
+        Entries of this table aged-out to make place for new
         entries.
 
         Dual Band Table
 
-        When AP sees probe request from any mobile station in 
+        When AP sees probe request from any mobile station in
         both 2.4GHz and 5GHz band AP will know that mobile station
         is capable of operating on both band. Dual band capable
-        mobile stations are recorded in a table known as dual band 
+        mobile stations are recorded in a table known as dual band
         table which is stored in internal database of controller.
         This record are kept to make sure 5GHz capable mobile station
         should join 5GHz band only.
         Entries in the table will be age out to make space for
-        new entries. 
+        new entries.
         The AP will not respond to the dual band mobile station's
         2.4GHz probe until is removed from the dual band
         table. AP fills the dual band table in the
@@ -229,8 +230,8 @@ ciscoLwappDot11MIB MODULE-IDENTITY
         RSSI
 
         Received Signal Strength Indication (RSSI), the IEEE 802.11
-        standard defines a mechanism by which RF energy is to be 
-        measured by the circuitry on a wireless NIC. Its value is 
+        standard defines a mechanism by which RF energy is to be
+        measured by the circuitry on a wireless NIC. Its value is
         measured in dBm and ranges from -128 to 0.
 
 
@@ -283,7 +284,7 @@ ciscoLwappDot11MIB MODULE-IDENTITY
         ciscoLwappDot11MIBConfigGroup
         ciscoLwappDot11MIBNotifsGroup
         ciscoLwappDot11MIBStatusGroup
-        Added ciscoLwappDot11MIBComplianceRev1 in 
+        Added ciscoLwappDot11MIBComplianceRev1 in
         OBJECT-COMPLIANCE."
     REVISION        "200701040000Z"
     DESCRIPTION
@@ -311,13 +312,13 @@ cldStatus  OBJECT IDENTIFIER
 -- ********************************************************************
 
 cldHtMacOperationsTable OBJECT-TYPE
-    SYNTAX          SEQUENCE OF CldHtMacOperationsEntry 
+    SYNTAX          SEQUENCE OF CldHtMacOperationsEntry
     MAX-ACCESS      not-accessible
     STATUS          current
     DESCRIPTION
         "This table represents the operational parameters
-        at the MAC layer for the 802.11n networks managed 
-        through the controller. An agent adds an entry to 
+        at the MAC layer for the 802.11n networks managed
+        through the controller. An agent adds an entry to
         this table for every 802.11n band on startup."
     ::= { cldConfig 1 }
 
@@ -328,7 +329,7 @@ cldHtMacOperationsEntry OBJECT-TYPE
     DESCRIPTION
         "Each entry represents the operational parameters
         at the MAC layer for 802.11n networks."
-    INDEX           { cldHtDot11nBand } 
+    INDEX           { cldHtDot11nBand }
     ::= { cldHtMacOperationsTable 1 }
 
 CldHtMacOperationsEntry ::= SEQUENCE {
@@ -347,7 +348,7 @@ cldHtDot11nBand OBJECT-TYPE
     STATUS          current
     DESCRIPTION
         "This object identifies the 802.11n band the parameters
-        correspond to." 
+        correspond to."
     ::= { cldHtMacOperationsEntry 1 }
 
 cldHtDot11nChannelBandwidth OBJECT-TYPE
@@ -357,7 +358,7 @@ cldHtDot11nChannelBandwidth OBJECT-TYPE
     DESCRIPTION
         "This object represents the channel bandwidth for
         the 802.11n networks in the particular band."
-    DEFVAL          { twenty } 
+    DEFVAL          { twenty }
     ::= { cldHtMacOperationsEntry 2 }
 
 cldHtDot11nRifsEnable OBJECT-TYPE
@@ -366,10 +367,10 @@ cldHtDot11nRifsEnable OBJECT-TYPE
     STATUS          current
     DESCRIPTION
         "This object is used to indicate the support for RIFS on
-        this band. A value of 'true' indicates that RIFS is 
-        enabled for this band. A value of 'false' indicates that 
+        this band. A value of 'true' indicates that RIFS is
+        enabled for this band. A value of 'false' indicates that
         RIFS is disabled for this band."
-    DEFVAL          { true } 
+    DEFVAL          { true }
     ::= { cldHtMacOperationsEntry 3 }
 
 cldHtDot11nAmsduEnable OBJECT-TYPE
@@ -378,10 +379,10 @@ cldHtDot11nAmsduEnable OBJECT-TYPE
     STATUS          current
     DESCRIPTION
         "This object is used to indicate the support for 802.11n
-        A-MSDU on this band. A value of 'true' indicates that 
-        A-MSDU is enabled for this band. A value of 'false' 
+        A-MSDU on this band. A value of 'true' indicates that
+        A-MSDU is enabled for this band. A value of 'false'
         indicates that A-MSDU is disabled for this band."
-    DEFVAL          { true } 
+    DEFVAL          { true }
     ::= { cldHtMacOperationsEntry 4 }
 
 cldHtDot11nAmpduEnable OBJECT-TYPE
@@ -390,10 +391,10 @@ cldHtDot11nAmpduEnable OBJECT-TYPE
     STATUS          current
     DESCRIPTION
         "This object is used to indicate the support for 802.11n
-        A-MPDU on this band. A value of 'true' indicates that 
-        A-MPDU is enabled for this band. A value of 'false' 
+        A-MPDU on this band. A value of 'true' indicates that
+        A-MPDU is enabled for this band. A value of 'false'
         indicates that A-MPDU is disabled for this band."
-    DEFVAL          { true } 
+    DEFVAL          { true }
     ::= { cldHtMacOperationsEntry 5 }
 
 cldHtDot11nGuardIntervalEnable OBJECT-TYPE
@@ -402,11 +403,11 @@ cldHtDot11nGuardIntervalEnable OBJECT-TYPE
     STATUS          current
     DESCRIPTION
         "This object is used to indicate the support for 802.11n
-        guard interval on this band. A value of 'true' indicates 
-        that guard interval is enabled for this band. A value of 
+        guard interval on this band. A value of 'true' indicates
+        that guard interval is enabled for this band. A value of
         'false' indicates that guard interval is disabled for
         this band."
-    DEFVAL          { true } 
+    DEFVAL          { true }
     ::= { cldHtMacOperationsEntry 6 }
 
 cldHtDot11nEnable OBJECT-TYPE
@@ -415,12 +416,12 @@ cldHtDot11nEnable OBJECT-TYPE
     STATUS          current
     DESCRIPTION
         "This object is used to enable the controller to manage
-        802.11n networks. A value of 'true' means the 802.11n 
+        802.11n networks. A value of 'true' means the 802.11n
         network is enabled. A value of 'false' means the 802.11n
         network is disabled."
-    DEFVAL          { true } 
+    DEFVAL          { true }
     ::= { cldHtMacOperationsEntry 7 }
- 
+
 
 
 -- ********************************************************************
@@ -434,8 +435,8 @@ cldMultipleCountryCode OBJECT-TYPE
     DESCRIPTION
         "This object represents the countries in which the
         controller is operating. Multiple countries can be set.
-        The countries are displayed using a 3 letter code 
-        for each country separated by a comma." 
+        The countries are displayed using a 3 letter code
+        for each country separated by a comma."
     ::= { cldConfig 2 }
 
 cldRegulatoryDomain OBJECT-TYPE
@@ -444,16 +445,16 @@ cldRegulatoryDomain OBJECT-TYPE
     STATUS          current
     DESCRIPTION
         "This object represents the regulatory domains for
-        the controller.Regulatory domain specifies allowed 
-        802.11 chanels.Multiple domains can be listed seperated 
-        with spaces." 
+        the controller.Regulatory domain specifies allowed
+        802.11 chanels.Multiple domains can be listed seperated
+        with spaces."
     ::= { cldConfig 3 }
 -- ********************************************************************
 -- *    MCS table
 -- ********************************************************************
 
 cld11nMcsTable OBJECT-TYPE
-    SYNTAX          SEQUENCE OF Cld11nMcsEntry 
+    SYNTAX          SEQUENCE OF Cld11nMcsEntry
     MAX-ACCESS      not-accessible
     STATUS          current
     DESCRIPTION
@@ -467,7 +468,7 @@ cld11nMcsEntry OBJECT-TYPE
     STATUS          current
     DESCRIPTION
         "An entry in this table represents a MCS data rate for 802.11n
-        band which is uniquely identified by cld11nMcsDataRateIndex 
+        band which is uniquely identified by cld11nMcsDataRateIndex
         and cld11nMcsBand.
         All entries are added by the controller at startup.
         cld11nMcsDataRate are calculated by different permutation of
@@ -477,7 +478,7 @@ cld11nMcsEntry OBJECT-TYPE
     INDEX           {
                         cld11nMcsBand,
                         cld11nMcsDataRateIndex
-                    } 
+                    }
     ::= { cld11nMcsTable 1 }
 
 Cld11nMcsEntry ::= SEQUENCE {
@@ -487,7 +488,7 @@ Cld11nMcsEntry ::= SEQUENCE {
         cld11nMcsSupportEnable TruthValue,
         cld11nMcsChannelWidth  Unsigned32,
         cld11nMcsGuardInterval Unsigned32,
-        cld11nMcsModulation    OCTET STRING
+        cld11nMcsModulation    DisplayString
 }
 
 cld11nMcsBand OBJECT-TYPE
@@ -496,7 +497,7 @@ cld11nMcsBand OBJECT-TYPE
     STATUS          current
     DESCRIPTION
         "This object identifies the 802.11n band to which
-        the data rate is applicable." 
+        the data rate is applicable."
     ::= { cld11nMcsEntry 1 }
 
 cld11nMcsDataRateIndex OBJECT-TYPE
@@ -505,7 +506,7 @@ cld11nMcsDataRateIndex OBJECT-TYPE
     STATUS          current
     DESCRIPTION
         "This object uniquely identifies the data rate
-        for a particular band." 
+        for a particular band."
     ::= { cld11nMcsEntry 2 }
 
 cld11nMcsDataRate OBJECT-TYPE
@@ -514,7 +515,7 @@ cld11nMcsDataRate OBJECT-TYPE
     STATUS          current
     DESCRIPTION
         "The object represents the data rate value for this
-        band." 
+        band."
     ::= { cld11nMcsEntry 3 }
 
 cld11nMcsSupportEnable OBJECT-TYPE
@@ -523,10 +524,10 @@ cld11nMcsSupportEnable OBJECT-TYPE
     STATUS          current
     DESCRIPTION
         "This object is used to enable or disable the data
-        rate.  
-        A value of 'true' indicates, MCS support is enabled. 
+        rate.
+        A value of 'true' indicates, MCS support is enabled.
         A value of 'false' indicates, MCS support is disabled."
-    DEFVAL          { true } 
+    DEFVAL          { true }
     ::= { cld11nMcsEntry 4 }
 
 cld11nMcsChannelWidth OBJECT-TYPE
@@ -536,7 +537,7 @@ cld11nMcsChannelWidth OBJECT-TYPE
     STATUS          current
     DESCRIPTION
         "The object represents the channel width for which
-        cld11nMcsDataRate has been calculated." 
+        cld11nMcsDataRate has been calculated."
     ::= { cld11nMcsEntry 5 }
 
 cld11nMcsGuardInterval OBJECT-TYPE
@@ -546,18 +547,18 @@ cld11nMcsGuardInterval OBJECT-TYPE
     STATUS          current
     DESCRIPTION
         "The object represents the guard interval for which
-        cld11nMcsDataRate has been calculated." 
+        cld11nMcsDataRate has been calculated."
     ::= { cld11nMcsEntry 6 }
 
 cld11nMcsModulation OBJECT-TYPE
-    SYNTAX          OCTET STRING (SIZE  (1..255))
+    SYNTAX          DisplayString (SIZE  (1..255))
     MAX-ACCESS      read-only
     STATUS          current
     DESCRIPTION
         "The object represents the modulation used to calculate
-        cld11nMcsDataRate." 
+        cld11nMcsDataRate."
     ::= { cld11nMcsEntry 7 }
- 
+
 
 -- *****************************
 -- LOAD BALANCING CONFIGURATION
@@ -576,14 +577,14 @@ cldVhtDot11acEnable OBJECT-TYPE
         802.11ac networks. A value of 'true' means the 802.11ac
         network is enabled. A value of 'false' means the 802.11ac
         network is disabled."
-    DEFVAL          { true } 
+    DEFVAL          { true }
     ::= { cld11acConfig 1 }
 -- ********************************************************************
 -- *    MCS table
 -- ********************************************************************
 
 cld11acMcsTable OBJECT-TYPE
-    SYNTAX          SEQUENCE OF Cld11acMcsEntry 
+    SYNTAX          SEQUENCE OF Cld11acMcsEntry
     MAX-ACCESS      not-accessible
     STATUS          current
     DESCRIPTION
@@ -597,13 +598,13 @@ cld11acMcsEntry OBJECT-TYPE
     STATUS          current
     DESCRIPTION
         "An entry in this table represents the additional  MCS data
-        rate for 802.11ac band which is uniquely identifies by 
+        rate for 802.11ac band which is uniquely identifies by
         cld11acMcsSpatialStreamIndex and cld11acMcsDataRateIndex.
         All entries are added by the controller at startup."
     INDEX           {
                         cld11acMcsSpatialStreamIndex,
                         cld11acMcsDataRateIndex
-                    } 
+                    }
     ::= { cld11acMcsTable 1 }
 
 Cld11acMcsEntry ::= SEQUENCE {
@@ -619,7 +620,7 @@ cld11acMcsSpatialStreamIndex OBJECT-TYPE
     DESCRIPTION
         "The object represents the spatial stream number
         related to MCS data rate settings on the 802.11ac
-        band." 
+        band."
     ::= { cld11acMcsEntry 1 }
 
 cld11acMcsDataRateIndex OBJECT-TYPE
@@ -628,7 +629,7 @@ cld11acMcsDataRateIndex OBJECT-TYPE
     STATUS          current
     DESCRIPTION
         "This object uniquely identifies the data rate
-        for a particular band." 
+        for a particular band."
     ::= { cld11acMcsEntry 2 }
 
 cld11acMcsSupportEnable OBJECT-TYPE
@@ -637,12 +638,12 @@ cld11acMcsSupportEnable OBJECT-TYPE
     STATUS          current
     DESCRIPTION
         "This object is used to enable or disable the data
-        rate.  
-        A value of 'true' indicates the MCS support is enabled. 
+        rate.
+        A value of 'true' indicates the MCS support is enabled.
         A value of 'false' indicates the MCS support is disabled."
-    DEFVAL          { true } 
+    DEFVAL          { true }
     ::= { cld11acMcsEntry 3 }
- 
+
 
 -- *****************************
 -- LOAD BALANCING CONFIGURATION
@@ -671,10 +672,10 @@ cldLoadBalancingEnable OBJECT-TYPE
         will be sent to mobile station from AP.
 
         From 7.0 release onwards Load Balancing cannot be configured
-        globally. cldLoadBalancingEnable will be read-only attribute 
+        globally. cldLoadBalancingEnable will be read-only attribute
         and it will always be enable.
         Use cLWlanLoadBalancingEnable to configure it per WLAN."
-    DEFVAL          { enable } 
+    DEFVAL          { enable }
     ::= { cldLoadBalancing 1 }
 
 cldLoadBalancingWindowSize OBJECT-TYPE
@@ -683,23 +684,23 @@ cldLoadBalancingWindowSize OBJECT-TYPE
     STATUS          current
     DESCRIPTION
         "This object specifies the threshold for the difference
-        between number of mobile station a given AP can have and 
-        mobile station count of the AP having minimum number of 
+        between number of mobile station a given AP can have and
+        mobile station count of the AP having minimum number of
         mobile station in the setup.
 
         For example, suppose in a network setup there are three
         APs are connected to Controller, say AP1, AP2 and AP3.
-        AP1 have 2 mobile stations, AP2 have 3 and AP3 have 4 
+        AP1 have 2 mobile stations, AP2 have 3 and AP3 have 4
         mobile stations.
-        Here AP1 have minimum number of mobile stations i.e. 2. 
+        Here AP1 have minimum number of mobile stations i.e. 2.
         Suppose window size is configured as 2. So every AP can have
-        2+2=4 mobile stations. So every 5th mobile station will be 
+        2+2=4 mobile stations. So every 5th mobile station will be
         load balanced.
 
         So, if any mobile station tries to join AP3, denial will be sent
         from AP3. For a mobile station denial will be send only for
         cldLoadBalancingDenialCount number of time."
-    DEFVAL          { 5 } 
+    DEFVAL          { 5 }
     ::= { cldLoadBalancing 2 }
 
 cldLoadBalancingDenialCount OBJECT-TYPE
@@ -723,7 +724,7 @@ cldLoadBalancingDenialCount OBJECT-TYPE
 
         Association attempts on an AP from any mobile station before
         associating any AP is called as sequence of assoociation."
-    DEFVAL          { 3 } 
+    DEFVAL          { 3 }
     ::= { cldLoadBalancing 3 }
 
 cldLoadBalancingTrafficThreshold OBJECT-TYPE
@@ -732,14 +733,14 @@ cldLoadBalancingTrafficThreshold OBJECT-TYPE
     MAX-ACCESS      read-write
     STATUS          current
     DESCRIPTION
-        "This object specifies the traffic threshold that  
-        triggers 'uplinkUsage' based load balancing. When the 
-        uplink usage of an AP is above this threshold, this AP 
-        is valid for 'uplinkUsage' based load balancing, if 
-        load balance is enabled by cLWlanLoadBalancingEnable 
+        "This object specifies the traffic threshold that
+        triggers 'uplinkUsage' based load balancing. When the
+        uplink usage of an AP is above this threshold, this AP
+        is valid for 'uplinkUsage' based load balancing, if
+        load balance is enabled by cLWlanLoadBalancingEnable
         and 'uplinkUsage' mode is chosen by cLWlanLoadBalancingMode
         at that WLAN."
-    DEFVAL          { 50 } 
+    DEFVAL          { 50 }
     ::= { cldLoadBalancing 4 }
 -- **************************
 -- BAND SELECT CONFIGURATION
@@ -767,10 +768,10 @@ cldBandSelectEnable OBJECT-TYPE
         only on 2.4 GHz radio.
 
         From 7.0 release onwards Band Select cannot be configured
-        globally. cldBandSelectEnable will be read-only attribute 
+        globally. cldBandSelectEnable will be read-only attribute
         and it will always be enable.
         Use cLWlanBandSelectEnable to configure it per WLAN."
-    DEFVAL          { enable } 
+    DEFVAL          { enable }
     ::= { cldBandSelect 1 }
 
 cldBandSelectCycleCount OBJECT-TYPE
@@ -780,14 +781,14 @@ cldBandSelectCycleCount OBJECT-TYPE
     DESCRIPTION
         "This object specifies band select probe mobile station count.
         Suppressed mobile stations and corresponding suppression count
-        are recorded in a suppression table which is stored in 
-        internal database of controllers. AP will suppress 
+        are recorded in a suppression table which is stored in
+        internal database of controllers. AP will suppress
         probe request of a mobile station till value of cycle count.
         AP will increment cycle counter when a mobile station scan
         a channel on 2.4GHz band after any time greater than
         cldBandSelectCycleThreshold.
-        When a mobile station cycle count is reached and if mobile 
-        station still sends a probe request then AP will respond 
+        When a mobile station cycle count is reached and if mobile
+        station still sends a probe request then AP will respond
         to it by probe response.
 
         For example, we assume at minimum, a mobile station
@@ -798,7 +799,7 @@ cldBandSelectCycleCount OBJECT-TYPE
         probe and the latest one. AP only increments the count if
         the difference of time between the latest and the last
         probe is more than 50 milliseconds."
-    DEFVAL          { 3 } 
+    DEFVAL          { 3 }
     ::= { cldBandSelect 2 }
 
 cldBandSelectCycleThreshold OBJECT-TYPE
@@ -808,8 +809,8 @@ cldBandSelectCycleThreshold OBJECT-TYPE
     STATUS          current
     DESCRIPTION
         "This object specifies time threshold for a new scanning
-        mobile station period. Mobile station cycle counter will 
-        increment only if mobile station scans same channel any 
+        mobile station period. Mobile station cycle counter will
+        increment only if mobile station scans same channel any
         time after cldBandSelectCycleThreshold is passed.
 
         For example, if a mobile station is scanning a channel after
@@ -818,7 +819,7 @@ cldBandSelectCycleThreshold OBJECT-TYPE
         after 300 seconds. But if mobile station is scanning after every
         250 milliseconds, then mobile station count will be incremented
         after 250 milliseconds."
-    DEFVAL          { 200 } 
+    DEFVAL          { 200 }
     ::= { cldBandSelect 3 }
 
 cldBandSelectAgeOutSuppression OBJECT-TYPE
@@ -837,7 +838,7 @@ cldBandSelectAgeOutSuppression OBJECT-TYPE
         had responded already. If there as no empty slot in
         the table, AP has to respond to all the new mobile station
         until space starts getting available."
-    DEFVAL          { 20 } 
+    DEFVAL          { 20 }
     ::= { cldBandSelect 4 }
 
 cldBandSelectAgeOutDualBand OBJECT-TYPE
@@ -851,7 +852,7 @@ cldBandSelectAgeOutDualBand OBJECT-TYPE
         When AP sees probe request from any mobile station in both
         2.4GHz and 5GHz band AP will know that mobile station is
         capable of operating on both band. Dual band capable
-        mobile stations are recorded in a dual band table which 
+        mobile stations are recorded in a dual band table which
         is stored in internal database of controllers.
         This record are kept to make sure 5GHz capable mobile station
         should join 5GHz band only.
@@ -868,7 +869,7 @@ cldBandSelectAgeOutDualBand OBJECT-TYPE
            probes.
         3) mobile station with just 5GHz probe detected and 5GHz
            association."
-    DEFVAL          { 60 } 
+    DEFVAL          { 60 }
     ::= { cldBandSelect 5 }
 
 cldBandSelectClientRssi OBJECT-TYPE
@@ -883,19 +884,19 @@ cldBandSelectClientRssi OBJECT-TYPE
         This configuration filter out far away mobile stations with
         low signal strength. This will limit the number of
         mobile stations on the table to a reasonable amount."
-    DEFVAL          { -80 } 
+    DEFVAL          { -80 }
     ::= { cldBandSelect 6 }
 -- ********************************************************************
 -- Channel table for all  Country codes
 -- ********************************************************************
 
 cldCountryTable OBJECT-TYPE
-    SYNTAX          SEQUENCE OF CldCountryEntry 
+    SYNTAX          SEQUENCE OF CldCountryEntry
     MAX-ACCESS      not-accessible
     STATUS          current
     DESCRIPTION
         "This table provides the list of countries and the
-        network parameters specific to the respective 
+        network parameters specific to the respective
         countries. It shows the 802.11 networks, managed by
         this controller, where controller can operate."
     ::= { cldStatus 1 }
@@ -911,7 +912,7 @@ cldCountryEntry OBJECT-TYPE
         Each entry also displays the corresponding channels
         supported by the cldCountryCode.
         All entries are added by the controller at startup."
-    INDEX           { cldCountryCode } 
+    INDEX           { cldCountryCode }
     ::= { cldCountryTable 1 }
 
 CldCountryEntry ::= SEQUENCE {
@@ -929,7 +930,7 @@ cldCountryCode OBJECT-TYPE
     STATUS          current
     DESCRIPTION
         "This object identifies the country for this entry.
-        The 3 letter country code for this index." 
+        The 3 letter country code for this index."
     ::= { cldCountryEntry 1 }
 
 cldCountryName OBJECT-TYPE
@@ -938,7 +939,7 @@ cldCountryName OBJECT-TYPE
     STATUS          current
     DESCRIPTION
         "This object represents the name of the country for
-        this entry." 
+        this entry."
     ::= { cldCountryEntry 2 }
 
 cldCountryDot11aChannels OBJECT-TYPE
@@ -947,7 +948,7 @@ cldCountryDot11aChannels OBJECT-TYPE
     STATUS          current
     DESCRIPTION
         "This object represents the 802.11a channels
-        separated by comma." 
+        separated by comma."
     ::= { cldCountryEntry 3 }
 
 cldCountryDot11bChannels OBJECT-TYPE
@@ -956,7 +957,7 @@ cldCountryDot11bChannels OBJECT-TYPE
     STATUS          current
     DESCRIPTION
         "This object represents the 802.11b channels
-        separated by comma." 
+        separated by comma."
     ::= { cldCountryEntry 4 }
 
 cldCountryDot11aDcaChannels OBJECT-TYPE
@@ -965,7 +966,7 @@ cldCountryDot11aDcaChannels OBJECT-TYPE
     STATUS          current
     DESCRIPTION
         "This object represents the 802.11a DCA channels
-        separated by comma." 
+        separated by comma."
     ::= { cldCountryEntry 5 }
 
 cldCountryDot11bDcaChannels OBJECT-TYPE
@@ -974,9 +975,9 @@ cldCountryDot11bDcaChannels OBJECT-TYPE
     STATUS          current
     DESCRIPTION
         "This object represents the 802.11b DCA channels
-        separated by comma." 
+        separated by comma."
     ::= { cldCountryEntry 6 }
- 
+
 
 
 -- ********************************************************************
@@ -995,9 +996,9 @@ cldCountryChangeNotifEnable OBJECT-TYPE
         ciscoLwappDot11CountryChangeNotif notification.
 
         A value of 'false' indicates that the agent doesn't
-        generate ciscoLwappDot11CountryChangeNotif 
+        generate ciscoLwappDot11CountryChangeNotif
         notification."
-    DEFVAL          { true } 
+    DEFVAL          { true }
     ::= { cldConfig 7 }
 
 -- ********************************************************************
@@ -1009,8 +1010,8 @@ ciscoLwappDot11CountryChangeNotif NOTIFICATION-TYPE
     STATUS          current
     DESCRIPTION
         "This notification is generated by the controller when the
-        country of operation of 802.11 networks is changed 
-        by the administrator. The new country code will be sent in 
+        country of operation of 802.11 networks is changed
+        by the administrator. The new country code will be sent in
         this notification."
    ::= { ciscoLwappDot11MIBNotifs 1 }
 -- ********************************************************************
