diff --git a/mibs/cisco/cat3750/v2/CISCO-MAC-NOTIFICATION-MIB.my b/mibs/cisco/cat3750/v2/CISCO-MAC-NOTIFICATION-MIB.my
index da2d4ad..fd661e7 100644
--- a/mibs/cisco/cat3750/v2/CISCO-MAC-NOTIFICATION-MIB.my
+++ b/mibs/cisco/cat3750/v2/CISCO-MAC-NOTIFICATION-MIB.my
@@ -1,5 +1,5 @@
 -- *****************************************************************
--- CISCO-MAC-NOTIFICATION-MIB.my:  Cisco MAC Notification MIB 
+-- CISCO-MAC-NOTIFICATION-MIB.my:  Cisco MAC Notification MIB
 --
 -- October 2001, Edward Pham
 -- February 2003, Padma Pullela
@@ -17,7 +17,7 @@ IMPORTS
                 FROM SNMPv2-SMI
         MODULE-COMPLIANCE, OBJECT-GROUP, NOTIFICATION-GROUP
                 FROM SNMPv2-CONF
-        TruthValue, TimeStamp, MacAddress, TEXTUAL-CONVENTION
+        TruthValue, TimeStamp, MacAddress, TEXTUAL-CONVENTION, DisplayString
                 FROM SNMPv2-TC
         ifIndex
                 FROM IF-MIB
@@ -49,17 +49,17 @@ ciscoMacNotificationMIB MODULE-IDENTITY
         "This MIB module is for configuration of the MAC notification
         feature. MAC notification is a mechanism to inform monitoring
         devices when there are MAC addresses learnt or removed from
-        the forwarding database of the monitored devices." 
+        the forwarding database of the monitored devices."
 
     REVISION   "200706110000Z"
     DESCRIPTION
-        "Fixed typo and  made changes to the description of 
-         cmnMACMoveObjects, cmnMACThresholdNotifEnabled and 
+        "Fixed typo and  made changes to the description of
+         cmnMACMoveObjects, cmnMACThresholdNotifEnabled and
          cmnMacThresholdExceedNotif."
 
     REVISION   "200303210000Z"
     DESCRIPTION
-        "Added cmnMACMoveObjects, cmnMACThresholdObjects." 
+        "Added cmnMACMoveObjects, cmnMACThresholdObjects."
 
     REVISION   "200110220000Z"
     DESCRIPTION
@@ -95,7 +95,7 @@ cmnGlobalFeatureEnabled  OBJECT-TYPE
     MAX-ACCESS    read-write
     STATUS        current
     DESCRIPTION
-        "Indicates whether the MAC notification feature is currently 
+        "Indicates whether the MAC notification feature is currently
         running in the device.
 
         Setting this object to false(2) disables the MAC notification
@@ -105,7 +105,7 @@ cmnGlobalFeatureEnabled  OBJECT-TYPE
         feature running in the device. If the feature is already
         running, setting to true(1) has no effect. Once the MAC
         notification is enabled, whether the feature is running at each
-        interface is controlled by the cmnIfConfigTable."   
+        interface is controlled by the cmnIfConfigTable."
     ::= { cmnGlobalObjects 1 }
 
 cmnNotificationInterval  OBJECT-TYPE
@@ -122,10 +122,10 @@ cmnNotificationInterval  OBJECT-TYPE
         cmnHistoryTable. If the value of cmnNotificationEnabled is
         false(2), the device will not send out the generated
         cmnMacChangedNotifications but it will archive these events
-        in the cmnHistoryTable.   
-        
-        If the value of this object is equal to 0, the device will 
-        generate cmnMacChangedNotifications and archive the MAC 
+        in the cmnHistoryTable.
+
+        If the value of this object is equal to 0, the device will
+        generate cmnMacChangedNotifications and archive the MAC
         change notification events in the cmnHistoryTable as soon as
         there is MAC address learnt or removed by the device.
 
@@ -136,21 +136,21 @@ cmnNotificationInterval  OBJECT-TYPE
     ::= { cmnGlobalObjects 2 }
 
 cmnMacAddressesLearnt OBJECT-TYPE
-    SYNTAX        Counter32 
+    SYNTAX        Counter32
     MAX-ACCESS    read-only
     STATUS        current
     DESCRIPTION
         "Indicates the number of MAC addresses learnt by the
-        device." 
+        device."
     ::= { cmnGlobalObjects 3 }
 
 cmnMacAddressesRemoved OBJECT-TYPE
-    SYNTAX        Counter32 
+    SYNTAX        Counter32
     MAX-ACCESS    read-only
     STATUS        current
     DESCRIPTION
         "Indicates the number of MAC addresses removed from the
-        forwarding database." 
+        forwarding database."
     ::= { cmnGlobalObjects 4 }
 
 cmnNotificationsEnabled OBJECT-TYPE
@@ -160,14 +160,14 @@ cmnNotificationsEnabled OBJECT-TYPE
      DESCRIPTION
          "Indicates whether cmnMacChangedNotification notifications
           will or will not be sent when there are MAC addresses
-          learnt or removed from the device's forwarding database. 
+          learnt or removed from the device's forwarding database.
           Disabling notifications does not prevent the MAC address
           info from being added to the cmnHistoryTable."
      DEFVAL { false }
      ::= { cmnGlobalObjects 5 }
 
 cmnNotificationsSent OBJECT-TYPE
-    SYNTAX         Counter32 
+    SYNTAX         Counter32
     MAX-ACCESS     read-only
     STATUS         current
     DESCRIPTION
@@ -194,9 +194,9 @@ cmnHistoryTable OBJECT-TYPE
     MAX-ACCESS not-accessible
     STATUS     current
     DESCRIPTION
-        "This table will archive the MAC change notification events 
+        "This table will archive the MAC change notification events
         generated by this device. The MAC change notification
-        events are archived here even if cmnMacChangesNotifications 
+        events are archived here even if cmnMacChangesNotifications
         are not actually sent."
     ::= { cmnGlobalObjects 8 }
 
@@ -216,7 +216,7 @@ CmnHistoryEntry ::=
         cmnHistIndex
                 Unsigned32,
         cmnHistMacChangedMsg
-                OCTET STRING,
+                DisplayString,
         cmnHistTimestamp
                 TimeStamp
     }
@@ -228,22 +228,22 @@ cmnHistIndex OBJECT-TYPE
     DESCRIPTION
         "An index that uniquely identifies a MAC change notification
         event previously generated by the device. This index starts at
-        1 and increases by one when a MAC change notification is 
+        1 and increases by one when a MAC change notification is
         generated.  When it reaches the maximum value, the agent wraps
         the value back to 1."
     ::= { cmnHistoryEntry 1 }
 
 cmnHistMacChangedMsg OBJECT-TYPE
-    SYNTAX        OCTET STRING (SIZE(1..254))
+    SYNTAX        DisplayString (SIZE(1..254))
     MAX-ACCESS    read-only
     STATUS        current
     DESCRIPTION
         "This object contains the information of a MAC change
         notification event. It consists of several tuples packed
         together in the format of '<tuple1><tuple2>...'.
-        
+
         Each tuple consist of 11 octets in the format of
-        '<operation><VLAN><MAC><dot1dBasePort>' where 
+        '<operation><VLAN><MAC><dot1dBasePort>' where
 
         <operation> is of size 1 octet and supports the following values
           0 - End of MIB object.
@@ -284,7 +284,7 @@ cmnIfConfigTable  OBJECT-TYPE
     STATUS        current
     DESCRIPTION
         "This table enables or disables the generation of notification
-        at each interface when MAC address is learnt or removed." 
+        at each interface when MAC address is learnt or removed."
     ::= { cmnInterfaceObjects 1 }
 
 cmnIfConfigEntry  OBJECT-TYPE
@@ -294,45 +294,45 @@ cmnIfConfigEntry  OBJECT-TYPE
     DESCRIPTION
         "Each entry contains the configuration for enabling the
         MAC notification at each interface that supports this
-        feature." 
+        feature."
     INDEX { ifIndex }
     ::= { cmnIfConfigTable 1 }
 
 CmnIfConfigEntry  ::= SEQUENCE {
-    cmnMacAddrLearntEnable    TruthValue, 
-    cmnMacAddrRemovedEnable   TruthValue 
+    cmnMacAddrLearntEnable    TruthValue,
+    cmnMacAddrRemovedEnable   TruthValue
 }
 
 cmnMacAddrLearntEnable OBJECT-TYPE
-    SYNTAX        TruthValue 
+    SYNTAX        TruthValue
     MAX-ACCESS    read-write
     STATUS        current
     DESCRIPTION
-        "Indicates whether this interface is enabled to send 
+        "Indicates whether this interface is enabled to send
         cmnMacChangedNotification when it learns a new MAC address. This
         variable has no effect when the value of cmnGlobalFeatureEnabled
         object is false(2).
 
         Setting this object to true(1) enables the sending of
-        cmnMacChangedNotification when this interface learns a 
+        cmnMacChangedNotification when this interface learns a
         new MAC address.
 
         Setting this object to false(2) disables the sending
         of cmnMacChangedNotification when this interface learns
-        a new MAC address."      
+        a new MAC address."
     DEFVAL { false }
     ::= { cmnIfConfigEntry 1 }
 
 cmnMacAddrRemovedEnable OBJECT-TYPE
-    SYNTAX        TruthValue 
+    SYNTAX        TruthValue
     MAX-ACCESS    read-write
     STATUS        current
     DESCRIPTION
-        "Indicates whether this interface is enabled to send 
+        "Indicates whether this interface is enabled to send
         cmnMacChangedNotification when a MAC address which it learnt
         previously is removed from the forwarding table. This variable
         has no effect when the value of cmnGlobalFeatureEnabled object
-        is false(2). 
+        is false(2).
 
         Setting this object to true(1) enables the sending of
         cmnMacChangedNotification when a MAC address which this
@@ -355,14 +355,14 @@ cmnMACMoveFeatureEnabled  OBJECT-TYPE
     MAX-ACCESS    read-write
     STATUS        current
     DESCRIPTION
-        "Specifies whether the MAC Move notification feature is 
+        "Specifies whether the MAC Move notification feature is
          currently running in the device.
 
-         Setting this object to false(2) disables the MAC Move 
+         Setting this object to false(2) disables the MAC Move
          notification feature globally.
 
-         Setting this object to true(1) will start the MAC Move 
-         notification feature running in the device."  
+         Setting this object to true(1) will start the MAC Move
+         notification feature running in the device."
     ::= { cmnMACMoveObjects 1 }
 
 cmnMACMoveNotificationsEnabled OBJECT-TYPE
@@ -376,8 +376,8 @@ cmnMACMoveNotificationsEnabled OBJECT-TYPE
 
           Setting this object to false(2) will not send the
           cmnMacMoveNotification notifications.
-   
-          Setting this object to true(1) will send the 
+
+          Setting this object to true(1) will send the
           cmnMacMoveNotification notifications."
     ::= { cmnMACMoveObjects 2 }
 
@@ -387,8 +387,8 @@ cmnMACMoveAddress OBJECT-TYPE
      STATUS          current
      DESCRIPTION
          "Indicates the MAC address that is moved between
-          cmnMACMoveFromPortId and cmnMACMoveToPortId on 
-          cmnMACMoveVlanNumber.  This object is instantiated only 
+          cmnMACMoveFromPortId and cmnMACMoveToPortId on
+          cmnMACMoveVlanNumber.  This object is instantiated only
           when cmnMACMoveFeatureEnabled value is set to true(1) and
           a MAC move is detected by the MAC move notification feature."
     ::= { cmnMACMoveObjects 3 }
@@ -399,21 +399,21 @@ cmnMACMoveVlanNumber OBJECT-TYPE
      STATUS          current
      DESCRIPTION
          "Indicates the VLAN on which the cmnMACMoveAddress is
-          moved from cmnMACMoveFromPortId to cmnMACMoveToPortId. 
+          moved from cmnMACMoveFromPortId to cmnMACMoveToPortId.
           This object is instantiated only when
           cmnMACMoveFeatureEnabled value is set to true(1) and a
           MAC move is detected by the MAC move notification feature."
     ::= { cmnMACMoveObjects 4 }
 
 cmnMACMoveFromPortId OBJECT-TYPE
-     SYNTAX          Integer32 (1..65535) 
+     SYNTAX          Integer32 (1..65535)
      MAX-ACCESS      read-only
      STATUS          current
      DESCRIPTION
-         "The value of dot1dBasePort for the bridge port from which 
-          the cmnMACMoveAddress is moved to cmnMACMoveToPortId on 
+         "The value of dot1dBasePort for the bridge port from which
+          the cmnMACMoveAddress is moved to cmnMACMoveToPortId on
           cmnMACMoveVlanNumber. This object is instantiated only when
-          cmnMACMoveFeatureEnabled value is set to true(1) and a 
+          cmnMACMoveFeatureEnabled value is set to true(1) and a
           MAC move is detected by the MAC move notification feature."
      REFERENCE
                "dot1dBasePort is defined in RFC1493."
@@ -424,10 +424,10 @@ cmnMACMoveToPortId OBJECT-TYPE
      MAX-ACCESS      read-only
      STATUS          current
      DESCRIPTION
-         "The value of dot1dBasePort for the bridge port to which 
-          the cmnMACMoveAddress is moved from cmnMACMoveFromPortId 
-          on cmnMACMoveVlanNumber.  This object is instantiated only 
-          when cmnMACMoveFeatureEnabled value is set to true(1) and  
+         "The value of dot1dBasePort for the bridge port to which
+          the cmnMACMoveAddress is moved from cmnMACMoveFromPortId
+          on cmnMACMoveVlanNumber.  This object is instantiated only
+          when cmnMACMoveFeatureEnabled value is set to true(1) and
           a MAC move is detected by the MAC move notification feature."
      REFERENCE
                "dot1dBasePort is defined in RFC1493."
@@ -438,10 +438,10 @@ cmnMACMoveTime OBJECT-TYPE
      MAX-ACCESS      read-only
      STATUS          current
      DESCRIPTION
-         "The value of sysUpTime when a cmnMACMoveAddress is moved 
-          between cmnMACMoveFromPortId and cmnMACMACMoveToPortId. 
-          This object is instantiated only when 
-          cmnMACMoveFeatureEnabled value is set to true(1) and  
+         "The value of sysUpTime when a cmnMACMoveAddress is moved
+          between cmnMACMoveFromPortId and cmnMACMACMoveToPortId.
+          This object is instantiated only when
+          cmnMACMoveFeatureEnabled value is set to true(1) and
           a MAC move is detected by the MAC move notification feature."
     ::= { cmnMACMoveObjects 7 }
 
@@ -454,14 +454,14 @@ cmnMACThresholdFeatureEnabled  OBJECT-TYPE
     MAX-ACCESS    read-write
     STATUS        current
     DESCRIPTION
-        "Specifies whether the MAC Threshold notification feature is 
+        "Specifies whether the MAC Threshold notification feature is
          currently running in the device.
 
-         Setting this object to false(2) disables the MAC Threshold 
+         Setting this object to false(2) disables the MAC Threshold
          notification feature globally.
- 
-         Setting this object to true(1) will start the MAC Threshold 
-         notification feature running in the device." 
+
+         Setting this object to true(1) will start the MAC Threshold
+         notification feature running in the device."
     ::= { cmnMACThresholdObjects 1 }
 
 cmnMACThresholdLimit  OBJECT-TYPE
@@ -469,7 +469,7 @@ cmnMACThresholdLimit  OBJECT-TYPE
     MAX-ACCESS    read-write
     STATUS        current
     DESCRIPTION
-         "Indicate the threshold limit of the forwarding table 
+         "Indicate the threshold limit of the forwarding table
           utilization."
     ::= { cmnMACThresholdObjects 2 }
 
@@ -479,8 +479,8 @@ cmnMACThresholdInterval  OBJECT-TYPE
     MAX-ACCESS    read-write
     STATUS        current
     DESCRIPTION
-        "Interval at which forwarding table utilization is compared 
-         against cmnMACThresholdLimit." 
+        "Interval at which forwarding table utilization is compared
+         against cmnMACThresholdLimit."
     ::= { cmnMACThresholdObjects 3 }
 
 cmnMACThresholdNotifEnabled OBJECT-TYPE
@@ -488,10 +488,10 @@ cmnMACThresholdNotifEnabled OBJECT-TYPE
      MAX-ACCESS      read-write
      STATUS          current
      DESCRIPTION
-         "Specifies whether cmnMacThresholdExceedNotif 
-          notifications will or will not be sent when the forwarding 
-          table utilization exceeds or equals to cmnMACThresholdLimit 
-          value. cmnMacThresholdExceedNotif notification is not sent 
+         "Specifies whether cmnMacThresholdExceedNotif
+          notifications will or will not be sent when the forwarding
+          table utilization exceeds or equals to cmnMACThresholdLimit
+          value. cmnMacThresholdExceedNotif notification is not sent
           when cmnMACThresholdLimit is set to zero."
     ::= { cmnMACThresholdObjects 4 }
 
@@ -500,12 +500,12 @@ cmnUtilizationTable OBJECT-TYPE
     MAX-ACCESS not-accessible
     STATUS     current
     DESCRIPTION
-        "cmnUtilizationTable  specifies the forwarding table 
-         utilization information. This table is instantiated only 
+        "cmnUtilizationTable  specifies the forwarding table
+         utilization information. This table is instantiated only
          when cmnMACThresholdFeatureEnabled value is set to true(1).
          Entries in this table are updated at the end of every
-         cmnMACThresholdInterval." 
-          
+         cmnMACThresholdInterval."
+
     ::= { cmnMACThresholdObjects 5 }
 
 cmnUtilizationEntry OBJECT-TYPE
@@ -513,9 +513,9 @@ cmnUtilizationEntry OBJECT-TYPE
     MAX-ACCESS not-accessible
     STATUS     current
     DESCRIPTION
-        "A conceptual row containing forwarding table utilization 
-         maintained by switching engine (identified by 
-         entPhysicalIndex). Each switching engine managed by this 
+        "A conceptual row containing forwarding table utilization
+         maintained by switching engine (identified by
+         entPhysicalIndex). Each switching engine managed by this
          MIB module can have at least one entry in this table."
     INDEX   { entPhysicalIndex }
     ::= { cmnUtilizationTable 1 }
@@ -531,12 +531,12 @@ CmnUtilizationEntry ::=
     }
 
 cmnUtilizationEntries OBJECT-TYPE
-    SYNTAX          Unsigned32 
+    SYNTAX          Unsigned32
     MAX-ACCESS      read-only
     STATUS          current
     DESCRIPTION
-        "Indicates the number of entries present in the forwarding 
-         table for the given entPhysicalIndex calculated at the 
+        "Indicates the number of entries present in the forwarding
+         table for the given entPhysicalIndex calculated at the
          end of cmnMACThresholdInterval."
     ::= { cmnUtilizationEntry 1 }
 
@@ -546,7 +546,7 @@ cmnUtilizationUtilization OBJECT-TYPE
     STATUS          current
     DESCRIPTION
         "Indicates the utilization of the forwarding table for the given
-         entPhysicalIndex calculated at the end of 
+         entPhysicalIndex calculated at the end of
          cmnMACThresholdInterval."
     ::= { cmnUtilizationEntry 2 }
 
@@ -555,13 +555,13 @@ cmnUtilizationTimeStamp OBJECT-TYPE
     MAX-ACCESS      read-only
     STATUS          current
     DESCRIPTION
-        "Indicates the sysUptime at which the cmnUtilizationUtilization 
+        "Indicates the sysUptime at which the cmnUtilizationUtilization
          is updated."
     ::= { cmnUtilizationEntry 3 }
 
 --
 --
--- Notification 
+-- Notification
 --
 
 cmnMIBNotificationPrefix OBJECT IDENTIFIER
@@ -571,7 +571,7 @@ cmnMIBNotifications
     OBJECT IDENTIFIER ::= { cmnMIBNotificationPrefix 0 }
 
 cmnMacChangedNotification NOTIFICATION-TYPE
-    OBJECTS { cmnHistMacChangedMsg, cmnHistTimestamp } 
+    OBJECTS { cmnHistMacChangedMsg, cmnHistTimestamp }
     STATUS  current
     DESCRIPTION
         "This notification is generated when there is enough MAC
@@ -579,42 +579,42 @@ cmnMacChangedNotification NOTIFICATION-TYPE
         message. This notification is also generated when there
         is at least one MAC address changed or removed and the amount
         of time elapsed from the previous notification is greater
-        than the maximum wait time denoted by 
-        cmnNotificationInterval object. 
+        than the maximum wait time denoted by
+        cmnNotificationInterval object.
 
         If there are more MAC addresses information than can fit into
         one cmmHistTrapContent object, then multiple notifications
-        will be generated." 
-    ::= { cmnMIBNotifications 1 } 
+        will be generated."
+    ::= { cmnMIBNotifications 1 }
 
 cmnMacMoveNotification NOTIFICATION-TYPE
-    OBJECTS { 
-              cmnMACMoveAddress, 
-              cmnMACMoveVlanNumber, 
-              cmnMACMoveFromPortId, 
+    OBJECTS {
+              cmnMACMoveAddress,
+              cmnMACMoveVlanNumber,
+              cmnMACMoveFromPortId,
               cmnMACMoveToPortId,
               cmnMACMoveTime
             }
     STATUS  current
     DESCRIPTION
-        "cmnMacMoveNotification is generated when a MAC address is 
+        "cmnMacMoveNotification is generated when a MAC address is
          moved between two interfaces."
-    ::= { cmnMIBNotifications 2 } 
+    ::= { cmnMIBNotifications 2 }
 
 cmnMacThresholdExceedNotif  NOTIFICATION-TYPE
     OBJECTS  {
-              cmnUtilizationUtilization, 
+              cmnUtilizationUtilization,
               cmnMACThresholdLimit,
-              cmnUtilizationTimeStamp 
+              cmnUtilizationTimeStamp
              }
     STATUS  current
     DESCRIPTION
-         "cmnMacThresholdExceedNotif is sent when 
-          cmnUtilizationUtilization exceeds or equals to the 
+         "cmnMacThresholdExceedNotif is sent when
+          cmnUtilizationUtilization exceeds or equals to the
           cmnMACThresholdLimit for a given entPhysicalIndex.
-          cmnMacThresholdExceedNotif is not sent when 
+          cmnMacThresholdExceedNotif is not sent when
           cmnMACThresholdLimit is set to zero"
-    ::= { cmnMIBNotifications 3 } 
+    ::= { cmnMIBNotifications 3 }
 
 --
 -- Conformance
@@ -648,7 +648,7 @@ cmnMIBComplianceVer1 MODULE-COMPLIANCE
      MODULE
      MANDATORY-GROUPS { cmnGlobalGroup,
                         cmnInterfaceGroup,
-                        cmnNotificationGroup 
+                        cmnNotificationGroup
                       }
 
      GROUP        cmnMACMoveGroup
@@ -661,7 +661,7 @@ cmnMIBComplianceVer1 MODULE-COMPLIANCE
 
      GROUP        cmnMACMoveNotifGroup
      DESCRIPTION  "This group is mandatory if the managed system
-                   supports Notifications for MAC Move notification 
+                   supports Notifications for MAC Move notification
                    feature."
 
      GROUP        cmnMACThresholdNotifGroup
@@ -691,7 +691,7 @@ cmnGlobalGroup OBJECT-GROUP
     STATUS current
     DESCRIPTION
         "A collection of objects providing the global configuration
-        and information for MAC notification." 
+        and information for MAC notification."
     ::= { cmnMIBGroups 1 }
 
 cmnInterfaceGroup OBJECT-GROUP
@@ -702,7 +702,7 @@ cmnInterfaceGroup OBJECT-GROUP
     STATUS current
     DESCRIPTION
         "A collection of objects providing the configuration information
-        for MAC notification at each interface." 
+        for MAC notification at each interface."
     ::= { cmnMIBGroups 2 }
 
 cmnNotificationGroup NOTIFICATION-GROUP
@@ -710,7 +710,7 @@ cmnNotificationGroup NOTIFICATION-GROUP
     STATUS current
     DESCRIPTION
         "The notification generated by the CISCO-MAC-NOTIFICATION-MIB."
-    ::= { cmnMIBGroups 3 } 
+    ::= { cmnMIBGroups 3 }
 
 cmnMACMoveGroup OBJECT-GROUP
     OBJECTS{
@@ -725,7 +725,7 @@ cmnMACMoveGroup OBJECT-GROUP
     STATUS current
     DESCRIPTION
         "A collection of objects providing the global configuration
-        and information for MAC Move notification feature." 
+        and information for MAC Move notification feature."
     ::= { cmnMIBGroups 5 }
 
 cmnMACThresholdGroup OBJECT-GROUP
@@ -741,7 +741,7 @@ cmnMACThresholdGroup OBJECT-GROUP
     STATUS current
     DESCRIPTION
         "A collection of objects providing the global configuration
-        and information for MAC Threshold notification feature." 
+        and information for MAC Threshold notification feature."
     ::= { cmnMIBGroups 6 }
 
 
