diff --git a/mibs/cisco/cat3750/v2/CISCO-PRIVATE-VLAN-MIB.my b/mibs/cisco/cat3750/v2/CISCO-PRIVATE-VLAN-MIB.my
index afe0268..9b70d35 100644
--- a/mibs/cisco/cat3750/v2/CISCO-PRIVATE-VLAN-MIB.my
+++ b/mibs/cisco/cat3750/v2/CISCO-PRIVATE-VLAN-MIB.my
@@ -1,8 +1,8 @@
 -- *****************************************************************
 -- CISCO-PRIVATE-VLAN-MIB - Cisco Private Vlan MIB
 --
--- June 2001, Liwei Wang 
--- June 2002, Prasanna Parthasarathy 
+-- June 2001, Liwei Wang
+-- June 2002, Prasanna Parthasarathy
 --
 -- Copyright (c) 2001-2002, 2005-2006 by cisco Systems, Inc.
 -- All rights reserved.
@@ -13,11 +13,11 @@ CISCO-PRIVATE-VLAN-MIB DEFINITIONS ::= BEGIN
 IMPORTS
     MODULE-IDENTITY, OBJECT-TYPE
         FROM SNMPv2-SMI
-    TEXTUAL-CONVENTION, TruthValue
+    TEXTUAL-CONVENTION, TruthValue, DisplayString
         FROM SNMPv2-TC
     ciscoMgmt
         FROM CISCO-SMI
-    vtpVlanEntry, vtpVlanEditEntry        
+    vtpVlanEntry, vtpVlanEditEntry
         FROM CISCO-VTP-MIB
     ifIndex
         FROM IF-MIB
@@ -43,7 +43,7 @@ ciscoPrivateVlanMIB MODULE-IDENTITY
              Cisco's switching devices."
     REVISION "200509080000Z"
     DESCRIPTION
-             "Added support for Private VLAN Promiscuous Trunk Ports. 
+             "Added support for Private VLAN Promiscuous Trunk Ports.
              by adding a new mode type"
 
     REVISION "200207240000Z"
@@ -51,16 +51,16 @@ ciscoPrivateVlanMIB MODULE-IDENTITY
              "Added support for Private VLAN Trunk Ports. Added the
              TCs for VlanIndex Bitmaps"
     REVISION "200105230000Z"
-    DESCRIPTION 
-            "Added support for Private VLAN port mode and SVI 
-            mapping." 
+    DESCRIPTION
+            "Added support for Private VLAN port mode and SVI
+            mapping."
     REVISION "200104170000Z"
     DESCRIPTION
             " The Initial version of this MIB module."
-    ::= { ciscoMgmt 173 } 
+    ::= { ciscoMgmt 173 }
 
 
--- 
+--
 -- Textual Conventions
 --
 
@@ -69,49 +69,49 @@ PrivateVlanType ::= TEXTUAL-CONVENTION
     DESCRIPTION
             "The VLAN type as defined for Private VLAN feature.
 
-            'normal' -- this VLAN is a normal VLAN (i.e., not a 
+            'normal' -- this VLAN is a normal VLAN (i.e., not a
                         private VLAN or private group).
 
             'primary' -- this VLAN is the primary VLAN as defined for
                          Private VLAN feature.
 
             'isolated' -- this VLAN is the isolated VLAN as
-                          defined for Private VLAN feature. All the 
-                          ports in the isolated VLAN can only talk 
-                          to the specifically designated ports 
-                          configured as promiscuous ports, i.e., 
-                          the ports even in the same isolated VLAN 
-                          can not talk to each other. 
+                          defined for Private VLAN feature. All the
+                          ports in the isolated VLAN can only talk
+                          to the specifically designated ports
+                          configured as promiscuous ports, i.e.,
+                          the ports even in the same isolated VLAN
+                          can not talk to each other.
 
                           Promiscuous ports are the ports that
-                          are performing the L2 mapping of the 
-                          secondary VLANs (isolated, community, 
-                          twoWayCommunity VLANs)  to their 
-                          associated primary VLANs. Promiscuous 
-                          ports with cpvlanPromPortTwoWayRemapCapable 
-                          values of true(1) can also perform the L2 
-                          mapping of primary VLANS to the 
-                          twoWayCommunity (5) VLANs associated with 
+                          are performing the L2 mapping of the
+                          secondary VLANs (isolated, community,
+                          twoWayCommunity VLANs)  to their
+                          associated primary VLANs. Promiscuous
+                          ports with cpvlanPromPortTwoWayRemapCapable
+                          values of true(1) can also perform the L2
+                          mapping of primary VLANS to the
+                          twoWayCommunity (5) VLANs associated with
                           them.
-            
+
              'community' -- this VLAN is the community VLAN as
-                           defined for Private VLAN feature. All the 
-                           ports in this community VLAN can behave 
-                           like ports in normal VLAN type except 
-                           that they can also receive egress packets 
-                           tagged with its associated primary VLAN 
-                           ID. 
+                           defined for Private VLAN feature. All the
+                           ports in this community VLAN can behave
+                           like ports in normal VLAN type except
+                           that they can also receive egress packets
+                           tagged with its associated primary VLAN
+                           ID.
 
              'twoWayCommunity' -- this VLAN is the twoWayCommunity
                            VLAN as defined for Private VLAN feature.
                            All the ports in this twoWayCommunity VLAN
                            behave the same as ports in community(4)
-                           VLAN. Promiscuous ports with  
+                           VLAN. Promiscuous ports with
                            cpvlanPromPortTwoWayRemapCapable value
-                           of true can also perform the L2 mapping 
+                           of true can also perform the L2 mapping
                            from primary VLANs to the twoWayCommunity
-                           VLANs associated with them.  
-                           
+                           VLANs associated with them.
+
              A VLAN of isolated(3), community(4) or twoWayCommunity(5)
              type is also called a secondary VLAN."
     SYNTAX     INTEGER {
@@ -132,20 +132,20 @@ VlanIndexOrZero ::= TEXTUAL-CONVENTION
             must therefore be defined as part of the
             description of any object which uses this syntax."
     SYNTAX    INTEGER(0..4095)
-                 
+
 VlanIndexBitmap ::= TEXTUAL-CONVENTION
     STATUS    current
     DESCRIPTION
-            "A string of octets containing one bit per VLAN for a 
-            total of 1024 VLANs in the management domain.  
-            The most significant bit of the octet string is the 
+            "A string of octets containing one bit per VLAN for a
+            total of 1024 VLANs in the management domain.
+            The most significant bit of the octet string is the
             lowest value VLAN of 1024 VLANs.
 
             Refer to the description on the MIB object that
             uses this textual convention to determine the meaning
             of bits that are set ('1') or cleared ('0').
 
-            The most significant bit of the bitmap is transmitted 
+            The most significant bit of the bitmap is transmitted
             first. Note that if the length of this string is less than
             128 octets, any 'missing' octets are assumed to contain
             the value zero. An NMS may omit any zero-valued octets
@@ -153,11 +153,11 @@ VlanIndexBitmap ::= TEXTUAL-CONVENTION
             size, and the agent may also omit zero-valued trailing
             octets, to reduce the size of GetResponse PDUs."
 
-    SYNTAX    OCTET STRING (SIZE (0..128))
+    SYNTAX    DisplayString (SIZE (0..128))
 
 
 cpvlanMIBObjects OBJECT IDENTIFIER ::= { ciscoPrivateVlanMIB 1 }
- 
+
 cpvlanVlanObjects OBJECT IDENTIFIER ::= { cpvlanMIBObjects 1 }
 
 cpvlanPortObjects OBJECT IDENTIFIER ::= { cpvlanMIBObjects 2 }
@@ -172,55 +172,55 @@ cpvlanVlanTable OBJECT-TYPE
     SYNTAX      SEQUENCE OF CpvlanVlanEntry
     MAX-ACCESS  not-accessible
     STATUS      current
-    DESCRIPTION 
-            "A table containing Private VLAN information on the 
+    DESCRIPTION
+            "A table containing Private VLAN information on the
             VLANs which currently exist."
     ::= { cpvlanVlanObjects 1 }
- 
+
 cpvlanVlanEntry OBJECT-TYPE
     SYNTAX      CpvlanVlanEntry
     MAX-ACCESS  not-accessible
     STATUS      current
     DESCRIPTION
-            "A conceptual row containing the Private VLAN 
-            information on the VLANs for a particular management 
+            "A conceptual row containing the Private VLAN
+            information on the VLANs for a particular management
             domain."
     AUGMENTS   { vtpVlanEntry }
     ::= { cpvlanVlanTable 1 }
 
 CpvlanVlanEntry ::= SEQUENCE {
     cpvlanVlanPrivateVlanType PrivateVlanType,
-    cpvlanVlanAssociatedPrimaryVlan VlanIndexOrZero 
-}     
+    cpvlanVlanAssociatedPrimaryVlan VlanIndexOrZero
+}
 
 cpvlanVlanPrivateVlanType OBJECT-TYPE
-    SYNTAX     PrivateVlanType 
+    SYNTAX     PrivateVlanType
     MAX-ACCESS read-only
     STATUS     current
-    DESCRIPTION 
-            "Indicated the VLAN type as defined for Private VLAN 
+    DESCRIPTION
+            "Indicated the VLAN type as defined for Private VLAN
             feature."
     ::= { cpvlanVlanEntry 1 }
 
 cpvlanVlanAssociatedPrimaryVlan OBJECT-TYPE
-    SYNTAX     VlanIndexOrZero 
+    SYNTAX     VlanIndexOrZero
     MAX-ACCESS read-only
     STATUS     current
-    DESCRIPTION 
-            "The VLAN ID of The associated primary VLAN used for 
-            the Private VLAN feature if cpvlanVlanPrivateVlanType 
-            has the value of isolated(3), community(4) or 
-            twoWayCommunity(5). If cpvlanVlanPrivateVlanType 
-            has the value of normal(1) or primary(2), then this 
+    DESCRIPTION
+            "The VLAN ID of The associated primary VLAN used for
+            the Private VLAN feature if cpvlanVlanPrivateVlanType
+            has the value of isolated(3), community(4) or
+            twoWayCommunity(5). If cpvlanVlanPrivateVlanType
+            has the value of normal(1) or primary(2), then this
             object has the value of 0.
 
-            Note that one isolated VLAN can only be associated with 
-            one unique primary VLAN. One primary VLAN can only be 
+            Note that one isolated VLAN can only be associated with
+            one unique primary VLAN. One primary VLAN can only be
             associated with one isolated VLAN.
 
-            One primary VLAN can be associated with multiple VLANs 
+            One primary VLAN can be associated with multiple VLANs
             of community or twoWayCommunity type; one community
-            or twoWayCommunity VLAN can only be associated with one 
+            or twoWayCommunity VLAN can only be associated with one
             unique primary VLAN."
      ::= { cpvlanVlanEntry 2 }
 
@@ -229,9 +229,9 @@ cpvlanVlanEditTable OBJECT-TYPE
     MAX-ACCESS  not-accessible
     STATUS      current
     DESCRIPTION
-            "A table containing Private VLAN information on the VLANs 
-            in the VLAN Edit Buffer for a particular management 
-            domain." 
+            "A table containing Private VLAN information on the VLANs
+            in the VLAN Edit Buffer for a particular management
+            domain."
     ::= { cpvlanVlanObjects 2 }
 
 cpvlanVlanEditEntry OBJECT-TYPE
@@ -239,48 +239,48 @@ cpvlanVlanEditEntry OBJECT-TYPE
     MAX-ACCESS  not-accessible
     STATUS      current
     DESCRIPTION
-            "A conceptual row containing the Private VLAN information 
-            on the VLANs in the VLAN Edit Buffer for a particular 
+            "A conceptual row containing the Private VLAN information
+            on the VLANs in the VLAN Edit Buffer for a particular
             management domain."
     AUGMENTS   { vtpVlanEditEntry }
     ::= { cpvlanVlanEditTable 1 }
 
 CpvlanVlanEditEntry ::= SEQUENCE {
     cpvlanVlanEditPrivateVlanType PrivateVlanType,
-    cpvlanVlanEditAssocPrimaryVlan VlanIndexOrZero 
+    cpvlanVlanEditAssocPrimaryVlan VlanIndexOrZero
 }
 
 cpvlanVlanEditPrivateVlanType OBJECT-TYPE
-    SYNTAX     PrivateVlanType 
+    SYNTAX     PrivateVlanType
     MAX-ACCESS read-write
     STATUS     current
     DESCRIPTION
             "The VLAN type as defined for Private VLAN feature.
 
-            Note that a VLAN's Private VLAN type can not be 
-            changed once it already has any ports in it."  
+            Note that a VLAN's Private VLAN type can not be
+            changed once it already has any ports in it."
     DEFVAL     { normal }
     ::= { cpvlanVlanEditEntry 1 }
 
 cpvlanVlanEditAssocPrimaryVlan OBJECT-TYPE
-    SYNTAX     VlanIndexOrZero 
+    SYNTAX     VlanIndexOrZero
     MAX-ACCESS read-write
     STATUS     current
     DESCRIPTION
-            "The VLAN ID of the associated primary VLAN used for 
-            the Private VLAN feature if 
-            cpvlanVlanEditPrivateVlanType has the value of 
-            isolated(3), community(4), twoWayCommunity(5). If 
-            cpvlanVlanEditPrivateVlanType has the value of normal(1) 
+            "The VLAN ID of the associated primary VLAN used for
+            the Private VLAN feature if
+            cpvlanVlanEditPrivateVlanType has the value of
+            isolated(3), community(4), twoWayCommunity(5). If
+            cpvlanVlanEditPrivateVlanType has the value of normal(1)
             or primary(2), then this object has the value of 0.
 
-            Note that one isolated VLAN can only be associated 
-            with one unique primary VLAN. One primary VLAN can only 
+            Note that one isolated VLAN can only be associated
+            with one unique primary VLAN. One primary VLAN can only
             be associated with one isolated VLAN.
 
-            One primary VLAN can be associated with multiple VLANs 
-            of community or twoWayCommunity type; one community 
-            or twoWayCommunity VLAN can only be associated with one 
+            One primary VLAN can be associated with multiple VLANs
+            of community or twoWayCommunity type; one community
+            or twoWayCommunity VLAN can only be associated with one
             unique primary VLAN."
      DEFVAL   { 0 }
      ::= { cpvlanVlanEditEntry 2 }
@@ -294,7 +294,7 @@ cpvlanPrivatePortTable OBJECT-TYPE
     MAX-ACCESS not-accessible
     STATUS     current
     DESCRIPTION
-            "A table containing information of the configuration of 
+            "A table containing information of the configuration of
             secondary VLAN on the ports of the device."
     ::= { cpvlanPortObjects 1 }
 
@@ -303,10 +303,10 @@ cpvlanPrivatePortEntry OBJECT-TYPE
     MAX-ACCESS not-accessible
     STATUS     current
     DESCRIPTION
-            "A conceptual row containing information of the 
-            configuration of secondary VLAN for each port. An entry 
+            "A conceptual row containing information of the
+            configuration of secondary VLAN for each port. An entry
             is created by the managed system for each interface which
-            can be configured as a private port for Private VLAN 
+            can be configured as a private port for Private VLAN
             feature."
     INDEX      { ifIndex }
     ::= { cpvlanPrivatePortTable 1 }
@@ -320,26 +320,26 @@ cpvlanPrivatePortSecondaryVlan OBJECT-TYPE
     MAX-ACCESS read-write
     STATUS     current
     DESCRIPTION
-            "The VLAN ID of the secondary VLAN configured on this 
-            private port. A private port is a port that allows 
-            the ingress traffic of the secondary VLAN as well as 
-            egress traffic of its associated primary VLAN, but 
-            blocks the egress traffic of the isolated VLAN while 
-            allowing the egress traffic of the community 
-            or twoWayCommunity VLAN depending on the type of the 
-            secondary VLAN. 
-
-            Note that a port can join a secondary VLAN only after 
-            this secondary VLAN has been associated with a primary 
-            VLAN, i.e., the cpvlanVlanAssociatedPrimaryVlan has a 
-            non-zero value.  The default value of 0 for this object 
+            "The VLAN ID of the secondary VLAN configured on this
+            private port. A private port is a port that allows
+            the ingress traffic of the secondary VLAN as well as
+            egress traffic of its associated primary VLAN, but
+            blocks the egress traffic of the isolated VLAN while
+            allowing the egress traffic of the community
+            or twoWayCommunity VLAN depending on the type of the
+            secondary VLAN.
+
+            Note that a port can join a secondary VLAN only after
+            this secondary VLAN has been associated with a primary
+            VLAN, i.e., the cpvlanVlanAssociatedPrimaryVlan has a
+            non-zero value.  The default value of 0 for this object
             means this port has not joined any secondary VLAN yet."
     DEFVAL     { 0 }
-    ::= { cpvlanPrivatePortEntry 1 }    
+    ::= { cpvlanPrivatePortEntry 1 }
 
 --
--- Table for remapping secondary VLAN to primary VLAN on promiscuous 
--- ports for Private VLAN feature 
+-- Table for remapping secondary VLAN to primary VLAN on promiscuous
+-- ports for Private VLAN feature
 --
 
 cpvlanPromPortTable OBJECT-TYPE
@@ -347,7 +347,7 @@ cpvlanPromPortTable OBJECT-TYPE
     MAX-ACCESS not-accessible
     STATUS     current
     DESCRIPTION
-            "A table containing information of secondary VLAN to 
+            "A table containing information of secondary VLAN to
             primary VLAN remapping on ports of the device."
     ::= { cpvlanPortObjects 2 }
 
@@ -356,22 +356,22 @@ cpvlanPromPortEntry OBJECT-TYPE
     MAX-ACCESS not-accessible
     STATUS     current
     DESCRIPTION
-            "A conceptual row containing information of secondary 
-            VLAN to primary VLAN remapping for each port. An entry 
+            "A conceptual row containing information of secondary
+            VLAN to primary VLAN remapping for each port. An entry
             is created by the managed system for each interface
-            which can be configured as a promiscuous port for 
+            which can be configured as a promiscuous port for
             Private VLAN feature."
     INDEX      { ifIndex }
     ::= { cpvlanPromPortTable 1 }
- 
+
 CpvlanPromPortEntry ::= SEQUENCE {
     cpvlanPromPortMultiPrimaryVlan     TruthValue,
-    cpvlanPromPortSecondaryRemap       OCTET STRING,
-    cpvlanPromPortSecondaryRemap2k     OCTET STRING,
-    cpvlanPromPortSecondaryRemap3k     OCTET STRING,
-    cpvlanPromPortSecondaryRemap4k     OCTET STRING,
+    cpvlanPromPortSecondaryRemap       DisplayString,
+    cpvlanPromPortSecondaryRemap2k     DisplayString,
+    cpvlanPromPortSecondaryRemap3k     DisplayString,
+    cpvlanPromPortSecondaryRemap4k     DisplayString,
     cpvlanPromPortTwoWayRemapCapable TruthValue
-} 
+}
 
 cpvlanPromPortMultiPrimaryVlan OBJECT-TYPE
     SYNTAX     TruthValue
@@ -385,82 +385,82 @@ cpvlanPromPortMultiPrimaryVlan OBJECT-TYPE
             cpvlanPromPortSecondaryRemap3k,
             cpvlanPromPortSecondaryRemap4k) can belong to multiple
             primary VLANs or not for this port. If this object value
-            is false(2), then the object values of 
-            cpvlanVlanAssociatedPrimaryVlan for the secondary VLANs 
-            with their bits turned on as '1' in those remapping 
-            objects must be the same. If this object value is 
-            true(1), then the object values of 
-            cpvlanVlanAssociatedPrimaryVlan for the secondary VLANs 
-            with their bits turned on as '1' in those remapping 
-            objects need not be the same. 
-    
-            A promiscuous port can remap a secondary VLAN to its 
-            associated primary VLAN for egress traffic on the fly 
+            is false(2), then the object values of
+            cpvlanVlanAssociatedPrimaryVlan for the secondary VLANs
+            with their bits turned on as '1' in those remapping
+            objects must be the same. If this object value is
+            true(1), then the object values of
+            cpvlanVlanAssociatedPrimaryVlan for the secondary VLANs
+            with their bits turned on as '1' in those remapping
+            objects need not be the same.
+
+            A promiscuous port can remap a secondary VLAN to its
+            associated primary VLAN for egress traffic on the fly
             as defined for Private VLAN feature."
-    ::= { cpvlanPromPortEntry 1 } 
+    ::= { cpvlanPromPortEntry 1 }
 
 cpvlanPromPortSecondaryRemap OBJECT-TYPE
-    SYNTAX     OCTET STRING (SIZE (0..128))
+    SYNTAX     DisplayString (SIZE (0..128))
     MAX-ACCESS read-write
     STATUS     current
     DESCRIPTION
             "A string of octets containing one bit per VLAN in the
             management domain on this port.  The first octet
-            corresponds to VLANs with VlanIndexOrZero values of 0 
-            through 7; the second octet to VLANs 8 through 15; etc.  
-            The most significant bit of each octet corresponds to 
+            corresponds to VLANs with VlanIndexOrZero values of 0
+            through 7; the second octet to VLANs 8 through 15; etc.
+            The most significant bit of each octet corresponds to
             the lowest value VlanIndexOrZero in that octet.
 
-            A bit can only be set to '1' when the bit is 
-            corresponding to a VLAN of Private VLAN isolated, 
-            community or twoWayCommunity type which has already 
-            been associated with a primary VLAN. Setting this bit 
-            to '1' makes this promiscuous port remap the secondary 
-            VLAN to its associated primary VLAN for egress traffic 
+            A bit can only be set to '1' when the bit is
+            corresponding to a VLAN of Private VLAN isolated,
+            community or twoWayCommunity type which has already
+            been associated with a primary VLAN. Setting this bit
+            to '1' makes this promiscuous port remap the secondary
+            VLAN to its associated primary VLAN for egress traffic
             on the fly, or remap the associated primary VLAN to
-            the secondary VLAN if the secondary VLAN is of 
-            twoWayCommunity type and the object value of 
-            cpvlanPromPortTwoWayRemapCapable for this promiscuous 
-            port is true(1).  
-        
+            the secondary VLAN if the secondary VLAN is of
+            twoWayCommunity type and the object value of
+            cpvlanPromPortTwoWayRemapCapable for this promiscuous
+            port is true(1).
+
             Note that if the length of this string is less than
             128 octets, any 'missing' octets are assumed to contain
             the value zero. An NMS may omit any zero-valued octets
-            from the end of this string in order to reduce SetPDU 
-            size, and the agent may also omit zero-valued trailing 
+            from the end of this string in order to reduce SetPDU
+            size, and the agent may also omit zero-valued trailing
             octets, to reduce the size of GetResponse PDUs."
     ::= { cpvlanPromPortEntry 2 }
 
 cpvlanPromPortSecondaryRemap2k OBJECT-TYPE
-    SYNTAX     OCTET STRING (SIZE (0..128))
+    SYNTAX     DisplayString (SIZE (0..128))
     MAX-ACCESS read-write
     STATUS     current
-    DESCRIPTION 
-            "A string of octets containing one bit per VLAN  for 
-            VLANs with VlanIndexOrZero values of 1024 through 2047 
-            in the management domain on this port.  The first octet 
-            corresponds to VLANs with VlanIndexOrZero values of 1024 
-            through 1031; the second octet to VLANs 1032 through 
-            1039; etc.  The most significant bit of each octet 
-            corresponds to the lowest value VlanIndexOrZero in 
-            that octet. 
-
-            A bit can only be set to '1' when the bit is 
-            corresponding to a VLAN of Private VLAN isolated or 
-            community type which has already been associated with a 
-            primary VLAN. Setting this bit to '1' makes this 
-            promiscuous port remap the secondary VLAN to its 
+    DESCRIPTION
+            "A string of octets containing one bit per VLAN  for
+            VLANs with VlanIndexOrZero values of 1024 through 2047
+            in the management domain on this port.  The first octet
+            corresponds to VLANs with VlanIndexOrZero values of 1024
+            through 1031; the second octet to VLANs 1032 through
+            1039; etc.  The most significant bit of each octet
+            corresponds to the lowest value VlanIndexOrZero in
+            that octet.
+
+            A bit can only be set to '1' when the bit is
+            corresponding to a VLAN of Private VLAN isolated or
+            community type which has already been associated with a
+            primary VLAN. Setting this bit to '1' makes this
+            promiscuous port remap the secondary VLAN to its
             associated primary VLAN for egress traffic on the fly,
-            or remap the associated primary VLAN to the secondary 
-            VLAN if the secondary VLAN is of twoWayCommunity type 
-            and the object value of cpvlanPromPortTwoWayRemapCapable 
-            for this promiscuous port is true(1). 
- 
+            or remap the associated primary VLAN to the secondary
+            VLAN if the secondary VLAN is of twoWayCommunity type
+            and the object value of cpvlanPromPortTwoWayRemapCapable
+            for this promiscuous port is true(1).
+
             Note that if the length of this string is less than
             128 octets, any 'missing' octets are assumed to contain
             the value zero. An NMS may omit any zero-valued octets
-            from the end of this string in order to reduce SetPDU 
-            size, and the agent may also omit zero-valued trailing 
+            from the end of this string in order to reduce SetPDU
+            size, and the agent may also omit zero-valued trailing
             octets, to reduce the size of GetResponse PDUs.
 
             This object is only instantiated on devices which support
@@ -468,35 +468,35 @@ cpvlanPromPortSecondaryRemap2k OBJECT-TYPE
     ::= { cpvlanPromPortEntry 3 }
 
 cpvlanPromPortSecondaryRemap3k OBJECT-TYPE
-    SYNTAX     OCTET STRING (SIZE (0..128))
+    SYNTAX     DisplayString (SIZE (0..128))
     MAX-ACCESS read-write
     STATUS     current
     DESCRIPTION
-            "A string of octets containing one bit per VLAN  for 
-            VLANs with VlanIndexOrZero values of 2048 through 3071 
-            in the management domain on this port.  The first octet 
-            corresponds to VLANs with VlanIndexOrZero values of 2048 
-            through 2055; the second octet to VLANs 2056 through 
-            2063; etc.  The most significant bit of each octet 
-            corresponds to the lowest value VlanIndexOrZero in 
+            "A string of octets containing one bit per VLAN  for
+            VLANs with VlanIndexOrZero values of 2048 through 3071
+            in the management domain on this port.  The first octet
+            corresponds to VLANs with VlanIndexOrZero values of 2048
+            through 2055; the second octet to VLANs 2056 through
+            2063; etc.  The most significant bit of each octet
+            corresponds to the lowest value VlanIndexOrZero in
             that octet.
 
-            A bit can only be set to '1' when the bit is 
-            corresponding to a VLAN of Private VLAN isolated or 
-            community type which has already been associated with a 
-            primary VLAN. Setting this bit to '1' makes this 
-            promiscuous port remap the secondary VLAN to its 
-            associated primary VLAN for egress traffic on the fly, 
-            or remap the associated primary VLAN to the secondary 
-            VLAN if the secondary VLAN is of twoWayCommunity type 
-            and the object value of cpvlanPromPortTwoWayRemapCapable 
-            for this promiscuous port is true(1). 
+            A bit can only be set to '1' when the bit is
+            corresponding to a VLAN of Private VLAN isolated or
+            community type which has already been associated with a
+            primary VLAN. Setting this bit to '1' makes this
+            promiscuous port remap the secondary VLAN to its
+            associated primary VLAN for egress traffic on the fly,
+            or remap the associated primary VLAN to the secondary
+            VLAN if the secondary VLAN is of twoWayCommunity type
+            and the object value of cpvlanPromPortTwoWayRemapCapable
+            for this promiscuous port is true(1).
 
             Note that if the length of this string is less than
             128 octets, any 'missing' octets are assumed to contain
             the value zero. An NMS may omit any zero-valued octets
-            from the end of this string in order to reduce SetPDU 
-            size, and the agent may also omit zero-valued trailing 
+            from the end of this string in order to reduce SetPDU
+            size, and the agent may also omit zero-valued trailing
             octets, to reduce the size of GetResponse PDUs.
 
             This object is only instantiated on devices which support
@@ -504,35 +504,35 @@ cpvlanPromPortSecondaryRemap3k OBJECT-TYPE
     ::= { cpvlanPromPortEntry 4 }
 
 cpvlanPromPortSecondaryRemap4k OBJECT-TYPE
-    SYNTAX     OCTET STRING (SIZE (0..128))
+    SYNTAX     DisplayString (SIZE (0..128))
     MAX-ACCESS read-write
     STATUS     current
     DESCRIPTION
-            "A string of octets containing one bit per VLAN  for 
-            VLANs with VlanIndexOrZero values of 3072 through 4095 
-            in the management domain on this port.  The first octet 
-            corresponds to VLANs with VlanIndexOrZero values of 3072 
-            through 3079; the second octet to VLANs 3080 through 
-            3087; etc.  The most significant bit of each octet 
-            corresponds to the lowest value VlanIndexOrZero in 
+            "A string of octets containing one bit per VLAN  for
+            VLANs with VlanIndexOrZero values of 3072 through 4095
+            in the management domain on this port.  The first octet
+            corresponds to VLANs with VlanIndexOrZero values of 3072
+            through 3079; the second octet to VLANs 3080 through
+            3087; etc.  The most significant bit of each octet
+            corresponds to the lowest value VlanIndexOrZero in
             that octet.
 
-            A bit can only be set to '1' when the bit is 
-            corresponding to a VLAN of Private VLAN isolated or 
-            community type which has already been associated with a 
-            primary VLAN. Setting this bit to '1' makes this 
-            promiscuous port remap the secondary VLAN to its 
-            associated primary VLAN for egress traffic on the fly, 
-            or remap the associated primary VLAN to the secondary 
-            VLAN if the secondary VLAN is of twoWayCommunity type 
-            and the object value of cpvlanPromPortTwoWayRemapCapable 
-            for this promiscuous port is true(1). 
+            A bit can only be set to '1' when the bit is
+            corresponding to a VLAN of Private VLAN isolated or
+            community type which has already been associated with a
+            primary VLAN. Setting this bit to '1' makes this
+            promiscuous port remap the secondary VLAN to its
+            associated primary VLAN for egress traffic on the fly,
+            or remap the associated primary VLAN to the secondary
+            VLAN if the secondary VLAN is of twoWayCommunity type
+            and the object value of cpvlanPromPortTwoWayRemapCapable
+            for this promiscuous port is true(1).
 
             Note that if the length of this string is less than
             128 octets, any 'missing' octets are assumed to contain
             the value zero. An NMS may omit any zero-valued octets
-            from the end of this string in order to reduce SetPDU 
-            size, and the agent may also omit zero-valued trailing 
+            from the end of this string in order to reduce SetPDU
+            size, and the agent may also omit zero-valued trailing
             octets, to reduce the size of GetResponse PDUs.
 
             This object is only instantiated on devices which support
@@ -546,14 +546,14 @@ cpvlanPromPortTwoWayRemapCapable OBJECT-TYPE
     DESCRIPTION
             "Indicates whether the port is capable to perform
             remapping from primary VLANs to the twoWayCommunity VLANs
-            associated with them for the Private VLAN feature. A 
-            promiscuous port with this object value of false(2) can 
-            only perform one-way remapping from secondary VLANs to 
-            their associated primary VLANs, while a promiscuous port 
-            with this object value of true(1) can also perform 
-            remapping from primary VLANs to the twoWayCommunity VLANs 
+            associated with them for the Private VLAN feature. A
+            promiscuous port with this object value of false(2) can
+            only perform one-way remapping from secondary VLANs to
+            their associated primary VLANs, while a promiscuous port
+            with this object value of true(1) can also perform
+            remapping from primary VLANs to the twoWayCommunity VLANs
             associated with them in addition to that."
-    ::= { cpvlanPromPortEntry 6 } 
+    ::= { cpvlanPromPortEntry 6 }
 
 
 --
@@ -575,43 +575,43 @@ cpvlanPortModeEntry OBJECT-TYPE
     STATUS     current
     DESCRIPTION
             "A conceptual row containing information of the
-            configuration of port mode on each port for the 
-            Private VLAN feature. An entry of this table is 
+            configuration of port mode on each port for the
+            Private VLAN feature. An entry of this table is
             created by the managed system when the capability
-            to be a Private VLAN port is detected on an 
-            interface." 
+            to be a Private VLAN port is detected on an
+            interface."
     INDEX      { ifIndex }
     ::= { cpvlanPortModeTable 1 }
 
 CpvlanPortModeEntry ::= SEQUENCE {
-    cpvlanPortMode INTEGER 
+    cpvlanPortMode INTEGER
 }
 
 cpvlanPortMode OBJECT-TYPE
     SYNTAX     INTEGER {
-                       nonPrivateVlan(1), 
+                       nonPrivateVlan(1),
                        host(2),
                        promiscuous(3),
                        secondaryTrunk(4),
                        promiscuousTrunk(5)
-               }        
+               }
     MAX-ACCESS read-write
     STATUS     current
     DESCRIPTION
-            "The Private VLAN port mode on this port.  
+            "The Private VLAN port mode on this port.
 
             nonPrivateVlan(1) -- this port is configured to be a
                                  non-Private-VLAN port.
 
-            host(2) -- this port is configured to be 
-                       a Private-VLAN host port, i.e., private 
+            host(2) -- this port is configured to be
+                       a Private-VLAN host port, i.e., private
                        port.
 
             promiscuous(3) -- this port is configured to be
                               a Private-VLAN promiscuous port.
 
             secondaryTrunk(4) -- this port is configured to be a
-                        Private-VLAN isolated trunk port or community 
+                        Private-VLAN isolated trunk port or community
                         trunk port.
 
             promiscuousTrunk(5) -- this port is configured to be a
@@ -629,8 +629,8 @@ cpvlanTrunkPortTable OBJECT-TYPE
     MAX-ACCESS not-accessible
     STATUS     current
     DESCRIPTION
-            "A table containing information of the configuration of 
-             a private vlan trunk port in the device. This includes 
+            "A table containing information of the configuration of
+             a private vlan trunk port in the device. This includes
              secondary and normal allowed VLAN, encapsulation type,
              trunk native vlan (as applied to private vlan trunks)."
     ::= { cpvlanPortObjects 4 }
@@ -640,8 +640,8 @@ cpvlanTrunkPortEntry OBJECT-TYPE
     MAX-ACCESS not-accessible
     STATUS     current
     DESCRIPTION
-            "A conceptual row containing information of the 
-            configuration of one private vlan trunk port. An entry 
+            "A conceptual row containing information of the
+            configuration of one private vlan trunk port. An entry
             is created by the managed system for each interface which
             can be configured as a private vlan trunk port"
     INDEX      { ifIndex }
@@ -672,7 +672,7 @@ cpvlanTrunkPortDynamicState OBJECT-TYPE
             operator to mandate the trunking behavior of the port
 
             on(1) dictates that the private VLAN port will always be a
-            trunk. If the negotiation is supported on this port, 
+            trunk. If the negotiation is supported on this port,
             negotiation will take place with the far end to attempt to
             bring the far end into trunking state.
 
@@ -683,7 +683,7 @@ cpvlanTrunkPortDynamicState OBJECT-TYPE
             takes place with the far end.
 
             If a port does not support negotiation, the value of on(1)
-            is not allowed. If the port's cpvlanTrunkPortEncapType is 
+            is not allowed. If the port's cpvlanTrunkPortEncapType is
             set to negotiate(3), onNoNegotiate(2) is not allowed."
 
        ::= { cpvlanTrunkPortEntry 1 }
@@ -694,22 +694,22 @@ cpvlanTrunkPortEncapType OBJECT-TYPE
     STATUS     current
     DESCRIPTION
             "The type of VLAN encapsulation desired to be used on this
-             private vlan trunk port. 
+             private vlan trunk port.
 
-             It is either a particular type, or 'negotiate'  meaning 
+             It is either a particular type, or 'negotiate'  meaning
              whatever type results from the negotiation.
 
              dot1Q(1) indicates that the port should accept and transmit
              packets with IEEE 802.1q VLAN encapsulation
 
-             isl(2) indicates that the port should accept and transmit 
+             isl(2) indicates that the port should accept and transmit
              packets with Inter Switch Link (ISL) VLAN encapsulation
 
              negotiate(3) indicates that the VLAN encapsulation is
-             negotiated with the far end. The negotiated VLAN 
-             encapsulation can be dot1Q or  isl. negotiate(2) is not 
-             allowed if the port does not support negotiation of 
-             VLAN encapsulation type or if its 
+             negotiated with the far end. The negotiated VLAN
+             encapsulation can be dot1Q or  isl. negotiate(2) is not
+             allowed if the port does not support negotiation of
+             VLAN encapsulation type or if its
              cpvlanTrunkPortDynamicState is set to onNoNegotiate(2)"
 
     ::= { cpvlanTrunkPortEntry 2 }
@@ -720,9 +720,9 @@ cpvlanTrunkPortNativeVlan OBJECT-TYPE
     STATUS     current
     DESCRIPTION
             "The VlanIndex of the VLAN which is represented by native
-             frames on this private vlan trunk port. For private vlan 
+             frames on this private vlan trunk port. For private vlan
              trunk ports that need to drop untagged frames or not
-             supporting the sending and receiving of native frames, 
+             supporting the sending and receiving of native frames,
              this value should be set to zero"
     ::= { cpvlanTrunkPortEntry 3 }
 
@@ -733,51 +733,51 @@ cpvlanTrunkPortSecondaryVlans OBJECT-TYPE
     DESCRIPTION
             "A string of octets containing one bit per VLAN for VLANs
             with values of 0 through 1023 in the management domain
-            on this port. 
+            on this port.
 
-            If the VLAN represented by setting a bit to '1' in the 
-            bitmap is a secondary VLAN that has already been 
+            If the VLAN represented by setting a bit to '1' in the
+            bitmap is a secondary VLAN that has already been
             associated with a primary VLAN, it allows this private
             VLAN trunk port to remap the secondary VLAN to its primary
             VLAN for ingress traffic and to remap its associated primary
             VLAN to the secondary VLAN for egress traffic.
 
-            If the bit corresponding to a VLAN is set to '1' and if 
+            If the bit corresponding to a VLAN is set to '1' and if
             the corresponding VLAN is a secondary VLAN, then the local
-            system is enabled to transmit and receive frames with 
+            system is enabled to transmit and receive frames with
             proper VLAN remapping via this Private VLAN trunk port.
 
             If the bit corresponding to a VLAN is set to '0', and if
             the corresponding VLAN is not configured as a normal VLAN,
-            then the system is disabled from sending and receiving 
+            then the system is disabled from sending and receiving
             frames on that VLAN via this Private VLAN trunk port."
- 
+
     ::= { cpvlanTrunkPortEntry 4 }
 
 cpvlanTrunkPortSecondaryVlans2k OBJECT-TYPE
     SYNTAX     VlanIndexBitmap
     MAX-ACCESS read-write
     STATUS     current
-    DESCRIPTION 
-            "A string of octets containing one bit per VLAN for VLANs 
+    DESCRIPTION
+            "A string of octets containing one bit per VLAN for VLANs
             with values of 1024 through 2047 in the management domain
-            on this port. 
+            on this port.
 
-            If the VLAN represented by setting a bit to '1' in the 
-            bitmap is a secondary VLAN that has already been 
+            If the VLAN represented by setting a bit to '1' in the
+            bitmap is a secondary VLAN that has already been
             associated with a primary VLAN, it allows this private
             VLAN trunk port to remap the secondary VLAN to its primary
             VLAN for ingress traffic and to remap its associated primary
             VLAN to the secondary VLAN for egress traffic.
 
-            If the bit corresponding to a VLAN is set to '1' and if 
+            If the bit corresponding to a VLAN is set to '1' and if
             the corresponding VLAN is a secondary VLAN, then the local
-            system is enabled to transmit and receive frames with 
+            system is enabled to transmit and receive frames with
             proper VLAN remapping via this Private VLAN trunk port.
 
             If the bit corresponding to a VLAN is set to '0', and if
             the corresponding VLAN is not configured as a normal VLAN,
-            then the system is disabled from sending and receiving 
+            then the system is disabled from sending and receiving
             frames on that VLAN via this Private VLAN trunk port.
 
             This object is only instantiated on devices which support
@@ -790,25 +790,25 @@ cpvlanTrunkPortSecondaryVlans3k OBJECT-TYPE
     MAX-ACCESS read-write
     STATUS     current
     DESCRIPTION
-            "A string of octets containing one bit per VLAN for VLANs 
-            with values of 2048 through 3071 in the management domain 
-            on this port. 
+            "A string of octets containing one bit per VLAN for VLANs
+            with values of 2048 through 3071 in the management domain
+            on this port.
 
-            If the VLAN represented by setting a bit to '1' in the 
-            bitmap is a secondary VLAN that has already been 
+            If the VLAN represented by setting a bit to '1' in the
+            bitmap is a secondary VLAN that has already been
             associated with a primary VLAN, it allows this private
             VLAN trunk port to remap the secondary VLAN to its primary
             VLAN for ingress traffic and to remap its associated primary
             VLAN to the secondary VLAN for egress traffic.
 
-            If the bit corresponding to a VLAN is set to '1' and if 
+            If the bit corresponding to a VLAN is set to '1' and if
             the corresponding VLAN is a secondary VLAN, then the local
-            system is enabled to transmit and receive frames with 
+            system is enabled to transmit and receive frames with
             proper VLAN remapping via this Private VLAN trunk port.
 
             If the bit corresponding to a VLAN is set to '0', and if
             the corresponding VLAN is not configured as a normal VLAN,
-            then the system is disabled from sending and receiving 
+            then the system is disabled from sending and receiving
             frames on that VLAN via this Private VLAN trunk port.
 
             This object is only instantiated on devices which support
@@ -821,25 +821,25 @@ cpvlanTrunkPortSecondaryVlans4k OBJECT-TYPE
     MAX-ACCESS read-write
     STATUS     current
     DESCRIPTION
-            "A string of octets containing one bit per VLAN for VLANs 
+            "A string of octets containing one bit per VLAN for VLANs
             with values of 3072 through 4095 in the management domain
-            on this port. 
+            on this port.
 
-            If the VLAN represented by setting a bit to '1' in the 
-            bitmap is a secondary VLAN that has already been 
+            If the VLAN represented by setting a bit to '1' in the
+            bitmap is a secondary VLAN that has already been
             associated with a primary VLAN, it allows this private
             VLAN trunk port to remap the secondary VLAN to its primary
             VLAN for ingress traffic and to remap its associated primary
             VLAN to the secondary VLAN for egress traffic.
 
-            If the bit corresponding to a VLAN is set to '1' and if 
+            If the bit corresponding to a VLAN is set to '1' and if
             the corresponding VLAN is a secondary VLAN, then the local
-            system is enabled to transmit and receive frames with 
+            system is enabled to transmit and receive frames with
             proper VLAN remapping via this Private VLAN trunk port.
 
             If the bit corresponding to a VLAN is set to '0', and if
             the corresponding VLAN is not configured as a normal VLAN,
-            then the system is disabled from sending and receiving 
+            then the system is disabled from sending and receiving
             frames on that VLAN via this Private VLAN trunk port.
 
             This object is only instantiated on devices which support
@@ -852,21 +852,21 @@ cpvlanTrunkPortNormalVlans OBJECT-TYPE
     MAX-ACCESS read-write
     STATUS     current
     DESCRIPTION
-            "A string of octets containing one bit per VLAN for VLANs 
-            with values of 0 through 1023 in the management domain 
+            "A string of octets containing one bit per VLAN for VLANs
+            with values of 0 through 1023 in the management domain
             on this port.
 
-            If the VLAN represented by setting a bit to '1' in the 
+            If the VLAN represented by setting a bit to '1' in the
             bitmap is a normal VLAN, it allows packets belonging to
-            this vlan on this Private VLAN trunk port. 
+            this vlan on this Private VLAN trunk port.
 
-            If the bit corresponding to a VLAN is set to '1' and if 
+            If the bit corresponding to a VLAN is set to '1' and if
             the corresponding VLAN is a normal VLAN, then the local
             system is enabled to transmit and receive frames as
             normal VLAN
 
             If the bit corresponding to a VLAN is set to '0', and if
-            the corresponding VLAN is not configured as a secondary 
+            the corresponding VLAN is not configured as a secondary
             VLAN, then the system is disabled from sending and receiving
             frames on that VLAN via this Private VLAN trunk port."
 
@@ -876,24 +876,24 @@ cpvlanTrunkPortNormalVlans2k OBJECT-TYPE
     SYNTAX     VlanIndexBitmap
     MAX-ACCESS read-write
     STATUS     current
-    DESCRIPTION 
-            "A string of octets containing one bit per VLAN for VLANs 
+    DESCRIPTION
+            "A string of octets containing one bit per VLAN for VLANs
             with values of 1024 through 2047 in the management domain
-            on this port. 
+            on this port.
 
-            If the VLAN represented by setting a bit to '1' in the 
+            If the VLAN represented by setting a bit to '1' in the
             bitmap is a normal VLAN, it allows packets belonging to
-            this vlan on this Private VLAN trunk port. 
+            this vlan on this Private VLAN trunk port.
 
-            If the bit corresponding to a VLAN is set to '1' and if 
+            If the bit corresponding to a VLAN is set to '1' and if
             the corresponding VLAN is a normal VLAN, then the local
             system is enabled to transmit and receive frames as
             normal VLAN
 
             If the bit corresponding to a VLAN is set to '0', and if
-            the corresponding VLAN is not configured as a secondary 
+            the corresponding VLAN is not configured as a secondary
             VLAN, then the system is disabled from sending and receiving
-            frames on that VLAN via this Private VLAN trunk port. 
+            frames on that VLAN via this Private VLAN trunk port.
 
             This object is only instantiated on devices which support
             the range of VlanIndexOrZero up to 4095."
@@ -905,23 +905,23 @@ cpvlanTrunkPortNormalVlans3k OBJECT-TYPE
     MAX-ACCESS read-write
     STATUS     current
     DESCRIPTION
-            "A string of octets containing one bit per VLAN for VLANs 
+            "A string of octets containing one bit per VLAN for VLANs
             with values of 2048 through 3071 in the management domain
-            on this port. 
+            on this port.
 
-            If the VLAN represented by setting a bit to '1' in the 
+            If the VLAN represented by setting a bit to '1' in the
             bitmap is a normal VLAN, it allows packets belonging to
-            this vlan on this Private VLAN trunk port. 
+            this vlan on this Private VLAN trunk port.
 
-            If the bit corresponding to a VLAN is set to '1' and if 
+            If the bit corresponding to a VLAN is set to '1' and if
             the corresponding VLAN is a normal VLAN, then the local
             system is enabled to transmit and receive frames as
             normal VLAN
 
             If the bit corresponding to a VLAN is set to '0', and if
-            the corresponding VLAN is not configured as a secondary 
+            the corresponding VLAN is not configured as a secondary
             VLAN, then the system is disabled from sending and receiving
-            frames on that VLAN via this Private VLAN trunk port. 
+            frames on that VLAN via this Private VLAN trunk port.
 
             This object is only instantiated on devices which support
             the range of VlanIndexOrZero up to 4095."
@@ -935,21 +935,21 @@ cpvlanTrunkPortNormalVlans4k OBJECT-TYPE
     DESCRIPTION
             "A string of octets containing one bit per VLAN for VLANs
             with values of 3072 through 4095 in the management domain
-            on this port. 
+            on this port.
 
-            If the VLAN represented by setting a bit to '1' in the 
+            If the VLAN represented by setting a bit to '1' in the
             bitmap is a normal VLAN, it allows packets belonging to
-            this vlan on this Private VLAN trunk port. 
+            this vlan on this Private VLAN trunk port.
 
-            If the bit corresponding to a VLAN is set to '1' and if 
+            If the bit corresponding to a VLAN is set to '1' and if
             the corresponding VLAN is a normal VLAN, then the local
             system is enabled to transmit and receive frames as
             normal VLAN
 
             If the bit corresponding to a VLAN is set to '0', and if
-            the corresponding VLAN is not configured as a secondary 
+            the corresponding VLAN is not configured as a secondary
             VLAN, then the system is disabled from sending and receiving
-            frames on that VLAN via this Private VLAN trunk port. 
+            frames on that VLAN via this Private VLAN trunk port.
 
             This object is only instantiated on devices which support
             the range of VlanIndexOrZero up to 4095."
@@ -961,17 +961,17 @@ cpvlanTrunkPortDynamicStatus OBJECT-TYPE
     MAX-ACCESS read-only
     STATUS     current
     DESCRIPTION
-            "Indicates the current dynamic trunking status of the 
+            "Indicates the current dynamic trunking status of the
             specified private VLAN port.
 
-            trunking(1) indicates that the private VLAN port is 
+            trunking(1) indicates that the private VLAN port is
             currently operating as a private VLAN trunk port
 
             notTrunking(2) indicates that the private VLAN port
-            is currently not trunking but is operating as an 
+            is currently not trunking but is operating as an
             access port."
 
- 
+
     ::= { cpvlanTrunkPortEntry 12 }
 
 cpvlanTrunkPortEncapOperType OBJECT-TYPE
@@ -979,24 +979,24 @@ cpvlanTrunkPortEncapOperType OBJECT-TYPE
     MAX-ACCESS read-only
     STATUS     current
     DESCRIPTION
-            "The type of VLAN encapsulation in use on this private 
-            vlan trunk port. 
- 
+            "The type of VLAN encapsulation in use on this private
+            vlan trunk port.
+
             dot1Q(1) indicates that the port accepts and transmits
             packets with IEEE 802.1q VLAN encapsulation
 
-            isl(2) indicates that the port accepts and transmits 
+            isl(2) indicates that the port accepts and transmits
             packets with Inter Switch Link (ISL) VLAN encapsulation
 
             If the cpvlanTrunkPortDynamicStatus is notTrunking(2) or
-            if the encapsulation type negotiation has not been 
+            if the encapsulation type negotiation has not been
             completed, the object is set to notApplicable(3)."
-      
+
 
     ::= { cpvlanTrunkPortEntry 13 }
 
 --
--- Private VLAN mapping for the Switch Virtual Interfaces  
+-- Private VLAN mapping for the Switch Virtual Interfaces
 --
 
 cpvlanSVIMappingTable OBJECT-TYPE
@@ -1004,9 +1004,9 @@ cpvlanSVIMappingTable OBJECT-TYPE
     MAX-ACCESS  not-accessible
     STATUS      current
     DESCRIPTION
-            "A table containing the configuration of 
-            primary VLAN SVI (Switch Virtual Interfaces) 
-            mapping for the secondary VLANs for the Private 
+            "A table containing the configuration of
+            primary VLAN SVI (Switch Virtual Interfaces)
+            mapping for the secondary VLANs for the Private
             VLAN feature."
     ::= { cpvlanSVIObjects 1 }
 
@@ -1015,38 +1015,38 @@ cpvlanSVIMappingEntry OBJECT-TYPE
     MAX-ACCESS  not-accessible
     STATUS      current
     DESCRIPTION
-            "A conceptual row containing the Primary VLAN 
-            SVI mapping configuration for the existing 
-            secondary VLANs. An entry is created by the managed 
-            system for each VLAN with corresponding VLAN's 
-            cpvlanVlanPrivateVlanType of isolated(3), 
+            "A conceptual row containing the Primary VLAN
+            SVI mapping configuration for the existing
+            secondary VLANs. An entry is created by the managed
+            system for each VLAN with corresponding VLAN's
+            cpvlanVlanPrivateVlanType of isolated(3),
             community(4), and twoWayCommunity(5)."
-   INDEX      { cpvlanSVIMappingVlanIndex } 
+   INDEX      { cpvlanSVIMappingVlanIndex }
     ::= { cpvlanSVIMappingTable 1 }
 
 CpvlanSVIMappingEntry ::= SEQUENCE {
-    cpvlanSVIMappingVlanIndex VlanIndexOrZero, 
-    cpvlanSVIMappingPrimarySVI VlanIndexOrZero 
+    cpvlanSVIMappingVlanIndex VlanIndexOrZero,
+    cpvlanSVIMappingPrimarySVI VlanIndexOrZero
 }
 
 cpvlanSVIMappingVlanIndex OBJECT-TYPE
     SYNTAX        VlanIndexOrZero
     MAX-ACCESS    not-accessible
     STATUS        current
-    DESCRIPTION 
+    DESCRIPTION
             "An index value that uniquely identifies the
             Virtual LAN associated with this information."
     ::= { cpvlanSVIMappingEntry 1 }
 
 cpvlanSVIMappingPrimarySVI OBJECT-TYPE
-    SYNTAX     VlanIndexOrZero 
+    SYNTAX     VlanIndexOrZero
     MAX-ACCESS read-write
     STATUS     current
     DESCRIPTION
             "The Vlan ID of the primary VLAN SVI this secondary
-            VLAN is mapped to for the Private VLAN feature. 
-            This object has the value of zero if this secondary 
-            VLAN is not mapped to any primary VLAN SVI."  
+            VLAN is mapped to for the Private VLAN feature.
+            This object has the value of zero if this secondary
+            VLAN is not mapped to any primary VLAN SVI."
      DEFVAL { 0 }
      ::= { cpvlanSVIMappingEntry 2 }
 
@@ -1063,9 +1063,9 @@ cpvlanMIBGroups      OBJECT IDENTIFIER ::= { cpvlanMIBConformance 2}
 --
 
 cpvlanMIBCompliance MODULE-COMPLIANCE
-    STATUS  current 
+    STATUS  current
     DESCRIPTION
-            "The compliance statement for Private VLAN feature 
+            "The compliance statement for Private VLAN feature
             implementation."
     MODULE  -- this module
              -- no MANDATORY-GROUPS
@@ -1073,35 +1073,35 @@ cpvlanMIBCompliance MODULE-COMPLIANCE
         OBJECT       cpvlanTrunkPortEncapType
         SYNTAX       INTEGER { dot1Q(1) }
         MIN-ACCESS   read-only
-        DESCRIPTION  
+        DESCRIPTION
                 "Write access is not required and only one of the
-                 3 enumerated values for the Private Trunk port 
-                 encapsulation types need to be supported, 
+                 3 enumerated values for the Private Trunk port
+                 encapsulation types need to be supported,
                  specifically: dot1Q(1)."
 
         GROUP cpvlanVlanGroup
-        DESCRIPTION 
-                "This group must be implemented on devices which has 
+        DESCRIPTION
+                "This group must be implemented on devices which has
                 Private VLAN feature support."
         GROUP cpvlanPrivatePortGroup
-        DESCRIPTION 
-                "This group must be implemented on devices which has 
+        DESCRIPTION
+                "This group must be implemented on devices which has
                 Private VLAN feature support."
         GROUP cpvlanPromPortGroup
         DESCRIPTION
-                "This group must be implemented on devices which has 
+                "This group must be implemented on devices which has
                 support for promiscuous port of Private VLAN feature."
         GROUP cpvlanPromPort4kGroup
         DESCRIPTION
-                "This group must be implemented on devices which has 
-                support for promiscuous port of Private VLAN feature 
+                "This group must be implemented on devices which has
+                support for promiscuous port of Private VLAN feature
                 for VlanIndexOrZero range of up to 4095."
         GROUP cpvlanTrunkPortGroup
-        DESCRIPTION 
+        DESCRIPTION
                 "This group is mandatory for a managed system which has
                 Private VLAN trunk ports support."
 
-    ::= { cpvlanMIBCompliances 1 }                             
+    ::= { cpvlanMIBCompliances 1 }
 
 --
 -- units of conformance
@@ -1110,36 +1110,36 @@ cpvlanVlanGroup OBJECT-GROUP
     OBJECTS { cpvlanVlanPrivateVlanType,
               cpvlanVlanAssociatedPrimaryVlan,
               cpvlanVlanEditPrivateVlanType,
-              cpvlanVlanEditAssocPrimaryVlan 
+              cpvlanVlanEditAssocPrimaryVlan
             }
     STATUS  current
     DESCRIPTION
-            "A collection of objects providing basic VLAN 
+            "A collection of objects providing basic VLAN
             configuration for Private VLAN feature."
     ::= { cpvlanMIBGroups 1 }
 
 cpvlanPrivatePortGroup OBJECT-GROUP
-    OBJECTS { cpvlanPrivatePortSecondaryVlan 
+    OBJECTS { cpvlanPrivatePortSecondaryVlan
             }
     STATUS  current
     DESCRIPTION
-            "A collection of objects providing basic 
-            private port configuration for Private VLAN 
+            "A collection of objects providing basic
+            private port configuration for Private VLAN
             feature."
     ::= { cpvlanMIBGroups 2 }
 
 
 cpvlanPromPortGroup OBJECT-GROUP
     OBJECTS { cpvlanPromPortMultiPrimaryVlan,
-              cpvlanPromPortSecondaryRemap, 
-              cpvlanPromPortTwoWayRemapCapable 
+              cpvlanPromPortSecondaryRemap,
+              cpvlanPromPortTwoWayRemapCapable
             }
     STATUS  current
     DESCRIPTION
-            "A collection of objects providing basic 
-            promiscuous port configuration for Private 
+            "A collection of objects providing basic
+            promiscuous port configuration for Private
             VLAN feature."
-    ::= { cpvlanMIBGroups 3 }  
+    ::= { cpvlanMIBGroups 3 }
 
 cpvlanPromPort4kGroup OBJECT-GROUP
     OBJECTS { cpvlanPromPortSecondaryRemap2k,
@@ -1148,18 +1148,18 @@ cpvlanPromPort4kGroup OBJECT-GROUP
             }
     STATUS  current
     DESCRIPTION
-            "A collection of objects providing basic promiscuous 
-            port configuration for Private VLAN feature on 
-            devices with VlanIndexOrZero range of up to 4095."   
+            "A collection of objects providing basic promiscuous
+            port configuration for Private VLAN feature on
+            devices with VlanIndexOrZero range of up to 4095."
     ::= { cpvlanMIBGroups 4 }
 
 cpvlanPortModeGroup OBJECT-GROUP
     OBJECTS { cpvlanPortMode }
     STATUS  current
     DESCRIPTION
-            "This is an optional group with a collection of 
-            objects providing basic port mode configuration for 
-            Private VLAN feature on devices which support 
+            "This is an optional group with a collection of
+            objects providing basic port mode configuration for
+            Private VLAN feature on devices which support
             Private VLAN port mode feature."
     ::= { cpvlanMIBGroups 5 }
 
@@ -1167,10 +1167,10 @@ cpvlanSVIMappingGroup OBJECT-GROUP
     OBJECTS { cpvlanSVIMappingPrimarySVI }
     STATUS  current
     DESCRIPTION
-            "This is an optional group with a collection of 
+            "This is an optional group with a collection of
             objects providing primary VLAN SVI mapping
-            configuration for the Private VLAN feature on 
-            devices which support Primary VLAN SVI mapping 
+            configuration for the Private VLAN feature on
+            devices which support Primary VLAN SVI mapping
             feature."
     ::= { cpvlanMIBGroups 6 }
 
@@ -1195,5 +1195,5 @@ cpvlanTrunkPortGroup OBJECT-GROUP
             trunk port configuration for Private VLAN feature."
     ::= { cpvlanMIBGroups 7 }
 
-END 
+END
 
