diff --git a/mibs/cisco/cat3750/v2/CISCO-CDP-MIB.my b/mibs/cisco/cat3750/v2/CISCO-CDP-MIB.my
index 269dd5e..2ace4b8 100644
--- a/mibs/cisco/cat3750/v2/CISCO-CDP-MIB.my
+++ b/mibs/cisco/cat3750/v2/CISCO-CDP-MIB.my
@@ -21,7 +21,7 @@ IMPORTS
         ciscoMgmt
                 FROM CISCO-SMI
         CiscoNetworkProtocol, CiscoNetworkAddress
-                FROM CISCO-TC 
+                FROM CISCO-TC
         VlanIndex
                 FROM CISCO-VTP-MIB
         ifIndex
@@ -54,24 +54,24 @@ ciscoCdpMIB   MODULE-IDENTITY
                 "Added cdpCtAddressTable which contains the following
                  objects:
                  cdpCtAddressIndex,
-                 cdpCtAddressType, 
+                 cdpCtAddressType,
                  cdpCtAddress."
         REVISION        "200111230000Z"
         DESCRIPTION
                 "Added cdpInterfaceExtTable which contains the following
                  objects:
-                 cdpInterfaceExtendedTrust, 
-                 cdpInterfaceCosForUntrustedPort." 
+                 cdpInterfaceExtendedTrust,
+                 cdpInterfaceCosForUntrustedPort."
         REVISION        "200104230000Z"
         DESCRIPTION
                 "Added the following objects:
-                 cdpGlobalDeviceIdFormatCpb, 
-                 cdpGlobalDeviceIdFormat." 
+                 cdpGlobalDeviceIdFormatCpb,
+                 cdpGlobalDeviceIdFormat."
         REVISION        "200011220000Z"
         DESCRIPTION
-                "Added the following objects: 
+                "Added the following objects:
                  cdpCacheApplianceID,
-                 cdpCacheVlanID, 
+                 cdpCacheVlanID,
                  cdpCachePowerConsumption,
                  cdpCacheMTU,
                  cdpCachePrimaryMgmtAddrType,
@@ -80,7 +80,7 @@ ciscoCdpMIB   MODULE-IDENTITY
                  cdpCacheSecondaryMgmtAddr,
                  cdpCacheLastChange,
                  cdpCachePhysLocation,
-                 cdpCacheSysName, 
+                 cdpCacheSysName,
                  cdpCacheSysObjectID,
                  cdpGlobalLastChange"
         REVISION        "9812100000Z"
@@ -88,9 +88,9 @@ ciscoCdpMIB   MODULE-IDENTITY
                 "Added cdpGlobalDeviceId object."
         REVISION        "9809160000Z"
         DESCRIPTION
-                "added these objects to cdpCacheTable:  
-                cdpCacheVTPMgmtDomain, 
-                cdpCacheNativeVLAN, 
+                "added these objects to cdpCacheTable:
+                cdpCacheVTPMgmtDomain,
+                cdpCacheNativeVLAN,
                 cdpCacheDuplex.
                 "
         REVISION        "9607080000Z"
@@ -139,7 +139,7 @@ cdpInterfaceEntry OBJECT-TYPE
     ::= { cdpInterfaceTable 1 }
 
 CdpInterfaceEntry ::= SEQUENCE {
-    cdpInterfaceIfIndex           Integer32, 
+    cdpInterfaceIfIndex           Integer32,
     cdpInterfaceEnable            TruthValue,
     cdpInterfaceMessageInterval   INTEGER,
     cdpInterfaceGroup             Integer32,
@@ -207,7 +207,7 @@ cdpInterfacePort OBJECT-TYPE
             it indicates the RFC1516 port number of the repeater
             port which corresponds to this interface."
     ::= { cdpInterfaceEntry 5 }
-    
+
 cdpInterfaceName OBJECT-TYPE
     SYNTAX     DisplayString
     MAX-ACCESS read-only
@@ -240,13 +240,13 @@ cdpInterfaceExtEntry OBJECT-TYPE
 
 CdpInterfaceExtEntry ::= SEQUENCE {
     cdpInterfaceExtendedTrust        INTEGER,
-    cdpInterfaceCosForUntrustedPort  Unsigned32 
+    cdpInterfaceCosForUntrustedPort  Unsigned32
 }
 
 cdpInterfaceExtendedTrust OBJECT-TYPE
     SYNTAX     INTEGER {
                    trusted(1),
-                   noTrust(2) 
+                   noTrust(2)
                }
     MAX-ACCESS read-write
     STATUS     current
@@ -265,7 +265,7 @@ cdpInterfaceExtendedTrust OBJECT-TYPE
 cdpInterfaceCosForUntrustedPort OBJECT-TYPE
     SYNTAX     Unsigned32 (0..7)
     MAX-ACCESS read-write
-    STATUS     current 
+    STATUS     current
     DESCRIPTION
         "Indicates the value to be sent by COS for Untrusted Ports TLV."
     ::= { cdpInterfaceExtEntry 2 }
@@ -306,9 +306,9 @@ CdpCacheEntry ::= SEQUENCE {
     cdpCacheDeviceId                DisplayString,
     cdpCacheDevicePort              DisplayString,
     cdpCachePlatform                DisplayString,
-    cdpCacheCapabilities            OCTET STRING, 
+    cdpCacheCapabilities            DisplayString,
     cdpCacheVTPMgmtDomain           DisplayString,
-    cdpCacheNativeVLAN              VlanIndex, 
+    cdpCacheNativeVLAN              VlanIndex,
     cdpCacheDuplex                  INTEGER,
     cdpCacheApplianceID             Unsigned32,
     cdpCacheVlanID                  Unsigned32,
@@ -364,8 +364,8 @@ cdpCacheAddress OBJECT-TYPE
           "The (first) network-layer address of the device
           as reported in the Address TLV of the most recently received
           CDP message.  For example, if the corresponding instance of
-          cacheAddressType had the value 'ip(1)', then this object 
-          would be an IPv4-address.  If the neighbor device is 
+          cacheAddressType had the value 'ip(1)', then this object
+          would be an IPv4-address.  If the neighbor device is
           SNMP-manageable, it is supposed to generate its CDP messages
           such that this address is one at which it will receive SNMP
           messages. Use cdpCtAddressTable to extract the remaining
@@ -418,7 +418,7 @@ cdpCachePlatform OBJECT-TYPE
     ::= { cdpCacheEntry 8 }
 
 cdpCacheCapabilities OBJECT-TYPE
-    SYNTAX     OCTET STRING (SIZE (0..4))
+    SYNTAX     DisplayString (SIZE (0..4))
     MAX-ACCESS read-only
     STATUS     current
     DESCRIPTION
@@ -431,30 +431,30 @@ cdpCacheCapabilities OBJECT-TYPE
 
     ::= { cdpCacheEntry 9 }
 
-cdpCacheVTPMgmtDomain   OBJECT-TYPE	
+cdpCacheVTPMgmtDomain   OBJECT-TYPE
     SYNTAX    DisplayString (SIZE (0..32))
     MAX-ACCESS read-only
     STATUS     current
     DESCRIPTION
-        "The VTP Management Domain for the remote device's interface, 
+        "The VTP Management Domain for the remote device's interface,
         as reported in the most recently received CDP message.
         This object is not instantiated if no VTP Management Domain field
         (TLV) was reported in the most recently received CDP message."
     REFERENCE "managementDomainName in CISCO-VTP-MIB"
     ::= { cdpCacheEntry 10 }
- 
+
 cdpCacheNativeVLAN OBJECT-TYPE
     SYNTAX    VlanIndex
     MAX-ACCESS read-only
     STATUS     current
     DESCRIPTION
-        "The remote device's interface's native VLAN, as reported in the 
+        "The remote device's interface's native VLAN, as reported in the
         most recent CDP message.  The value 0 indicates
         no native VLAN field (TLV) was reported in the most
         recent CDP message."
     ::= { cdpCacheEntry 11 }
 
-cdpCacheDuplex OBJECT-TYPE	
+cdpCacheDuplex OBJECT-TYPE
     SYNTAX    INTEGER {
                   unknown(1),
                   halfduplex(2),
@@ -463,35 +463,35 @@ cdpCacheDuplex OBJECT-TYPE
     MAX-ACCESS read-only
     STATUS     current
     DESCRIPTION
-        "The remote device's interface's duplex mode, as reported in the 
+        "The remote device's interface's duplex mode, as reported in the
         most recent CDP message.  The value unknown(1) indicates
         no duplex mode field (TLV) was reported in the most
         recent CDP message."
     ::= { cdpCacheEntry 12 }
 
-cdpCacheApplianceID OBJECT-TYPE	
+cdpCacheApplianceID OBJECT-TYPE
     SYNTAX     Unsigned32 (0..255)
     MAX-ACCESS read-only
     STATUS     current
     DESCRIPTION
-        "The remote device's Appliance ID, as reported in the 
+        "The remote device's Appliance ID, as reported in the
         most recent CDP message. This object is not instantiated if
         no Appliance VLAN-ID field (TLV) was reported in the most
         recently received CDP message."
     ::= { cdpCacheEntry 13 }
 
-cdpCacheVlanID OBJECT-TYPE	
+cdpCacheVlanID OBJECT-TYPE
     SYNTAX     Unsigned32 (0..4095)
     MAX-ACCESS read-only
     STATUS     current
     DESCRIPTION
-        "The remote device's VoIP VLAN ID, as reported in the 
+        "The remote device's VoIP VLAN ID, as reported in the
         most recent CDP message. This object is not instantiated if
         no Appliance VLAN-ID field (TLV) was reported in the most
         recently received CDP message."
     ::= { cdpCacheEntry 14 }
 
-cdpCachePowerConsumption OBJECT-TYPE	
+cdpCachePowerConsumption OBJECT-TYPE
     SYNTAX     Unsigned32
     UNITS      "milliwatts"
     MAX-ACCESS read-only
@@ -503,7 +503,7 @@ cdpCachePowerConsumption OBJECT-TYPE
         recently received CDP message."
     ::= { cdpCacheEntry 15 }
 
-cdpCacheMTU OBJECT-TYPE	
+cdpCacheMTU OBJECT-TYPE
     SYNTAX     Unsigned32
     MAX-ACCESS read-only
     STATUS     current
@@ -514,7 +514,7 @@ cdpCacheMTU OBJECT-TYPE
         (TLV) was reported in the most recently received CDP message."
     ::= { cdpCacheEntry 16 }
 
-cdpCacheSysName OBJECT-TYPE	
+cdpCacheSysName OBJECT-TYPE
     SYNTAX     DisplayString (SIZE (0..255))
     MAX-ACCESS read-only
     STATUS     current
@@ -525,7 +525,7 @@ cdpCacheSysName OBJECT-TYPE
         reported in the most recently received CDP message."
     ::= { cdpCacheEntry 17 }
 
-cdpCacheSysObjectID OBJECT-TYPE	
+cdpCacheSysObjectID OBJECT-TYPE
     SYNTAX     OBJECT IDENTIFIER
     MAX-ACCESS read-only
     STATUS     current
@@ -544,21 +544,21 @@ cdpCachePrimaryMgmtAddrType OBJECT-TYPE
             corresponding instance of cdpCachePrimaryMgmtAddress."
     ::= { cdpCacheEntry 19 }
 
-cdpCachePrimaryMgmtAddr OBJECT-TYPE	
-    SYNTAX     CiscoNetworkAddress 
+cdpCachePrimaryMgmtAddr OBJECT-TYPE
+    SYNTAX     CiscoNetworkAddress
     MAX-ACCESS read-only
     STATUS     current
     DESCRIPTION
             "This object indicates the (first) network layer
             address at which the device will accept SNMP messages
-            as reported in the first address in the 
+            as reported in the first address in the
             Management-Address TLV of the most recently received
-            CDP message.  If the corresponding instance of 
+            CDP message.  If the corresponding instance of
             cdpCachePrimaryMgmtAddrType has the value 'ip(1)',
-            then this object would be an IP-address. If the 
-            remote device is not currently manageable via any 
-            network protocol, then it reports the special value 
-            of the IPv4 address 0.0.0.0, and that address is 
+            then this object would be an IP-address. If the
+            remote device is not currently manageable via any
+            network protocol, then it reports the special value
+            of the IPv4 address 0.0.0.0, and that address is
             recorded in this object.  If the most recently received
             CDP message did not contain the Management-Address
             TLV, then this object is not instanstiated."
@@ -573,27 +573,27 @@ cdpCacheSecondaryMgmtAddrType OBJECT-TYPE
             corresponding instance of cdpCacheSecondaryMgmtAddress."
     ::= { cdpCacheEntry 21 }
 
-cdpCacheSecondaryMgmtAddr OBJECT-TYPE	
-    SYNTAX     CiscoNetworkAddress 
+cdpCacheSecondaryMgmtAddr OBJECT-TYPE
+    SYNTAX     CiscoNetworkAddress
     MAX-ACCESS read-only
     STATUS     current
     DESCRIPTION
             "This object indicates the alternate network layer
             address at which the device will accept SNMP messages
-            as reported in the second address in the 
+            as reported in the second address in the
             Management-Address TLV of the most recently received
             CDP message.  If the corresponding instance of
             cdpCacheSecondaryMgmtAddrType has the value 'ip(1)',
-            then this object would be an IP-address. If the 
-            remote device reports the special value of the 
-            IPv4 address 0.0.0.0, that address is recorded in 
+            then this object would be an IP-address. If the
+            remote device reports the special value of the
+            IPv4 address 0.0.0.0, that address is recorded in
             this object.  If the most recently received CDP
             message did not contain the Management-Address
             TLV, or if that TLV contained only one address, then
             this object is not instanstiated."
     ::= { cdpCacheEntry 22 }
 
-cdpCachePhysLocation OBJECT-TYPE	
+cdpCachePhysLocation OBJECT-TYPE
     SYNTAX     DisplayString
     MAX-ACCESS read-only
     STATUS     current
@@ -603,10 +603,10 @@ cdpCachePhysLocation OBJECT-TYPE
         to, the remote device's interface over which the CDP packet is
         sent. This object is not instantiated if no Physical Location
         field (TLV) was reported by the most recently received CDP
-        message." 
+        message."
     ::= { cdpCacheEntry 23 }
 
-cdpCacheLastChange OBJECT-TYPE	
+cdpCacheLastChange OBJECT-TYPE
     SYNTAX     TimeStamp
     MAX-ACCESS read-only
     STATUS     current
@@ -623,7 +623,7 @@ cdpCtAddressTable OBJECT-TYPE
     MAX-ACCESS not-accessible
     STATUS     current
     DESCRIPTION
-            "The (conceptual) table containing the list of 
+            "The (conceptual) table containing the list of
             network-layer addresses of a neighbor interface,
             as reported in the Address TLV of the most recently
             received CDP message. The first address included in
@@ -638,15 +638,15 @@ cdpCtAddressEntry OBJECT-TYPE
     STATUS     current
     DESCRIPTION
             "An entry (conceptual row) in the cdpCtAddressTable,
-            containing the information on one address received via CDP 
-            on one interface from one device.  Entries appear 
+            containing the information on one address received via CDP
+            on one interface from one device.  Entries appear
             when a CDP advertisement is received from a neighbor
             device, with an Address TLV.  Entries disappear when
-            CDP is disabled on the interface, or globally. An entry 
+            CDP is disabled on the interface, or globally. An entry
             or entries would also disappear if the most recently
             received CDP packet contain fewer address entries in the
             Address TLV, than are currently present in the CDP cache."
-    INDEX      { cdpCacheIfIndex, cdpCacheDeviceIndex, 
+    INDEX      { cdpCacheIfIndex, cdpCacheDeviceIndex,
                  cdpCtAddressIndex }
     ::= { cdpCtAddressTable 1 }
 
@@ -661,7 +661,7 @@ cdpCtAddressIndex OBJECT-TYPE
     MAX-ACCESS not-accessible
     STATUS     current
     DESCRIPTION
-            "The index of the address entry for a given 
+            "The index of the address entry for a given
             cdpCacheIfIndex,cdpCacheDeviceIndex pair. It
             has the value N-1 for the N-th address in the
             Address TLV"
@@ -733,7 +733,7 @@ cdpGlobalDeviceId OBJECT-TYPE
     STATUS     current
     DESCRIPTION
         "The device ID advertised by this device. The format of this
-         device id is characterized by the value of 
+         device id is characterized by the value of
          cdpGlobalDeviceIdFormat object."
     ::= { cdpGlobal 4 }
 
@@ -748,11 +748,11 @@ cdpGlobalLastChange  OBJECT-TYPE
     ::= { cdpGlobal 5 }
 
 cdpGlobalDeviceIdFormatCpb  OBJECT-TYPE
-    SYNTAX     BITS { 
-                 serialNumber(0), 
+    SYNTAX     BITS {
+                 serialNumber(0),
                  macAddress(1),
-                 other (2) 
-               } 
+                 other (2)
+               }
     MAX-ACCESS read-only
     STATUS     current
     DESCRIPTION
@@ -769,11 +769,11 @@ cdpGlobalDeviceIdFormatCpb  OBJECT-TYPE
     ::= { cdpGlobal 6 }
 
 cdpGlobalDeviceIdFormat  OBJECT-TYPE
-    SYNTAX     INTEGER { 
-                 serialNumber(1), 
+    SYNTAX     INTEGER {
+                 serialNumber(1),
                  macAddress(2),
-                 other(3) 
-               } 
+                 other(3)
+               }
     MAX-ACCESS read-write
     STATUS     current
     DESCRIPTION
@@ -781,18 +781,18 @@ cdpGlobalDeviceIdFormat  OBJECT-TYPE
         corresponding instance of cdpGlobalDeviceId. User can only
         specify the formats that the device is capable of as
         denoted in cdpGlobalDeviceIdFormatCpb object.
-        
-        serialNumber(1) indicates that the value of cdpGlobalDeviceId 
+
+        serialNumber(1) indicates that the value of cdpGlobalDeviceId
         object is in the form of an ASCII string contain the device
-        serial number. 
-        
-        macAddress(2) indicates that the value of cdpGlobalDeviceId 
+        serial number.
+
+        macAddress(2) indicates that the value of cdpGlobalDeviceId
         object is in the form of Layer 2 MAC address.
 
         other(3) indicates that the value of cdpGlobalDeviceId object
         is in the form of a platform specific ASCII string contain
         info that identifies the device. For example: ASCII string
-        contains serialNumber appended/prepened with system name." 
+        contains serialNumber appended/prepened with system name."
     ::= { cdpGlobal 7 }
 
 -- conformance information
@@ -840,7 +840,7 @@ ciscoCdpMIBComplianceV12R02 MODULE-COMPLIANCE
             "The compliance statement for the CDP MIB."
     MODULE  -- this module
         MANDATORY-GROUPS { ciscoCdpMIBGroupV12R02 }
- 
+
     ::= { ciscoCdpMIBCompliances 4 }
 
 ciscoCdpMIBCompliance5 MODULE-COMPLIANCE
@@ -849,18 +849,18 @@ ciscoCdpMIBCompliance5 MODULE-COMPLIANCE
             "The compliance statement for the CDP MIB."
     MODULE  -- this module
         MANDATORY-GROUPS { ciscoCdpMIBGroupV12R02 }
- 
+
     ::= { ciscoCdpMIBCompliances 5 }
-    
+
 ciscoCdpMIBComplianceV12R03 MODULE-COMPLIANCE
-    STATUS  current 
+    STATUS  current
     DESCRIPTION
 	"The compliance statement for the CDP MIB."
     MODULE  -- this module
         MANDATORY-GROUPS { ciscoCdpMIBGroupV12R03,
                            ciscoCdpCtAddressGroup
                          }
-    
+
     GROUP ciscoCdpV2MIBGroup
     DESCRIPTION
         "This group is mandatory for agents implementing CDP version 2"
@@ -869,7 +869,7 @@ ciscoCdpMIBComplianceV12R03 MODULE-COMPLIANCE
     DESCRIPTION
         "This group is mandatory for agents implementing Extended Trust
          TLV and COS for Untrusted Port TLV of CDP version 2."
-        
+
    ::= { ciscoCdpMIBCompliances 6 }
 
 -- units of conformance
@@ -928,9 +928,9 @@ ciscoCdpMIBGroupV12R02 OBJECT-GROUP
             "A collection of objects for use with the Cisco
             Discovery Protocol."
     ::= { ciscoCdpMIBGroups 5 }
-    
+
 ciscoCdpV2MIBGroup OBJECT-GROUP
-    OBJECTS { 
+    OBJECTS {
               cdpCacheApplianceID, cdpCacheVlanID,
               cdpCachePowerConsumption, cdpCacheMTU,
               cdpCacheSysName, cdpCacheSysObjectID,
@@ -947,9 +947,9 @@ ciscoCdpV2MIBGroup OBJECT-GROUP
             "A collection of objects for use with the Cisco
             Discovery Protocol version 2."
     ::= { ciscoCdpMIBGroups 6 }
-    
+
 ciscoCdpV2IfExtGroup OBJECT-GROUP
-    OBJECTS { 
+    OBJECTS {
               cdpInterfaceExtendedTrust,
               cdpInterfaceCosForUntrustedPort
             }
@@ -961,7 +961,7 @@ ciscoCdpV2IfExtGroup OBJECT-GROUP
     ::= { ciscoCdpMIBGroups 7 }
 
 ciscoCdpCtAddressGroup OBJECT-GROUP
-    OBJECTS { 
+    OBJECTS {
               cdpCtAddressType,
               cdpCtAddress
             }
@@ -971,7 +971,7 @@ ciscoCdpCtAddressGroup OBJECT-GROUP
             Discovery Protocol to obtain the addresses from
             the Address TLV of a received CDP packet."
     ::= { ciscoCdpMIBGroups 8 }
-    
+
 ciscoCdpMIBGroupV12R03 OBJECT-GROUP
     OBJECTS { cdpInterfaceEnable,
               cdpInterfaceGroup, cdpInterfacePort, cdpInterfaceName,
