diff --git a/mibs/cisco/cat3750/v2/PIM-MIB.my b/mibs/cisco/cat3750/v2/PIM-MIB.my
index 2169673..5d40191 100644
--- a/mibs/cisco/cat3750/v2/PIM-MIB.my
+++ b/mibs/cisco/cat3750/v2/PIM-MIB.my
@@ -5,7 +5,7 @@
 --
 -- Copyright (c) 1994-1997, 1999, 2000, 2001 by cisco Systems, Inc.
 -- All rights reserved.
--- 
+--
 -- *****************************************************************
 
 -- This mib was extracted from RFC 2934.
@@ -16,7 +16,7 @@ IMPORTS
     MODULE-IDENTITY, OBJECT-TYPE, experimental,
     NOTIFICATION-TYPE,
     Integer32, IpAddress, TimeTicks  FROM SNMPv2-SMI
-    RowStatus, TruthValue            FROM SNMPv2-TC
+    RowStatus, TruthValue, DisplayString  FROM SNMPv2-TC
     MODULE-COMPLIANCE, OBJECT-GROUP  FROM SNMPv2-CONF
 --    NOTIFICATION-GROUP               FROM SNMPv2-CONF
     ipMRouteGroup, ipMRouteSource,
@@ -310,7 +310,7 @@ PimIpMRouteEntry ::= SEQUENCE {
     pimIpMRouteAssertMetric          Integer32,
     pimIpMRouteAssertMetricPref      Integer32,
     pimIpMRouteAssertRPTBit          TruthValue,
-    pimIpMRouteFlags                 OCTET STRING(SIZE(1))
+    pimIpMRouteFlags                 DisplayString(SIZE(1))
 }
 
 pimIpMRouteUpstreamAssertTimer OBJECT-TYPE
@@ -359,7 +359,7 @@ pimIpMRouteAssertRPTBit OBJECT-TYPE
     ::= { pimIpMRouteEntry 4 }
 
 pimIpMRouteFlags OBJECT-TYPE
-    SYNTAX     OCTET STRING (SIZE(1))
+    SYNTAX     DisplayString (SIZE(1))
     MAX-ACCESS read-only
     STATUS     current
     DESCRIPTION
