diff --git a/mibs/cisco/wlc/v2/Q-BRIDGE-MIB.my b/mibs/cisco/wlc/v2/Q-BRIDGE-MIB.my
index ce01ca0..9a69a32 100644
--- a/mibs/cisco/wlc/v2/Q-BRIDGE-MIB.my
+++ b/mibs/cisco/wlc/v2/Q-BRIDGE-MIB.my
@@ -22,7 +22,8 @@ IMPORTS
     RowStatus,
     TruthValue,
     TEXTUAL-CONVENTION,
-    MacAddress
+    MacAddress,
+    DisplayString
         FROM SNMPv2-TC
     SnmpAdminString
         FROM SNMP-FRAMEWORK-MIB
@@ -144,7 +145,7 @@ PortList ::= TEXTUAL-CONVENTION
         value of this object.  If that bit has a value of '1',
         then that port is included in the set of ports; the port
         is not included if its bit has a value of '0'."
-    SYNTAX          OCTET STRING
+    SYNTAX          DisplayString
 
 VlanIndex ::= TEXTUAL-CONVENTION
     STATUS          current
@@ -246,7 +247,7 @@ dot1qVlanVersionNumber OBJECT-TYPE
     DESCRIPTION
         "The version number of IEEE 802.1Q that this device
         supports."
-    REFERENCE       "IEEE 802.1Q/D11 Section 12.10.1.1" 
+    REFERENCE       "IEEE 802.1Q/D11 Section 12.10.1.1"
     ::= { dot1qBase 1 }
 
 dot1qMaxVlanId OBJECT-TYPE
@@ -257,7 +258,7 @@ dot1qMaxVlanId OBJECT-TYPE
         "The maximum IEEE 802.1Q VLAN-ID that this device
 
         supports."
-    REFERENCE       "IEEE 802.1Q/D11 Section 9.3.2.3" 
+    REFERENCE       "IEEE 802.1Q/D11 Section 9.3.2.3"
     ::= { dot1qBase 2 }
 
 dot1qMaxSupportedVlans OBJECT-TYPE
@@ -267,7 +268,7 @@ dot1qMaxSupportedVlans OBJECT-TYPE
     DESCRIPTION
         "The maximum number of IEEE 802.1Q VLANs that this
         device supports."
-    REFERENCE       "IEEE 802.1Q/D11 Section 12.10.1.1" 
+    REFERENCE       "IEEE 802.1Q/D11 Section 12.10.1.1"
     ::= { dot1qBase 3 }
 
 dot1qNumVlans OBJECT-TYPE
@@ -277,7 +278,7 @@ dot1qNumVlans OBJECT-TYPE
     DESCRIPTION
         "The current number of IEEE 802.1Q VLANs that are
         configured in this device."
-    REFERENCE       "IEEE 802.1Q/D11 Section 12.7.1.1" 
+    REFERENCE       "IEEE 802.1Q/D11 Section 12.7.1.1"
     ::= { dot1qBase 4 }
 
 dot1qGvrpStatus OBJECT-TYPE
@@ -297,19 +298,19 @@ dot1qGvrpStatus OBJECT-TYPE
 
         The value of this object MUST be retained across
         reinitializations of the management system."
-    DEFVAL          { enabled } 
+    DEFVAL          { enabled }
     ::= { dot1qBase 5 }
 -- -------------------------------------------------------------
---   
+--
 -- the dot1qTp subtree
 -- -------------------------------------------------------------
---   
+--
 -- -------------------------------------------------------------
 -- the current Filtering Database Table
 -- -------------------------------------------------------------
 
 dot1qFdbTable OBJECT-TYPE
-    SYNTAX          SEQUENCE OF Dot1qFdbEntry 
+    SYNTAX          SEQUENCE OF Dot1qFdbEntry
     MAX-ACCESS      not-accessible
     STATUS          current
     DESCRIPTION
@@ -325,7 +326,7 @@ dot1qFdbEntry OBJECT-TYPE
     MAX-ACCESS      not-accessible
     STATUS          current
     DESCRIPTION     "Information about a specific Filtering Database."
-    INDEX           { dot1qFdbId } 
+    INDEX           { dot1qFdbId }
     ::= { dot1qFdbTable 1 }
 
 Dot1qFdbEntry ::= SEQUENCE {
@@ -337,7 +338,7 @@ dot1qFdbId OBJECT-TYPE
     SYNTAX          Unsigned32
     MAX-ACCESS      not-accessible
     STATUS          current
-    DESCRIPTION     "The identity of this Filtering Database." 
+    DESCRIPTION     "The identity of this Filtering Database."
     ::= { dot1qFdbEntry 1 }
 
 dot1qFdbDynamicCount OBJECT-TYPE
@@ -347,9 +348,9 @@ dot1qFdbDynamicCount OBJECT-TYPE
     DESCRIPTION
         "The current number of dynamic entries in this
         Filtering Database."
-    REFERENCE       "IEEE 802.1Q/D11 Section 12.7.1.1.3" 
+    REFERENCE       "IEEE 802.1Q/D11 Section 12.7.1.1.3"
     ::= { dot1qFdbEntry 2 }
- 
+
 
 -- -------------------------------------------------------------
 -- Multiple Forwarding Databases for 802.1Q Transparent Devices
@@ -359,7 +360,7 @@ dot1qFdbDynamicCount OBJECT-TYPE
 -- -------------------------------------------------------------
 
 dot1qTpFdbTable OBJECT-TYPE
-    SYNTAX          SEQUENCE OF Dot1qTpFdbEntry 
+    SYNTAX          SEQUENCE OF Dot1qTpFdbEntry
     MAX-ACCESS      not-accessible
     STATUS          current
     DESCRIPTION
@@ -382,13 +383,13 @@ dot1qTpFdbEntry OBJECT-TYPE
     INDEX           {
                         dot1qFdbId,
                         dot1qTpFdbAddress
-                    } 
+                    }
     ::= { dot1qTpFdbTable 1 }
 
 Dot1qTpFdbEntry ::= SEQUENCE {
         dot1qTpFdbAddress MacAddress,
         dot1qTpFdbPort    Integer32,
-        dot1qTpFdbStatus  INTEGER 
+        dot1qTpFdbStatus  INTEGER
 }
 
 dot1qTpFdbAddress OBJECT-TYPE
@@ -397,7 +398,7 @@ dot1qTpFdbAddress OBJECT-TYPE
     STATUS          current
     DESCRIPTION
         "A unicast MAC address for which the device has
-        forwarding and/or filtering information." 
+        forwarding and/or filtering information."
     ::= { dot1qTpFdbEntry 1 }
 
 dot1qTpFdbPort OBJECT-TYPE
@@ -415,7 +416,7 @@ dot1qTpFdbPort OBJECT-TYPE
         Implementors are encouraged to assign the port value to
         this object whenever it is learned, even for addresses
         for which the corresponding value of dot1qTpFdbStatus is
-        not learned(3)." 
+        not learned(3)."
     ::= { dot1qTpFdbEntry 2 }
 
 dot1qTpFdbStatus OBJECT-TYPE
@@ -451,16 +452,16 @@ dot1qTpFdbStatus OBJECT-TYPE
                 ports has this address.
             mgmt(5) - the value of the corresponding instance of
                 dot1qTpFdbAddress is also the value of an
-                existing instance of dot1qStaticAddress." 
+                existing instance of dot1qStaticAddress."
     ::= { dot1qTpFdbEntry 3 }
- 
+
 
 -- -------------------------------------------------------------
 -- Dynamic Group Registration Table
 -- -------------------------------------------------------------
 
 dot1qTpGroupTable OBJECT-TYPE
-    SYNTAX          SEQUENCE OF Dot1qTpGroupEntry 
+    SYNTAX          SEQUENCE OF Dot1qTpGroupEntry
     MAX-ACCESS      not-accessible
     STATUS          current
     DESCRIPTION
@@ -486,7 +487,7 @@ dot1qTpGroupEntry OBJECT-TYPE
     INDEX           {
                         dot1qVlanIndex,
                         dot1qTpGroupAddress
-                    } 
+                    }
     ::= { dot1qTpGroupTable 1 }
 
 Dot1qTpGroupEntry ::= SEQUENCE {
@@ -501,7 +502,7 @@ dot1qTpGroupAddress OBJECT-TYPE
     STATUS          current
     DESCRIPTION
         "The destination Group MAC address in a frame to which
-        this entry's filtering information applies." 
+        this entry's filtering information applies."
     ::= { dot1qTpGroupEntry 1 }
 
 dot1qTpGroupEgressPorts OBJECT-TYPE
@@ -513,7 +514,7 @@ dot1qTpGroupEgressPorts OBJECT-TYPE
         frames destined for this Group MAC address are currently
         being explicitly forwarded.  This does not include ports
         for which this address is only implicitly forwarded, in
-        the dot1qForwardAllPorts list." 
+        the dot1qForwardAllPorts list."
     ::= { dot1qTpGroupEntry 2 }
 
 dot1qTpGroupLearnt OBJECT-TYPE
@@ -523,16 +524,16 @@ dot1qTpGroupLearnt OBJECT-TYPE
     DESCRIPTION
         "The subset of ports in dot1qTpGroupEgressPorts that
         were learned by GMRP or some other dynamic mechanism, in
-        this Filtering database." 
+        this Filtering database."
     ::= { dot1qTpGroupEntry 3 }
- 
+
 
 -- -------------------------------------------------------------
 -- Service Requirements subtree
 -- -------------------------------------------------------------
 
 dot1qForwardAllTable OBJECT-TYPE
-    SYNTAX          SEQUENCE OF Dot1qForwardAllEntry 
+    SYNTAX          SEQUENCE OF Dot1qForwardAllEntry
     MAX-ACCESS      not-accessible
     STATUS          current
     DESCRIPTION
@@ -555,7 +556,7 @@ dot1qForwardAllEntry OBJECT-TYPE
         of ports to which all multicasts should be forwarded,
         configured statically by management or dynamically by
         GMRP."
-    INDEX           { dot1qVlanIndex } 
+    INDEX           { dot1qVlanIndex }
     ::= { dot1qForwardAllTable 1 }
 
 Dot1qForwardAllEntry ::= SEQUENCE {
@@ -573,7 +574,7 @@ dot1qForwardAllPorts OBJECT-TYPE
         multicast group-addressed frames are to be forwarded.
         This includes ports for which this need has been
         determined dynamically by GMRP, or configured statically
-        by management." 
+        by management."
     ::= { dot1qForwardAllEntry 1 }
 
 dot1qForwardAllStaticPorts OBJECT-TYPE
@@ -596,7 +597,7 @@ dot1qForwardAllStaticPorts OBJECT-TYPE
         ports.
 
         The value of this object MUST be retained across
-        reinitializations of the management system." 
+        reinitializations of the management system."
     ::= { dot1qForwardAllEntry 2 }
 
 dot1qForwardAllForbiddenPorts OBJECT-TYPE
@@ -614,13 +615,13 @@ dot1qForwardAllForbiddenPorts OBJECT-TYPE
         string of zeros of appropriate length.
 
         The value of this object MUST be retained across
-        reinitializations of the management system." 
+        reinitializations of the management system."
     ::= { dot1qForwardAllEntry 3 }
- 
+
 
 
 dot1qForwardUnregisteredTable OBJECT-TYPE
-    SYNTAX          SEQUENCE OF Dot1qForwardUnregisteredEntry 
+    SYNTAX          SEQUENCE OF Dot1qForwardUnregisteredEntry
     MAX-ACCESS      not-accessible
     STATUS          current
     DESCRIPTION
@@ -644,7 +645,7 @@ dot1qForwardUnregisteredEntry OBJECT-TYPE
         more specific forwarding information shall be forwarded.
         This is configured statically by management or
         dynamically by GMRP."
-    INDEX           { dot1qVlanIndex } 
+    INDEX           { dot1qVlanIndex }
     ::= { dot1qForwardUnregisteredTable 1 }
 
 Dot1qForwardUnregisteredEntry ::= SEQUENCE {
@@ -663,7 +664,7 @@ dot1qForwardUnregisteredPorts OBJECT-TYPE
         more specific forwarding information will be forwarded.
         This includes ports for which this need has been
         determined dynamically by GMRP, or configured statically
-        by management." 
+        by management."
     ::= { dot1qForwardUnregisteredEntry 1 }
 
 dot1qForwardUnregisteredStaticPorts OBJECT-TYPE
@@ -686,7 +687,7 @@ dot1qForwardUnregisteredStaticPorts OBJECT-TYPE
         value of dot1qForwardAllStaticPorts.
 
         The value of this object MUST be retained across
-        reinitializations of the management system." 
+        reinitializations of the management system."
     ::= { dot1qForwardUnregisteredEntry 2 }
 
 dot1qForwardUnregisteredForbiddenPorts OBJECT-TYPE
@@ -704,16 +705,16 @@ dot1qForwardUnregisteredForbiddenPorts OBJECT-TYPE
         is a string of zeros of appropriate length.
 
         The value of this object MUST be retained across
-        reinitializations of the management system." 
+        reinitializations of the management system."
     ::= { dot1qForwardUnregisteredEntry 3 }
- 
+
 
 -- -------------------------------------------------------------
 -- The Static (Destination-Address Filtering) Database
 -- -------------------------------------------------------------
 
 dot1qStaticUnicastTable OBJECT-TYPE
-    SYNTAX          SEQUENCE OF Dot1qStaticUnicastEntry 
+    SYNTAX          SEQUENCE OF Dot1qStaticUnicastEntry
     MAX-ACCESS      not-accessible
     STATUS          current
     DESCRIPTION
@@ -749,14 +750,14 @@ dot1qStaticUnicastEntry OBJECT-TYPE
                         dot1qFdbId,
                         dot1qStaticUnicastAddress,
                         dot1qStaticUnicastReceivePort
-                    } 
+                    }
     ::= { dot1qStaticUnicastTable 1 }
 
 Dot1qStaticUnicastEntry ::= SEQUENCE {
         dot1qStaticUnicastAddress       MacAddress,
         dot1qStaticUnicastReceivePort   Integer32,
         dot1qStaticUnicastAllowedToGoTo PortList,
-        dot1qStaticUnicastStatus        INTEGER 
+        dot1qStaticUnicastStatus        INTEGER
 }
 
 dot1qStaticUnicastAddress OBJECT-TYPE
@@ -766,7 +767,7 @@ dot1qStaticUnicastAddress OBJECT-TYPE
     DESCRIPTION
         "The destination MAC address in a frame to which this
         entry's filtering information applies.  This object must
-        take the value of a unicast address." 
+        take the value of a unicast address."
     ::= { dot1qStaticUnicastEntry 1 }
 
 dot1qStaticUnicastReceivePort OBJECT-TYPE
@@ -778,7 +779,7 @@ dot1qStaticUnicastReceivePort OBJECT-TYPE
         from which a frame must be received in order for this
         entry's filtering information to apply.  A value of zero
         indicates that this entry applies on all ports of the
-        device for which there is no other applicable entry." 
+        device for which there is no other applicable entry."
     ::= { dot1qStaticUnicastEntry 2 }
 
 dot1qStaticUnicastAllowedToGoTo OBJECT-TYPE
@@ -801,7 +802,7 @@ dot1qStaticUnicastAllowedToGoTo OBJECT-TYPE
         The value of this object MUST be retained across
         reinitializations of the management system."
     REFERENCE
-        "IEEE 802.1Q/D11 Table 8-5, ISO/IEC 15802-3 Table 7-5" 
+        "IEEE 802.1Q/D11 Table 8-5, ISO/IEC 15802-3 Table 7-5"
     ::= { dot1qStaticUnicastEntry 3 }
 
 dot1qStaticUnicastStatus OBJECT-TYPE
@@ -833,13 +834,13 @@ dot1qStaticUnicastStatus OBJECT-TYPE
 
         The value of this object MUST be retained across
         reinitializations of the management system."
-    DEFVAL          { permanent } 
+    DEFVAL          { permanent }
     ::= { dot1qStaticUnicastEntry 4 }
- 
+
 
 
 dot1qStaticMulticastTable OBJECT-TYPE
-    SYNTAX          SEQUENCE OF Dot1qStaticMulticastEntry 
+    SYNTAX          SEQUENCE OF Dot1qStaticMulticastEntry
     MAX-ACCESS      not-accessible
     STATUS          current
     DESCRIPTION
@@ -875,7 +876,7 @@ dot1qStaticMulticastEntry OBJECT-TYPE
                         dot1qVlanIndex,
                         dot1qStaticMulticastAddress,
                         dot1qStaticMulticastReceivePort
-                    } 
+                    }
     ::= { dot1qStaticMulticastTable 1 }
 
 Dot1qStaticMulticastEntry ::= SEQUENCE {
@@ -883,7 +884,7 @@ Dot1qStaticMulticastEntry ::= SEQUENCE {
         dot1qStaticMulticastReceivePort          Integer32,
         dot1qStaticMulticastStaticEgressPorts    PortList,
         dot1qStaticMulticastForbiddenEgressPorts PortList,
-        dot1qStaticMulticastStatus               INTEGER 
+        dot1qStaticMulticastStatus               INTEGER
 }
 
 dot1qStaticMulticastAddress OBJECT-TYPE
@@ -893,7 +894,7 @@ dot1qStaticMulticastAddress OBJECT-TYPE
     DESCRIPTION
         "The destination MAC address in a frame to which this
         entry's filtering information applies.  This object must
-        take the value of a Multicast or Broadcast address." 
+        take the value of a Multicast or Broadcast address."
     ::= { dot1qStaticMulticastEntry 1 }
 
 dot1qStaticMulticastReceivePort OBJECT-TYPE
@@ -905,7 +906,7 @@ dot1qStaticMulticastReceivePort OBJECT-TYPE
         from which a frame must be received in order for this
         entry's filtering information to apply.  A value of zero
         indicates that this entry applies on all ports of the
-        device for which there is no other applicable entry." 
+        device for which there is no other applicable entry."
     ::= { dot1qStaticMulticastEntry 2 }
 
 dot1qStaticMulticastStaticEgressPorts OBJECT-TYPE
@@ -923,7 +924,7 @@ dot1qStaticMulticastStaticEgressPorts OBJECT-TYPE
         appropriate length.
 
         The value of this object MUST be retained across
-        reinitializations of the management system." 
+        reinitializations of the management system."
     ::= { dot1qStaticMulticastEntry 3 }
 
 dot1qStaticMulticastForbiddenEgressPorts OBJECT-TYPE
@@ -941,7 +942,7 @@ dot1qStaticMulticastForbiddenEgressPorts OBJECT-TYPE
         appropriate length.
 
         The value of this object MUST be retained across
-        reinitializations of the management system." 
+        reinitializations of the management system."
     ::= { dot1qStaticMulticastEntry 4 }
 
 dot1qStaticMulticastStatus OBJECT-TYPE
@@ -973,9 +974,9 @@ dot1qStaticMulticastStatus OBJECT-TYPE
 
         The value of this object MUST be retained across
         reinitializations of the management system."
-    DEFVAL          { permanent } 
+    DEFVAL          { permanent }
     ::= { dot1qStaticMulticastEntry 5 }
- 
+
 
 
 -- -------------------------------------------------------------
@@ -989,11 +990,11 @@ dot1qVlanNumDeletes OBJECT-TYPE
         "The number of times a VLAN entry has been deleted from
         the dot1qVlanCurrentTable (for any reason).  If an entry
         is deleted, then inserted, and then deleted, this
-        counter will be incremented by 2." 
+        counter will be incremented by 2."
     ::= { dot1qVlan 1 }
 
 dot1qVlanCurrentTable OBJECT-TYPE
-    SYNTAX          SEQUENCE OF Dot1qVlanCurrentEntry 
+    SYNTAX          SEQUENCE OF Dot1qVlanCurrentEntry
     MAX-ACCESS      not-accessible
     STATUS          current
     DESCRIPTION
@@ -1015,7 +1016,7 @@ dot1qVlanCurrentEntry OBJECT-TYPE
     INDEX           {
                         dot1qVlanTimeMark,
                         dot1qVlanIndex
-                    } 
+                    }
     ::= { dot1qVlanCurrentTable 1 }
 
 Dot1qVlanCurrentEntry ::= SEQUENCE {
@@ -1034,7 +1035,7 @@ dot1qVlanTimeMark OBJECT-TYPE
     STATUS          current
     DESCRIPTION
         "A TimeFilter for this entry.  See the TimeFilter
-        textual convention to see how this works." 
+        textual convention to see how this works."
     ::= { dot1qVlanCurrentEntry 1 }
 
 dot1qVlanIndex OBJECT-TYPE
@@ -1042,7 +1043,7 @@ dot1qVlanIndex OBJECT-TYPE
     MAX-ACCESS      not-accessible
     STATUS          current
     DESCRIPTION
-        "The VLAN-ID or other identifier referring to this VLAN." 
+        "The VLAN-ID or other identifier referring to this VLAN."
     ::= { dot1qVlanCurrentEntry 2 }
 
 dot1qVlanFdbId OBJECT-TYPE
@@ -1057,7 +1058,7 @@ dot1qVlanFdbId OBJECT-TYPE
         the VLAN is created: either dynamically by GVRP, or by
         management, in dot1qVlanStaticTable.  Allocation of this
         value follows the learning constraints defined for this
-        VLAN in dot1qLearningConstraintsTable." 
+        VLAN in dot1qLearningConstraintsTable."
     ::= { dot1qVlanCurrentEntry 3 }
 
 dot1qVlanCurrentEgressPorts OBJECT-TYPE
@@ -1067,7 +1068,7 @@ dot1qVlanCurrentEgressPorts OBJECT-TYPE
     DESCRIPTION
         "The set of ports that are transmitting traffic for
         this VLAN as either tagged or untagged frames."
-    REFERENCE       "IEEE 802.1Q/D11 Section 12.10.2.1" 
+    REFERENCE       "IEEE 802.1Q/D11 Section 12.10.2.1"
     ::= { dot1qVlanCurrentEntry 4 }
 
 dot1qVlanCurrentUntaggedPorts OBJECT-TYPE
@@ -1077,7 +1078,7 @@ dot1qVlanCurrentUntaggedPorts OBJECT-TYPE
     DESCRIPTION
         "The set of ports that are transmitting traffic for
         this VLAN as untagged frames."
-    REFERENCE       "IEEE 802.1Q/D11 Section 12.10.2.1" 
+    REFERENCE       "IEEE 802.1Q/D11 Section 12.10.2.1"
     ::= { dot1qVlanCurrentEntry 5 }
 
 dot1qVlanStatus OBJECT-TYPE
@@ -1103,7 +1104,7 @@ dot1qVlanStatus OBJECT-TYPE
 
             and will remain so until removed by GVRP.  There
             is no static entry for this VLAN, and it will be
-            removed when the last port leaves the VLAN." 
+            removed when the last port leaves the VLAN."
     ::= { dot1qVlanCurrentEntry 6 }
 
 dot1qVlanCreationTime OBJECT-TYPE
@@ -1111,16 +1112,16 @@ dot1qVlanCreationTime OBJECT-TYPE
     MAX-ACCESS      read-only
     STATUS          current
     DESCRIPTION
-        "The value of sysUpTime when this VLAN was created." 
+        "The value of sysUpTime when this VLAN was created."
     ::= { dot1qVlanCurrentEntry 7 }
- 
+
 
 -- -------------------------------------------------------------
 -- The Static VLAN Database
 -- -------------------------------------------------------------
 
 dot1qVlanStaticTable OBJECT-TYPE
-    SYNTAX          SEQUENCE OF Dot1qVlanStaticEntry 
+    SYNTAX          SEQUENCE OF Dot1qVlanStaticEntry
     MAX-ACCESS      not-accessible
     STATUS          current
     DESCRIPTION
@@ -1137,7 +1138,7 @@ dot1qVlanStaticEntry OBJECT-TYPE
     DESCRIPTION
         "Static information for a VLAN configured into the
         device by (local or network) management."
-    INDEX           { dot1qVlanIndex } 
+    INDEX           { dot1qVlanIndex }
     ::= { dot1qVlanStaticTable 1 }
 
 Dot1qVlanStaticEntry ::= SEQUENCE {
@@ -1155,7 +1156,7 @@ dot1qVlanStaticName OBJECT-TYPE
     DESCRIPTION
         "An administratively assigned string, which may be used
         to identify the VLAN."
-    REFERENCE       "IEEE 802.1Q/D11 Section 12.10.2.1" 
+    REFERENCE       "IEEE 802.1Q/D11 Section 12.10.2.1"
     ::= { dot1qVlanStaticEntry 1 }
 
 dot1qVlanStaticEgressPorts OBJECT-TYPE
@@ -1172,7 +1173,7 @@ dot1qVlanStaticEgressPorts OBJECT-TYPE
         the set of ports in dot1qVlanForbiddenEgressPorts.  The
         default value of this object is a string of zeros of
         appropriate length, indicating not fixed."
-    REFERENCE       "IEEE 802.1Q/D11 Section 12.7.7.3, 11.2.3.2.3" 
+    REFERENCE       "IEEE 802.1Q/D11 Section 12.7.7.3, 11.2.3.2.3"
     ::= { dot1qVlanStaticEntry 2 }
 
 dot1qVlanForbiddenEgressPorts OBJECT-TYPE
@@ -1190,7 +1191,7 @@ dot1qVlanForbiddenEgressPorts OBJECT-TYPE
         in dot1qVlanStaticEgressPorts.  The default value of
         this object is a string of zeros of appropriate length,
         excluding all ports from the forbidden set."
-    REFERENCE       "IEEE 802.1Q/D11 Section 12.7.7.3, 11.2.3.2.3" 
+    REFERENCE       "IEEE 802.1Q/D11 Section 12.7.7.3, 11.2.3.2.3"
     ::= { dot1qVlanStaticEntry 3 }
 
 dot1qVlanStaticUntaggedPorts OBJECT-TYPE
@@ -1208,16 +1209,16 @@ dot1qVlanStaticUntaggedPorts OBJECT-TYPE
         manager might attempt to set more than one VLAN to be untagged
         on egress where the device does not support this IEEE 802.1Q
         option."
-    REFERENCE       "IEEE 802.1Q/D11 Section 12.10.2.1" 
+    REFERENCE       "IEEE 802.1Q/D11 Section 12.10.2.1"
     ::= { dot1qVlanStaticEntry 4 }
 
 dot1qVlanStaticRowStatus OBJECT-TYPE
     SYNTAX          RowStatus
     MAX-ACCESS      read-create
     STATUS          current
-    DESCRIPTION     "This object indicates the status of this entry." 
+    DESCRIPTION     "This object indicates the status of this entry."
     ::= { dot1qVlanStaticEntry 5 }
- 
+
 
 
 dot1qNextFreeLocalVlanIndex OBJECT-TYPE
@@ -1241,14 +1242,14 @@ dot1qNextFreeLocalVlanIndex OBJECT-TYPE
         and the creation re-tried with the new value.
 
         This value will automatically change when the current value is
-        used to create a new row." 
+        used to create a new row."
     ::= { dot1qVlan 4 }
 -- -------------------------------------------------------------
 -- The VLAN Port Configuration Table
 -- -------------------------------------------------------------
 
 dot1qPortVlanTable OBJECT-TYPE
-    SYNTAX          SEQUENCE OF Dot1qPortVlanEntry 
+    SYNTAX          SEQUENCE OF Dot1qPortVlanEntry
     MAX-ACCESS      not-accessible
     STATUS          current
     DESCRIPTION
@@ -1263,8 +1264,8 @@ dot1qPortVlanEntry OBJECT-TYPE
     DESCRIPTION
         "Information controlling VLAN configuration for a port
         on the device.  This is indexed by dot1dBasePort."
-    AUGMENTS           { dot1dBasePortEntry  } 
- 
+    AUGMENTS           { dot1dBasePortEntry  }
+
     ::= { dot1qPortVlanTable 1 }
 
 Dot1qPortVlanEntry ::= SEQUENCE {
@@ -1288,7 +1289,7 @@ dot1qPvid OBJECT-TYPE
         The value of this object MUST be retained across
         reinitializations of the management system."
     REFERENCE       "IEEE 802.1Q/D11 Section 12.10.1.1"
-    DEFVAL          { 1 } 
+    DEFVAL          { 1 }
     ::= { dot1qPortVlanEntry 1 }
 
 dot1qPortAcceptableFrameTypes OBJECT-TYPE
@@ -1314,7 +1315,7 @@ dot1qPortAcceptableFrameTypes OBJECT-TYPE
         The value of this object MUST be retained across
         reinitializations of the management system."
     REFERENCE       "IEEE 802.1Q/D11 Section 12.10.1.3"
-    DEFVAL          { admitAll } 
+    DEFVAL          { admitAll }
     ::= { dot1qPortVlanEntry 2 }
 
 dot1qPortIngressFiltering OBJECT-TYPE
@@ -1335,7 +1336,7 @@ dot1qPortIngressFiltering OBJECT-TYPE
         The value of this object MUST be retained across
         reinitializations of the management system."
     REFERENCE       "IEEE 802.1Q/D11 Section 12.10.1.4"
-    DEFVAL          { false } 
+    DEFVAL          { false }
     ::= { dot1qPortVlanEntry 3 }
 
 dot1qPortGvrpStatus OBJECT-TYPE
@@ -1357,7 +1358,7 @@ dot1qPortGvrpStatus OBJECT-TYPE
 
         The value of this object MUST be retained across
         reinitializations of the management system."
-    DEFVAL          { enabled } 
+    DEFVAL          { enabled }
     ::= { dot1qPortVlanEntry 4 }
 
 dot1qPortGvrpFailedRegistrations OBJECT-TYPE
@@ -1366,7 +1367,7 @@ dot1qPortGvrpFailedRegistrations OBJECT-TYPE
     STATUS          current
     DESCRIPTION
         "The total number of failed GVRP registrations, for any
-        reason, on this port." 
+        reason, on this port."
     ::= { dot1qPortVlanEntry 5 }
 
 dot1qPortGvrpLastPduOrigin OBJECT-TYPE
@@ -1375,7 +1376,7 @@ dot1qPortGvrpLastPduOrigin OBJECT-TYPE
     STATUS          current
     DESCRIPTION
         "The Source MAC Address of the last GVRP message
-        received on this port." 
+        received on this port."
     ::= { dot1qPortVlanEntry 6 }
 
 dot1qPortRestrictedVlanRegistration OBJECT-TYPE
@@ -1393,16 +1394,16 @@ dot1qPortRestrictedVlanRegistration OBJECT-TYPE
         The value of this object MUST be retained across
         reinitializations of the management system."
     REFERENCE       "IEEE 802.1u clause 11.2.3.2.3, 12.10.1.7."
-    DEFVAL          { false } 
+    DEFVAL          { false }
     ::= { dot1qPortVlanEntry 7 }
- 
+
 
 -- -------------------------------------------------------------
 -- Per port VLAN Statistics Table
 -- -------------------------------------------------------------
 
 dot1qPortVlanStatisticsTable OBJECT-TYPE
-    SYNTAX          SEQUENCE OF Dot1qPortVlanStatisticsEntry 
+    SYNTAX          SEQUENCE OF Dot1qPortVlanStatisticsEntry
     MAX-ACCESS      not-accessible
     STATUS          current
     DESCRIPTION
@@ -1432,7 +1433,7 @@ dot1qPortVlanStatisticsEntry OBJECT-TYPE
     INDEX           {
                         dot1dBasePort,
                         dot1qVlanIndex
-                    } 
+                    }
     ::= { dot1qPortVlanStatisticsTable 1 }
 
 Dot1qPortVlanStatisticsEntry ::= SEQUENCE {
@@ -1457,7 +1458,7 @@ dot1qTpVlanPortInFrames OBJECT-TYPE
         for this VLAN.  This object includes received bridge
         management frames classified as belonging to this VLAN
         (e.g., GMRP, but not GVRP or STP."
-    REFERENCE       "IEEE 802.1Q/D11 Section 12.6.1.1.3(a)" 
+    REFERENCE       "IEEE 802.1Q/D11 Section 12.6.1.1.3(a)"
     ::= { dot1qPortVlanStatisticsEntry 1 }
 
 dot1qTpVlanPortOutFrames OBJECT-TYPE
@@ -1470,7 +1471,7 @@ dot1qTpVlanPortOutFrames OBJECT-TYPE
         VLAN.  This includes bridge management frames originated
         by this device that are classified as belonging to this
         VLAN (e.g., GMRP, but not GVRP or STP)."
-    REFERENCE       "IEEE 802.1Q/D11 Section 12.6.1.1.3(d)" 
+    REFERENCE       "IEEE 802.1Q/D11 Section 12.6.1.1.3(d)"
     ::= { dot1qPortVlanStatisticsEntry 2 }
 
 dot1qTpVlanPortInDiscards OBJECT-TYPE
@@ -1483,7 +1484,7 @@ dot1qTpVlanPortInDiscards OBJECT-TYPE
         VLAN and that were discarded due to VLAN-related reasons.
         Specifically, the IEEE 802.1Q counters for Discard
         Inbound and Discard on Ingress Filtering."
-    REFERENCE       "IEEE 802.1Q/D11 Section 12.6.1.1.3" 
+    REFERENCE       "IEEE 802.1Q/D11 Section 12.6.1.1.3"
     ::= { dot1qPortVlanStatisticsEntry 3 }
 
 dot1qTpVlanPortInOverflowFrames OBJECT-TYPE
@@ -1493,7 +1494,7 @@ dot1qTpVlanPortInOverflowFrames OBJECT-TYPE
     DESCRIPTION
         "The number of times the associated
         dot1qTpVlanPortInFrames counter has overflowed."
-    REFERENCE       "ISO/IEC 15802-3 Section 14.6.1.1.3" 
+    REFERENCE       "ISO/IEC 15802-3 Section 14.6.1.1.3"
     ::= { dot1qPortVlanStatisticsEntry 4 }
 
 dot1qTpVlanPortOutOverflowFrames OBJECT-TYPE
@@ -1503,7 +1504,7 @@ dot1qTpVlanPortOutOverflowFrames OBJECT-TYPE
     DESCRIPTION
         "The number of times the associated
         dot1qTpVlanPortOutFrames counter has overflowed."
-    REFERENCE       "ISO/IEC 15802-3 Section 14.6.1.1.3" 
+    REFERENCE       "ISO/IEC 15802-3 Section 14.6.1.1.3"
     ::= { dot1qPortVlanStatisticsEntry 5 }
 
 dot1qTpVlanPortInOverflowDiscards OBJECT-TYPE
@@ -1513,13 +1514,13 @@ dot1qTpVlanPortInOverflowDiscards OBJECT-TYPE
     DESCRIPTION
         "The number of times the associated
         dot1qTpVlanPortInDiscards counter has overflowed."
-    REFERENCE       "ISO/IEC 15802-3 Section 14.6.1.1.3" 
+    REFERENCE       "ISO/IEC 15802-3 Section 14.6.1.1.3"
     ::= { dot1qPortVlanStatisticsEntry 6 }
- 
+
 
 
 dot1qPortVlanHCStatisticsTable OBJECT-TYPE
-    SYNTAX          SEQUENCE OF Dot1qPortVlanHCStatisticsEntry 
+    SYNTAX          SEQUENCE OF Dot1qPortVlanHCStatisticsEntry
     MAX-ACCESS      not-accessible
     STATUS          current
     DESCRIPTION
@@ -1537,7 +1538,7 @@ dot1qPortVlanHCStatisticsEntry OBJECT-TYPE
     INDEX           {
                         dot1dBasePort,
                         dot1qVlanIndex
-                    } 
+                    }
     ::= { dot1qPortVlanHCStatisticsTable 1 }
 
 Dot1qPortVlanHCStatisticsEntry ::= SEQUENCE {
@@ -1560,7 +1561,7 @@ dot1qTpVlanPortHCInFrames OBJECT-TYPE
         for this VLAN.  This object includes received bridge
         management frames classified as belonging to this VLAN
         (e.g., GMRP, but not GVRP or STP)."
-    REFERENCE       "IEEE 802.1Q/D11 Section 12.6.1.1.3(a)" 
+    REFERENCE       "IEEE 802.1Q/D11 Section 12.6.1.1.3(a)"
     ::= { dot1qPortVlanHCStatisticsEntry 1 }
 
 dot1qTpVlanPortHCOutFrames OBJECT-TYPE
@@ -1573,7 +1574,7 @@ dot1qTpVlanPortHCOutFrames OBJECT-TYPE
         VLAN.  This includes bridge management frames originated
         by this device that are classified as belonging to this
         VLAN (e.g., GMRP, but not GVRP or STP)."
-    REFERENCE       "IEEE 802.1Q/D11 Section 12.6.1.1.3(d)" 
+    REFERENCE       "IEEE 802.1Q/D11 Section 12.6.1.1.3(d)"
     ::= { dot1qPortVlanHCStatisticsEntry 2 }
 
 dot1qTpVlanPortHCInDiscards OBJECT-TYPE
@@ -1586,16 +1587,16 @@ dot1qTpVlanPortHCInDiscards OBJECT-TYPE
         VLAN and that were discarded due to VLAN-related reasons.
         Specifically, the IEEE 802.1Q counters for Discard
         Inbound and Discard on Ingress Filtering."
-    REFERENCE       "IEEE 802.1Q/D11 Section 12.6.1.1.3" 
+    REFERENCE       "IEEE 802.1Q/D11 Section 12.6.1.1.3"
     ::= { dot1qPortVlanHCStatisticsEntry 3 }
- 
+
 
 -- -------------------------------------------------------------
 -- The VLAN Learning Constraints Table
 -- -------------------------------------------------------------
 
 dot1qLearningConstraintsTable OBJECT-TYPE
-    SYNTAX          SEQUENCE OF Dot1qLearningConstraintsEntry 
+    SYNTAX          SEQUENCE OF Dot1qLearningConstraintsEntry
     MAX-ACCESS      not-accessible
     STATUS          current
     DESCRIPTION
@@ -1612,7 +1613,7 @@ dot1qLearningConstraintsEntry OBJECT-TYPE
     INDEX           {
                         dot1qConstraintVlan,
                         dot1qConstraintSet
-                    } 
+                    }
     ::= { dot1qLearningConstraintsTable 1 }
 
 Dot1qLearningConstraintsEntry ::= SEQUENCE {
@@ -1628,7 +1629,7 @@ dot1qConstraintVlan OBJECT-TYPE
     STATUS          current
     DESCRIPTION
         "The index of the row in dot1qVlanCurrentTable for the
-        VLAN constrained by this entry." 
+        VLAN constrained by this entry."
     ::= { dot1qLearningConstraintsEntry 1 }
 
 dot1qConstraintSet OBJECT-TYPE
@@ -1638,7 +1639,7 @@ dot1qConstraintSet OBJECT-TYPE
     DESCRIPTION
         "The identity of the constraint set to which
         dot1qConstraintVlan belongs.  These values may be chosen
-        by the management station." 
+        by the management station."
     ::= { dot1qLearningConstraintsEntry 2 }
 
 dot1qConstraintType OBJECT-TYPE
@@ -1656,16 +1657,16 @@ dot1qConstraintType OBJECT-TYPE
             dot1qConstraintSet.
         shared(2) - the VLAN, dot1qConstraintVlan, shares
             the same filtering database as all other VLANs
-            in the same set, defined by dot1qConstraintSet." 
+            in the same set, defined by dot1qConstraintSet."
     ::= { dot1qLearningConstraintsEntry 3 }
 
 dot1qConstraintStatus OBJECT-TYPE
     SYNTAX          RowStatus
     MAX-ACCESS      read-create
     STATUS          current
-    DESCRIPTION     "The status of this entry." 
+    DESCRIPTION     "The status of this entry."
     ::= { dot1qLearningConstraintsEntry 4 }
- 
+
 
 
 dot1qConstraintSetDefault OBJECT-TYPE
@@ -1678,7 +1679,7 @@ dot1qConstraintSetDefault OBJECT-TYPE
         in dot1qLearningConstraintsTable.
 
         The value of this object MUST be retained across
-        reinitializations of the management system." 
+        reinitializations of the management system."
     ::= { dot1qVlan 9 }
 
 dot1qConstraintTypeDefault OBJECT-TYPE
@@ -1696,14 +1697,14 @@ dot1qConstraintTypeDefault OBJECT-TYPE
 
         The value of this object MUST be retained across
 
-        reinitializations of the management system." 
+        reinitializations of the management system."
     ::= { dot1qVlan 10 }
 -- -------------------------------------------------------------
 -- dot1vProtocol subtree
 -- -------------------------------------------------------------
 
 dot1vProtocolGroupTable OBJECT-TYPE
-    SYNTAX          SEQUENCE OF Dot1vProtocolGroupEntry 
+    SYNTAX          SEQUENCE OF Dot1vProtocolGroupEntry
     MAX-ACCESS      not-accessible
     STATUS          current
     DESCRIPTION
@@ -1723,12 +1724,12 @@ dot1vProtocolGroupEntry OBJECT-TYPE
     INDEX           {
                         dot1vProtocolTemplateFrameType,
                         dot1vProtocolTemplateProtocolValue
-                    } 
+                    }
     ::= { dot1vProtocolGroupTable 1 }
 
 Dot1vProtocolGroupEntry ::= SEQUENCE {
         dot1vProtocolTemplateFrameType     INTEGER ,
-        dot1vProtocolTemplateProtocolValue OCTET STRING,
+        dot1vProtocolTemplateProtocolValue DisplayString,
         dot1vProtocolGroupId               Integer32,
         dot1vProtocolGroupRowStatus        RowStatus
 }
@@ -1746,11 +1747,11 @@ dot1vProtocolTemplateFrameType OBJECT-TYPE
     DESCRIPTION
         "The data-link encapsulation format or the
         'detagged_frame_type' in a Protocol Template."
-    REFERENCE       "IEEE 802.1v clause 8.6.2" 
+    REFERENCE       "IEEE 802.1v clause 8.6.2"
     ::= { dot1vProtocolGroupEntry 1 }
 
 dot1vProtocolTemplateProtocolValue OBJECT-TYPE
-    SYNTAX          OCTET STRING (SIZE  (2  |  5))
+    SYNTAX          DisplayString (SIZE  (2  |  5))
     MAX-ACCESS      not-accessible
     STATUS          current
     DESCRIPTION
@@ -1768,7 +1769,7 @@ dot1vProtocolTemplateProtocolValue OBJECT-TYPE
             Point (LSAP) pair: first octet for Destination Service
             Access Point (DSAP) and second octet for Source Service
             Access Point (SSAP)."
-    REFERENCE       "IEEE 802.1v clause 8.6.2" 
+    REFERENCE       "IEEE 802.1v clause 8.6.2"
     ::= { dot1vProtocolGroupEntry 2 }
 
 dot1vProtocolGroupId OBJECT-TYPE
@@ -1778,20 +1779,20 @@ dot1vProtocolGroupId OBJECT-TYPE
     DESCRIPTION
         "Represents a group of protocols that are associated
         together when assigning a VID to a frame."
-    REFERENCE       "IEEE 802.1v clause 8.6.3, 12.10.2.1" 
+    REFERENCE       "IEEE 802.1v clause 8.6.3, 12.10.2.1"
     ::= { dot1vProtocolGroupEntry 3 }
 
 dot1vProtocolGroupRowStatus OBJECT-TYPE
     SYNTAX          RowStatus
     MAX-ACCESS      read-create
     STATUS          current
-    DESCRIPTION     "This object indicates the status of this entry." 
+    DESCRIPTION     "This object indicates the status of this entry."
     ::= { dot1vProtocolGroupEntry 4 }
- 
+
 
 
 dot1vProtocolPortTable OBJECT-TYPE
-    SYNTAX          SEQUENCE OF Dot1vProtocolPortEntry 
+    SYNTAX          SEQUENCE OF Dot1vProtocolPortEntry
     MAX-ACCESS      not-accessible
     STATUS          current
     DESCRIPTION
@@ -1808,7 +1809,7 @@ dot1vProtocolPortEntry OBJECT-TYPE
     INDEX           {
                         dot1dBasePort,
                         dot1vProtocolPortGroupId
-                    } 
+                    }
     ::= { dot1vProtocolPortTable 1 }
 
 Dot1vProtocolPortEntry ::= SEQUENCE {
@@ -1824,7 +1825,7 @@ dot1vProtocolPortGroupId OBJECT-TYPE
     DESCRIPTION
         "Designates a group of protocols in the Protocol
         Group Database."
-    REFERENCE       "IEEE 802.1v clause 8.6.3, 12.10.1.2" 
+    REFERENCE       "IEEE 802.1v clause 8.6.3, 12.10.1.2"
     ::= { dot1vProtocolPortEntry 1 }
 
 dot1vProtocolPortGroupVid OBJECT-TYPE
@@ -1834,16 +1835,16 @@ dot1vProtocolPortGroupVid OBJECT-TYPE
     DESCRIPTION
         "The VID associated with a group of protocols for
         each port."
-    REFERENCE       "IEEE 802.1v clause 8.4.4, 12.10.1.2" 
+    REFERENCE       "IEEE 802.1v clause 8.4.4, 12.10.1.2"
     ::= { dot1vProtocolPortEntry 2 }
 
 dot1vProtocolPortRowStatus OBJECT-TYPE
     SYNTAX          RowStatus
     MAX-ACCESS      read-create
     STATUS          current
-    DESCRIPTION     "This object indicates the status of this entry." 
+    DESCRIPTION     "This object indicates the status of this entry."
     ::= { dot1vProtocolPortEntry 3 }
- 
+
 
 -- -------------------------------------------------------------
 -- IEEE 802.1Q MIB - Conformance Information
